<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Thomas A. Sasani" />
  <meta name="author" content="Aaron R. Quinlan" />
  <meta name="author" content="Kelley Harris" />
  <meta name="dcterms.date" content="2023-02-22" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>Discovering epistasis between germline mutator alleles in mice</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta property="og:type" content="article" />
  <meta name="dc.title" content="Discovering epistasis between germline mutator alleles in mice" />
  <meta name="citation_title" content="Discovering epistasis between germline mutator alleles in mice" />
  <meta property="og:title" content="Discovering epistasis between germline mutator alleles in mice" />
  <meta property="twitter:title" content="Discovering epistasis between germline mutator alleles in mice" />
  <meta name="dc.date" content="2023-02-22" />
  <meta name="citation_publication_date" content="2023-02-22" />
  <meta property="article:published_time" content="2023-02-22" />
  <meta name="dc.modified" content="2023-02-22T14:32:33+00:00" />
  <meta property="article:modified_time" content="2023-02-22T14:32:33+00:00" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Thomas A. Sasani" />
  <meta name="citation_author_institution" content="Department of Human Genetics, University of Utah" />
  <meta name="citation_author_orcid" content="0000-0003-2317-1374" />
  <meta name="twitter:creator" content="@tomsasani" />
  <meta name="citation_author" content="Aaron R. Quinlan" />
  <meta name="citation_author_institution" content="Department of Human Genetics, University of Utah" />
  <meta name="citation_author_institution" content="Department of Biomedical Informatics, University of Utah" />
  <meta name="citation_author_orcid" content="0000-0003-1756-0859" />
  <meta name="twitter:creator" content="@aaronquinlan" />
  <meta name="citation_author" content="Kelley Harris" />
  <meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington" />
  <meta name="citation_author_orcid" content="0000-0003-0302-2523" />
  <meta name="twitter:creator" content="@Kelley__Harris" />
  <link rel="canonical" href="https://quinlan-lab.github.io/mutator-epistasis-manuscript/" />
  <meta property="og:url" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/" />
  <meta property="twitter:url" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/" />
  <meta name="citation_pdf_url" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://quinlan-lab.github.io/mutator-epistasis-manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://quinlan-lab.github.io/mutator-epistasis-manuscript/v/7261c41a9500345a3100ac6e1e449e5d1aca192c/" />
  <meta name="manubot_html_url_versioned" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/v/7261c41a9500345a3100ac6e1e449e5d1aca192c/" />
  <meta name="manubot_pdf_url_versioned" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/v/7261c41a9500345a3100ac6e1e449e5d1aca192c/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Discovering epistasis between germline mutator alleles in mice</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://quinlan-lab.github.io/mutator-epistasis-manuscript/v/7261c41a9500345a3100ac6e1e449e5d1aca192c/">permalink</a>)
was automatically generated
from <a href="https://github.com/quinlan-lab/mutator-epistasis-manuscript/tree/7261c41a9500345a3100ac6e1e449e5d1aca192c">quinlan-lab/mutator-epistasis-manuscript@7261c41</a>
on February 22, 2023.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Thomas A. Sasani</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2317-1374">0000-0003-2317-1374</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/tomsasani">tomsasani</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/tomsasani">tomsasani</a>
<br>
<small>
Department of Human Genetics, University of Utah
· Funded by Grant XXXXXXXX
</small></p></li>
<li><p><strong>Aaron R. Quinlan</strong>
<sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1756-0859">0000-0003-1756-0859</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/aaronquinlan">aaronquinlan</a>
<br>
<small>
Department of Human Genetics, University of Utah; Department of Biomedical Informatics, University of Utah
</small></p></li>
<li><p><strong>Kelley Harris</strong>
<sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0302-2523">0000-0003-0302-2523</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/Kelley__Harris">Kelley__Harris</a>
<br>
<small>
Department of Genome Sciences, University of Washington
</small></p></li>
</ul>
<div id="correspondence">
<p>✉ — Correspondence possible via <a href="https://github.com/quinlan-lab/mutator-epistasis-manuscript/issues">GitHub Issues</a>
or email to
Aaron R. Quinlan &lt;aquinlan@genetics.utah.edu&gt;,
Kelley Harris &lt;harriske@uw.edu&gt;.</p>
</div>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Maintaining genome integrity in the mammalian germline is essential and enormously complex. Hundreds of proteins comprise pathways involved in DNA replication, and hundreds more are mobilized to repair DNA damage <span class="citation" data-cites="PMID:28485537">[<a href="#ref-PMID:28485537" role="doc-biblioref"><strong>PMID:28485537?</strong></a>]</span>. While loss-of-function mutations in any of the genes encoding these proteins might lead to elevated mutation rates, <em>mutator alleles</em> have largely eluded detection in mammals.</p>
<p>DNA replication and repair proteins often recognize particular sequence motifs or excise lesions at specific nucleotides. Thus, we might expect that the spectrum of de novo mutations — i.e, the frequency of each individual mutation type (C&gt;T, A&gt;G, etc.) — will differ between genomes that harbor either a mutator or wild-type allele at a given locus. Previously, we used quantitative trait locus (QTL) mapping to discover a mutator allele near the DNA repair gene <em>Mutyh</em> that increases the rate of <em>de novo</em> C&gt;A germline mutation in a collection of recombinant inbred lines (RILs) known as the BXDs <span class="citation" data-cites="PMID:35545679 PMID:33472028">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>,<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>.</p>
<p>In this study, we developed a new method to detect alleles that affect the mutation spectrum in biparental RILs. By applying this method to mutation data from the BXDs, we confirmed the activity of the germline mutator locus near <em>Mutyh</em>, and discovered an additional C&gt;A germline mutator locus on chromosome 6 that overlaps <em>Ogg1</em>, a key partner of <em>Mutyh</em> in base-excision repair of oxidative DNA damage <span class="citation" data-cites="PMID:17581577">[<a href="#ref-PMID:17581577" role="doc-biblioref"><strong>PMID:17581577?</strong></a>]</span>. Strikingly, BXDs with the mutator allele near <em>Ogg1</em> do not exhibit elevated rates of C&gt;A germline mutation unless they also possess the mutator allele near <em>Mutyh</em>, but BXDs with both alleles exhibit even higher C&gt;A mutation rates than those with either one alone.</p>
<p>To our knowledge, these new methods for analyzing mutation spectra reveal the first evidence of epistasis between mammalian germline mutator alleles, and may be applicable to mutation data from humans and other model organisms.</p>
<h2 id="introduction">Introduction</h2>
<p>The germline mutation rate is a fundamental parameter in population genetics, and reflects the complex interplay between DNA replication and repair pathways, exogenous sources of DNA damage, and life-history traits. <em>Mutator alleles</em> may explain some of the within- and between-species variation in germline mutation rates <span class="citation" data-cites="PMID:32619789">[<a href="#ref-PMID:32619789" role="doc-biblioref"><strong>PMID:32619789?</strong></a>]</span>, but have proven challenging to identify in mammalian genomes.</p>
<!--In microorganisms like *Escherichia coli*, mutator alleles have emerged during experimental evolution [@PMID:], and human cancer genomes often harbor deleterious mutator alleles as well [@PMID:27197248].-->
<p>Germline mutator alleles are difficult to detect for a number of reasons, including the fidelity of germline genome replication and the effects of selection on mutators. On average, humans are born with 30 to 50 single-nucleotide <em>de novo</em> germline mutations per haploid genome <span class="citation" data-cites="PMID:28959963 PMID:31549960">[<a href="#ref-PMID:28959963" role="doc-biblioref"><strong>PMID:28959963?</strong></a>,<a href="#ref-PMID:31549960" role="doc-biblioref"><strong>PMID:31549960?</strong></a>]</span>; in mice, that number is closer to 10 or 15 <span class="citation" data-cites="PMID:31492841">[<a href="#ref-PMID:31492841" role="doc-biblioref"><strong>PMID:31492841?</strong></a>]</span>. Due to the low baseline germline mutation rate in many mammals, it can be challenging to ascertain sequencing data from enough haplotypes to reliably detect those with significantly elevated <em>de novo</em> mutation counts. Moreover, in a population of sufficiently large <span class="math inline">\(N_e\)</span> (effective population size), large-effect mutator alleles will likely be efficiently purged by negative selection. The estimated selection coefficient on a mutator allele is approximately <span class="math inline">\(2s \Delta U\)</span> <span class="citation" data-cites="PMID:27739533">[<a href="#ref-PMID:27739533" role="doc-biblioref"><strong>PMID:27739533?</strong></a>]</span>, where <span class="math inline">\(s\)</span> is the mean selective coefficient on a new deleterious mutation and <span class="math inline">\(\Delta U\)</span> is the excess number of new deleterious mutations caused by the mutator allele; the product of <span class="math inline">\(s\)</span> and <span class="math inline">\(\Delta U\)</span> is multiplied by <span class="math inline">\(2\)</span> to account for the average number of generations for which mutator is linked to the excess mutations it causes.</p>
<p>Compared to haplotypes that harbor wild-type alleles at a particular locus, those harboring mutator alleles will likely carry an excess of total germline mutations. Indeed, candidate germline mutator loci have been discovered in human genomes by identifying haplotypes with significantly more derived alleles than the population mean <span class="citation" data-cites="PMID:28095480">[<a href="#ref-PMID:28095480" role="doc-biblioref"><strong>PMID:28095480?</strong></a>]</span>. However, protein-coding genes involved in DNA replication and repair often recognize particular sequence motifs or excise lesions at specific nucleotides <span class="citation" data-cites="PMID:32619789">[<a href="#ref-PMID:32619789" role="doc-biblioref"><strong>PMID:32619789?</strong></a>]</span>. Thus, we might also expect the spectrum of de novo mutations — that is, the frequency of each individual mutation type (C&gt;T, A&gt;G, etc.) — to differ between genomes that carry either a mutator or wild-type allele at a given locus.</p>
<p>In 2022, we discovered a germline mutator allele in mice by analyzing whole-genome sequencing data from 152 recombinant inbred lines (RILs). Commonly known as the <ins>B</ins>X<ins>D</ins>s <span class="citation" data-cites="PMID:33472028">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>, these RILs were derived from either F2 or advanced intercrosses of C57<ins>B</ins>L/6J and <ins>D</ins>BA/2J, two laboratory strains that exhibit significant differences in their germline mutation spectra <span class="citation" data-cites="PMID:30753674">[<a href="#ref-PMID:30753674" role="doc-biblioref"><strong>PMID:30753674?</strong></a>]</span>. Since the BXD RILs were maintained via brother-sister mating for up to 180 generations and housed in a controlled laboratory environment, they were an ideal population for mutator allele discovery. Namely, each line accumulated hundreds or thousands of germline mutations on a nearly-homozygous linear mosaic of parental <ins>B</ins> and <ins>D</ins> haplotypes, while the effects of negative selection on new and standing variation were attenuated by strict inbreeding <span class="citation" data-cites="LF8DCRcP">[<a href="#ref-LF8DCRcP" role="doc-biblioref">1</a>]</span>. We used quantitative trait locus (QTL) mapping to identify a locus on chromosome 4 that was strongly associated with the C&gt;A germline mutation rate in the BXDs <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>. The QTL overlapped <em>Mutyh</em>, which encodes a protein that normally prevents C&gt;A mutations by repairing oxidative DNA damage <span class="citation" data-cites="PMID:17581577">[<a href="#ref-PMID:17581577" role="doc-biblioref"><strong>PMID:17581577?</strong></a>]</span>, and we hypothesized that missense mutations in <em>Mutyh</em> were responsible for a 50% increase in the C&gt;A mutation rate between BXDs with either parental haplotype at the QTL <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.</p>
<p>In this study, we developed a new method to detect alleles that affect the mutation spectrum in biparental RILs, and applied it to <em>de novo</em> germline mutation data from the BXDs. We assessed its power to detect candidate mutator alleles, re-identified the mutator near <em>Mutyh</em>, and discovered compelling evidence of epistasis between two germline mutator alleles that augment the C&gt;A germline mutation rate.</p>
<h2 id="materials-and-methods">Materials and Methods</h2>
<h3 id="identifying-de-novo-germline-mutations-in-the-bxd-rils">Identifying <em>de novo</em> germline mutations in the BXD RILs</h3>
<p>The BXD resource currently comprises a total of 152 recombinant inbred lines (RILs). RILs were derived from either F2 or advanced intercrosses, and subsequently inbred by brother-sister mating for up to 180 generations <span class="citation" data-cites="PMID:33472028">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>. Previously, we analyzed whole-genome sequencing data from the BXDs and identified candidate <em>de novo</em> germline mutations in each line <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>. A detailed description of the methods used for DNA extraction, sequencing, alignment, and variant processing, as well as the characteristics of the <em>de novo</em> mutations, are available in a previous manuscript <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.</p>
<p>Briefly, we identified private single-nucleotide mutations in each BXD that were absent from all other RILs, as well as from the C57BL/6J and DBA/2J parents. We required each private variant to be meet the following criteria:</p>
<ul>
<li><p>genotyped as either homozygous or heterozygous for the alternate allele, with at least 90% of sequencing reads supporting the alternate allele</p></li>
<li><p>supported by at least 10 sequencing reads</p></li>
<li><p>Phred-scaled genotype quality of at least 20</p></li>
<li><p>must not overlap regions of the genome annotated as segmental duplications or simple repeats in GRCm38/mm10</p></li>
<li><p>must occur on a parental haplotype that was inherited by at least one other BXD at the same locus; these other BXDs must be homozygous for the reference allele at the variant site</p></li>
</ul>
<h3 id="a-new-approach-to-discover-germline-mutator-alleles">A new approach to discover germline mutator alleles</h3>
<p>Using the existing catalog of <em>de novo</em> germline mutations in the BXDs, we developed a new approach to discover loci that affect the germline <em>de novo</em> mutation spectrum in biparental RILs (Figure <a href="#fig:toy-diagram">1</a>).</p>
<div id="fig:toy-diagram" class="fignos">
<figure>
<img src="images/Figure%201.png" style="width:7in" alt="Figure 1: Overview of inter-haplotype distance method for discovering mutator alleles. a) A population of four haplotypes has been genotyped at three informative markers; each haplotype also harbors private de novo germline mutations. At each informative marker, we compute an aggregate de novo germline mutation spectrum in the haplotypes that carry either parental allele, and calculate the cosine distance between the two aggregate spectra. b) We repeat the process outlined in a) for every informative marker along the genome. c) To assess the significance of any cosine distance peaks in b), we perform a permutation test by shuffling the labels associated with each haplotype’s mutation data and running a genome-wide distance scan. In each of N permutations, we record the maximum distance encountered at any locus in the distance scan. Finally, we calculate the 1 - p percentile of that maximum distance distribution to obtain a genome-wide cosine distance threshold at the specified value of p." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Overview of inter-haplotype distance method for discovering mutator alleles.</strong> <strong>a)</strong> A population of four haplotypes has been genotyped at three informative markers; each haplotype also harbors private <em>de novo</em> germline mutations. At each informative marker, we compute an aggregate <em>de novo</em> germline mutation spectrum in the haplotypes that carry either parental allele, and calculate the cosine distance between the two aggregate spectra. <strong>b)</strong> We repeat the process outlined in a) for every informative marker along the genome. <strong>c)</strong> To assess the significance of any cosine distance peaks in b), we perform a permutation test by shuffling the labels associated with each haplotype’s mutation data and running a genome-wide distance scan. In each of <span class="math inline">\(N\)</span> permutations, we record the maximum distance encountered at any locus in the distance scan. Finally, we calculate the <span class="math inline">\(1 - p\)</span> percentile of that maximum distance distribution to obtain a genome-wide cosine distance threshold at the specified value of <span class="math inline">\(p\)</span>.</figcaption>
</figure>
</div>
<p>We assume that a collection of haplotypes has been genotyped at informative markers, and that <em>de novo</em> germline mutations have been identified on each haplotype.</p>
<p>At each informative marker, we divide haplotypes into two groups based on the parental allele that they inherited. We then compute a <span class="math inline">\(k\)</span>-mer mutation spectrum using the aggregate mutation counts in each haplotype group. The <span class="math inline">\(k\)</span>-mer mutation spectrum contains the frequency of every possible <span class="math inline">\(k\)</span>-mer mutation type in a collection of mutations, and can be represented as a vector of size <span class="math inline">\(6 \times 4^{k - 1}\)</span> after collapsing by strand complement. For example, the <span class="math inline">\(1\)</span>-mer mutation spectrum is 6-element vector that contains the frequencies of C&gt;T, C&gt;G, C&gt;A, A&gt;G, A&gt;T, and A&gt;C mutations.</p>
<p>At each marker, we then calculate the cosine distance between the aggregate mutation spectra of haplotypes with either parental allele. The cosine distance between two vectors <span class="math inline">\(\mathbf{A}\)</span> and <span class="math inline">\(\mathbf{B}\)</span> is defined as</p>
<p><span class="math display">\[D_C = 1 - \frac{\mathbf{A} \cdot \mathbf{B}}{||\mathbf{A}|| \ ||\mathbf{B}||}\]</span></p>
<p>where <span class="math inline">\(||\mathbf{A}||\)</span> and <span class="math inline">\(||\mathbf{B}||\)</span> are the <span class="math inline">\(L^2\)</span> (or Euclidean) norms of <span class="math inline">\(\mathbf{A}\)</span> and <span class="math inline">\(\mathbf{B}\)</span>, respectively. The cosine distance metric has a number of favorable properties for comparing mutation spectra. Since cosine distance does not take the magnitude of vectors into account, it can be used to compare two spectra with unequal total mutation counts (even if those total counts are relatively small). Additionally, by calculating the cosine distance between mutation <em>spectra</em>, we avoid the need to perform separate comparisons of mutation counts at each individual <span class="math inline">\(k\)</span>-mer mutation type.</p>
<p>Similar to existing methods for quantitative trait locus mapping <span class="citation" data-cites="PMID:30591514">[<a href="#ref-PMID:30591514" role="doc-biblioref"><strong>PMID:30591514?</strong></a>]</span>, we use permutation tests to establish genome-wide cosine distance thresholds. In each of <span class="math inline">\(N\)</span> permutation trials, we randomly shuffle the per-haplotype mutation data such that haplotype labels no longer correspond to the correct mutation counts. Using the shuffled mutation data, we perform a genome-wide distance scan as described above, and record the maximum distance observed at any locus. After <span class="math inline">\(N\)</span> permutations (usually 10,000), we compute the <span class="math inline">\(1 - p\)</span> percentile of the maximum distance distribution, and use that percentile value as a genome-wide significance threshold (for example, at <span class="math inline">\(p = 0.05\)</span>).</p>
<p>The inter-haplotype distance method was implemented in Python, and relies heavily on the following Python libraries: <code>numpy</code>, <code>pandas</code>, <code>matplotlib</code>, <code>scikit-learn</code>, <code>pandera</code>, <code>seaborn</code>, and <code>numba</code> <span class="citation" data-cites="Bp0bGkXP 8YNFNGgR 1026Gxdsi lF0Fr9LL 34sZyTFd fsZYWnTE MBYIZnrX">[<a href="#ref-Bp0bGkXP" role="doc-biblioref">2</a>,<a href="#ref-8YNFNGgR" role="doc-biblioref">3</a>,<a href="#ref-1026Gxdsi" role="doc-biblioref">4</a>,<a href="#ref-lF0Fr9LL" role="doc-biblioref">5</a>,<a href="#ref-34sZyTFd" role="doc-biblioref">6</a>,<a href="#ref-fsZYWnTE" role="doc-biblioref">7</a>,<a href="#ref-MBYIZnrX" role="doc-biblioref">8</a>]</span>.</p>
<p>Additional documentation is available on GitHub <span class="citation" data-cites="14uVnBReQ">[<a href="#ref-14uVnBReQ" role="doc-biblioref">9</a>]</span>, along with a reproducible Snakemake <span class="citation" data-cites="alBakI22">[<a href="#ref-alBakI22" role="doc-biblioref">10</a>]</span> workflow for running the method from start to finish using the BXDs (including downloading the mutation data, downloading genotypes, and running a genome-wide distance scan).</p>
<h3 id="simulations-to-assess-the-power-of-the-inter-haplotype-distance-approach">Simulations to assess the power of the inter-haplotype distance approach</h3>
<p>We performed a series of simple simulations to estimate our power to detect alleles that affect the germline mutation spectrum in biparental RILs using the inter-haplotype distance method.</p>
<p>First, we simulate the <span class="math inline">\(k\)</span>-mer mutation spectrum in a population of <span class="math inline">\(h\)</span> haplotypes. We assume that exactly <span class="math inline">\(\frac{h}{2}\)</span> of the haplotypes are under the effects of a mutator allele that increases the mutation rate of a particular mutation type(s) by an effect size <span class="math inline">\(e\)</span>. We simulate <span class="math inline">\(m\)</span> mutations on each haplotype as follows:</p>
<p>We first define a vector of <span class="math inline">\(1\)</span>-mer mutation probabilities:</p>
<p><span class="math display">\[P = \left( 0.4, \ 0.1, \ 0.075, \ 0.075, \ 0.075, \ 0.275 \right)\]</span></p>
<p>These probabilities sum to 1 and correspond to the expected frequencies of C&gt;T, C&gt;A, C&gt;G, A&gt;T, A&gt;C, and A&gt;G <em>de novo</em> germline mutations in mice, respectively <span class="citation" data-cites="PMID:31492841">[<a href="#ref-PMID:31492841" role="doc-biblioref"><strong>PMID:31492841?</strong></a>]</span>.</p>
<p>If we are simulating the <span class="math inline">\(3\)</span>-mer mutation spectrum, we modify the vector of mutation probabilities <span class="math inline">\(P\)</span> to be length 96, and assign every 3-mer mutation type a value of <span class="math inline">\(\frac{P_c}{16}\)</span>, where <span class="math inline">\(P_c\)</span> is the probability of the “central” mutation type associated with the 3-mer mutation type. In other words, each of the 16 possible N<ins>C</ins>N&gt;N<ins>T</ins>N 3-mer mutation types would be assigned a mutation probability of <span class="math inline">\(\frac{P_c}{16} = \frac{0.4}{16} = 0.025\)</span>.</p>
<p>To simulate the mutation spectrum on the <em>wild-type</em> haplotypes, we define a matrix <span class="math inline">\(C\)</span> of size <span class="math inline">\((\frac{h}{2}, n)\)</span>, where <span class="math inline">\(n = 6 \times 4^{k - 1}\)</span> (i.e., the number of <span class="math inline">\(k\)</span>-mer mutation types being simulated). First, we generate a vector of lambda values by scaling the mutation probabilities by the number of mutations we wish to simulate:</p>
<p><span class="math display">\[\lambda = Pm\]</span></p>
<p>Then, we populate the matrix <span class="math inline">\(C\)</span> by taking a single Poisson draw from the vector of <span class="math inline">\(\lambda\)</span> values for each mutation type on each haplotype. Thus, for every row <span class="math inline">\(i\)</span> in the matrix (i.e., for every haplotype), we perform the following for the mutation type <span class="math inline">\(j\)</span>:</p>
<p><span class="math display">\[C_{i, j} = \mathrm {Pois}(\lambda_{j})\]</span></p>
<p>To simulate the mutation spectrum on the <span class="math inline">\(\frac{h}{2}\)</span> <em>mutator</em> haplotypes, we define a new matrix <span class="math inline">\(C^{\prime}\)</span> of size <span class="math inline">\((\frac{h}{2}, n)\)</span> as defined above. We then multiply the lambda value of a particular mutation type (or multiple mutation types) by the mutator effect size <span class="math inline">\(e\)</span> to obtain <span class="math inline">\(\lambda^{\prime}\)</span>. Then, for every row <span class="math inline">\(i\)</span> in the matrix:</p>
<p><span class="math display">\[C^{\prime}_{i, j} = \mathrm {Pois}(\lambda^{\prime}_{j})\]</span></p>
<p>When <span class="math inline">\(k = 1\)</span>, we only augment the effect size of one mutation type at a time, but when <span class="math inline">\(k = 3\)</span>, we augment a fraction (25%, 50%, or 100%) of the <span class="math inline">\(3\)</span>-mer mutation types associated with a single “base” mutation type.</p>
<p>After generating mutator and wild-type haplotypes, we compute the aggregate mutation spectrum in either group by summing the columns of <span class="math inline">\(C\)</span> and <span class="math inline">\(C^{\prime}\)</span>. We then calculate the cosine distance between the two aggregate spectra, which we call the “focal” distance <span class="math inline">\(D_f\)</span>. To determine whether <span class="math inline">\(D_f\)</span> is greater than what we’d expect by chance, we perform a permutation test.</p>
<p>First, we concatenate the matrices of wild-type and mutator haplotype spectra:</p>
<p><span class="math display">\[A = \begin{bmatrix}
C \\
C^{\prime}
\end{bmatrix}\]</span></p>
<p>Then, in each of <span class="math inline">\(N = 1,000\)</span> trials, we randomly permute the rows of <span class="math inline">\(A\)</span>. In every permutation, we consider the row indices from <span class="math inline">\(\left[0, \frac{h}{2} \right)\)</span> to correspond to the wild-type haplotypes, and the row indices from <span class="math inline">\(\left[ \frac{h}{2}, h \right)\)</span> to correspond to the mutator haplotypes. We then compute the cosine distance between the aggregate spectra of the wild-type and mutator haplotypes. If fewer than 5% of the <span class="math inline">\(N\)</span> permutations produces a cosine distance greater than or equal to <span class="math inline">\(D_f\)</span>, we say that the approach succesfully identified the mutator allele. For every combination of simulation parameters (<span class="math inline">\(h\)</span>, <span class="math inline">\(m\)</span>, <span class="math inline">\(e\)</span>, and so on) we perform 100 trials and record the number of trials in which we successfully identify the mutator allele.</p>
<h3 id="applying-the-inter-haplotype-distance-method-to-the-bxds">Applying the inter-haplotype distance method to the BXDs</h3>
<p>We downloaded previously-generated BXD <em>de novo</em> germline mutation data from the GitHub repository associated with our previous manuscript, which was also archived at Zenodo <span class="citation" data-cites="TYYBVYN3 z6H1lPG4 PMID:35545679">[<a href="#ref-TYYBVYN3" role="doc-biblioref">11</a>,<a href="#ref-z6H1lPG4" role="doc-biblioref">12</a>,<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>, and downloaded a CSV file of BXD genotypes at 7,320 informative markers from GeneNetwork <span class="citation" data-cites="t8fRZlGg PMID:27933521">[<a href="#ref-t8fRZlGg" role="doc-biblioref">13</a>,<a href="#ref-PMID:27933521" role="doc-biblioref"><strong>PMID:27933521?</strong></a>]</span>. We also downloaded relevant metadata about each BXD RIL from the manuscript describing the updated BXD resource <span class="citation" data-cites="PMID:33472028">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>.</p>
<p>As in our previous manuscript <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>, we included mutation data from a subset of the 152 BXDs in our inter-haplotype distance scans. We removed any BXDs that had been inbred for fewer than 20 generations, as it takes approximately 20 generations of strict brother-sister mating for an RIL genome to become ~99.8% homozygous <span class="citation" data-cites="8Z2SLpxu">[<a href="#ref-8Z2SLpxu" role="doc-biblioref">14</a>]</span>. As a result, any potential mutator allele would almost certainly be either fixed or lost after 20 generations. If fixed, the allele would remain linked to any excess mutations it causes for the duration of subsequent inbreeding, and its effects would be detectable using our methods. We also removed the BXD68 RIL from our genome-wide scans, since we previously discovered a hyper-mutator phenotype in that strain; the C&gt;A germline mutation rate in BXD68 is over 5 times the population mean, likely due to a private deleterious nonsynonymous mutation in <em>Mutyh</em> <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>. In total, we included 94 BXD RILs in our genome-wide scans.</p>
<p>We used Snakemake <span class="citation" data-cites="1FskMy3ZE">[<a href="#ref-1FskMy3ZE" role="doc-biblioref">15</a>]</span> to write a reproducible workflow for running the inter-haplotype distance method on the BXD dataset, which has been deposited in the GitHub repository associated with this manuscript <span class="citation" data-cites="14uVnBReQ">[<a href="#ref-14uVnBReQ" role="doc-biblioref">9</a>]</span>.</p>
<h3 id="identifying-candidate-mutator-alleles-overlapping-the-chromosome-6-peak">Identifying candidate mutator alleles overlapping the chromosome 6 peak</h3>
<p>We investigated the region implicated by our inter-haplotype distance approach on chromosome 6 by subsetting the joint-genotyped BXD VCF file (European Nucleotide Archive accession <strong>PRJEB45429</strong> <span class="citation" data-cites="V7hHut0M">[<a href="#ref-V7hHut0M" role="doc-biblioref">16</a>]</span>) using <code>bcftools</code> <span class="citation" data-cites="PMID:33590861">[<a href="#ref-PMID:33590861" role="doc-biblioref"><strong>PMID:33590861?</strong></a>]</span>. To predict the functional impacts of both single-nucleotide variants and indels on splicing, protein structure, etc., we annotated variants in the BXD VCF using the following <code>snpEff</code> <span class="citation" data-cites="PMID:22728672">[<a href="#ref-PMID:22728672" role="doc-biblioref"><strong>PMID:22728672?</strong></a>]</span> command:</p>
<pre><code> java -Xmx16g -jar /path/to/snpeff/jarfile GRCm38.75 /path/to/bxd/vcf &gt; /path/to/uncompressed/output/vcf</code></pre>
<p>and used <code>cyvcf2</code> <span class="citation" data-cites="PMID:28165109">[<a href="#ref-PMID:28165109" role="doc-biblioref"><strong>PMID:28165109?</strong></a>]</span> to iterate over the annotated VCF file in order to identify nonsynonymous fixed differences between the parental C57BL/6J and DBA/2J strains.</p>
<h3 id="comparing-mutation-spectra-between-mouse-genomes-project-strains">Comparing mutation spectra between Mouse Genomes Project strains</h3>
<p>We downloaded mutation data from a previously published analysis <span class="citation" data-cites="PMID:30753674">[<a href="#ref-PMID:30753674" role="doc-biblioref"><strong>PMID:30753674?</strong></a>]</span> (Supplementary File 1, Excel Table S3) that identified strain-private mutations in 29 strains that were originally whole-genome sequenced as part of the Sanger Mouse Genomes (MGP) project <span class="citation" data-cites="PMID:21921910">[<a href="#ref-PMID:21921910" role="doc-biblioref"><strong>PMID:21921910?</strong></a>]</span>. When comparing counts of each mutation type between MGP strains that harbored either <em>D</em> or <em>B</em> alleles at the chromosome 4 or chromosome 6 mutator loci, we adjusted mutation counts by the number of callable A, T, C, or G nucleotides in each strain as described previously <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.</p>
<h3 id="querying-genenetwork-for-evidence-of-eqtls-at-the-mutator-locus">Querying GeneNetwork for evidence of eQTLs at the mutator locus</h3>
<p>We used the online GeneNetwork resource <span class="citation" data-cites="PMID:27933521">[<a href="#ref-PMID:27933521" role="doc-biblioref"><strong>PMID:27933521?</strong></a>]</span>, which contains array- and RNA-seq-derived expression measurements in a wide variety of tissues from numerous datasets, to find <em>cis</em>-eQTLs for the DNA repair genes we implicated under the cosine distance peak on chromosome 6. On the GeneNetwork homepage (genenetwork.org), we selected the “BXD Family” <strong>Group</strong> and used the <strong>Type</strong> dropdown menu to select each of the specific expression datasets described in Table <a href="#tbl:eqtl-results">3</a>. In the <strong>Get Any</strong> text box, we then entered the specified gene name and clicked <strong>Search</strong>. After selecting the appropriate data record on the next page, we used the <strong>Mapping Tools</strong> dropdown to run Haley-Knott regression <span class="citation" data-cites="PMID:16718932">[<a href="#ref-PMID:16718932" role="doc-biblioref"><strong>PMID:16718932?</strong></a>]</span> with the following parameters: WGS-based marker genotypes, 1,000 permutations for LOD threshold calculations, and controlling for BXD genotypes at the <code>rsm10000007390</code> marker.</p>
<p>The exact names of the expression datasets we used for each tissue are shown in Table <a href="#tbl:eqtl-provenance">1</a> below:</p>
<div id="tbl:eqtl-provenance" class="tablenos">
<table id="tbl:eqtl-provenance">
<caption><span>Table 1:</span> Names of gene expression datasets used for each tissue type on GeneNetwork </caption>
<colgroup>
<col style="width: 33%" />
<col style="width: 33%" />
<col style="width: 33%" />
</colgroup>
<thead>
<tr class="header">
<th>Tissue name</th>
<th>Complete name of GeneNetwork expression data</th>
<th>GeneNetwork trait ID</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Kidney</td>
<td><code>Mouse kidney M430v2 Sex Balanced (Aug06) RMA</code></td>
<td><code>1448815_at</code></td>
</tr>
<tr class="even">
<td>Gastrointestinal</td>
<td><code>UTHSC Mouse BXD Gastrointestinal Affy MoGene 1.0 ST Gene Level (Apr14) RMA</code></td>
<td><code>10540639</code></td>
</tr>
<tr class="odd">
<td>Hematopoetic stem cells</td>
<td><code>UMCG Stem Cells ILM6v1.1 (Apr09) transformed</code></td>
<td><code>ILM1940279</code></td>
</tr>
<tr class="even">
<td>Hematopoetic progenitor cells</td>
<td><code>UMCG Progenitor Cells ILM6v1.1 (Apr09) transformed</code></td>
<td><code>ILM1940279</code></td>
</tr>
<tr class="odd">
<td>Spleen</td>
<td><code>UTHSC Affy MoGene 1.0 ST Spleen (Dec10) RMA</code></td>
<td><code>10540639</code></td>
</tr>
<tr class="even">
<td>Liver</td>
<td><code>UTHSC BXD Liver RNA-Seq Avg (Oct19) TPM Log2</code></td>
<td><code>ENSMUST00000032406</code></td>
</tr>
<tr class="odd">
<td>Heart</td>
<td><code>NHLBI BXD All Ages Heart RNA-Seq (Nov20) TMP Log2 **</code></td>
<td><code>ENSMUSG00000030271</code></td>
</tr>
<tr class="even">
<td>Eye</td>
<td><code>UTHSC BXD All Ages Eye RNA-Seq (Nov20) TPM Log2 **</code></td>
<td><code>ENSMUSG00000030271</code></td>
</tr>
</tbody>
</table>
</div>
<h3 id="calculating-the-frequencies-of-candidate-mutator-alleles-in-wild-mice">Calculating the frequencies of candidate mutator alleles in wild mice</h3>
<p>For each of the three candidate nonsynonymous mutations we identified under the cosine distance peak on chromosome 6 (Table <a href="#tbl:missense-diffs">2</a>), we queried a VCF file containing genome-wide variation in 67 wild-derived mice from four species of <em>Mus</em> <span class="citation" data-cites="PMID:27622383">[<a href="#ref-PMID:27622383" role="doc-biblioref"><strong>PMID:27622383?</strong></a>]</span>. We calculated the allele frequency of each nonsynonymous mutation in each of the four species or subspecies (<em>Mus musculus domesticus</em>, <em>Mus musculus musculus</em>, <em>Mus musculus castaneus</em>, and <em>Mus spretus</em>), including genotypes that met the following criteria:</p>
<ul>
<li><p>supported by at least 10 sequencing reads</p></li>
<li><p>Phred-scaled genotype quality of at least 20</p></li>
</ul>
<h2 id="results">Results</h2>
<h3 id="benchmarking-the-inter-haplotype-distance-method-using-simulations">Benchmarking the inter-haplotype distance method using simulations</h3>
<p>We first tested the inter-haplotype cosine distance approach using simulated data (Materials and Methods). We find that the method’s power is mostly limited by the initial mutation rate of the <span class="math inline">\(k\)</span>-mer mutation type affected by the mutator allele and the total number of <em>de novo</em> germline mutations in the dataset (that is, the product of the number of haplotypes and the mean number of mutations per haplotype) (Figure <a href="#fig:simulations">2</a>). For example, given 50 haplotypes with an average of 500 <em>de novo</em> germline mutations each, our method has nearly 90% power detect a mutator allele that increases the C&gt;T <em>de novo</em> mutation rate by 10%. However, the method only has about 10% power to detect a mutator of identical effect size that affects the C&gt;G mutation rate, since C&gt;G mutations are expected to make up a much smaller fraction of all <em>de novo</em> germline mutations to begin with. These simulations also demonstrate that our method is well-powered to detect large-effect mutator alleles (e.g., those that increase the mutation rate of a specific <span class="math inline">\(k\)</span>-mer by 50%), even with a relatively small number of mutations per haplotype.</p>
<div id="fig:simulations" class="fignos">
<figure>
<img src="images/sims.png" style="width:7in" alt="Figure 2: Simulations to assess the power of the inter-haplotype distance method. We simulated de novo germline mutations on the specified number of haplotypes, such that 50% of haplotypes were affected by a mutator allele that increased the mutation rate of the specified k-mer by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate). The colors of the lines indicate the number of simulated mutations on each haplotype (before augmenting the mutation rate with a mutator allele). Given a specific combination of parameters, the y-axis denotes the fraction of 100 simulations in which the simulated mutator allele could be detected at a p-value of 0.05. Shaded areas indicate the standard deviation of that fraction." />
<figcaption aria-hidden="true"><span>Figure 2:</span> <strong>Simulations to assess the power of the inter-haplotype distance method.</strong> We simulated <em>de novo</em> germline mutations on the specified number of haplotypes, such that 50% of haplotypes were affected by a mutator allele that increased the mutation rate of the specified <span class="math inline">\(k\)</span>-mer by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate). The colors of the lines indicate the number of simulated mutations on each haplotype (before augmenting the mutation rate with a mutator allele). Given a specific combination of parameters, the y-axis denotes the fraction of 100 simulations in which the simulated mutator allele could be detected at a p-value of 0.05. Shaded areas indicate the standard deviation of that fraction.</figcaption>
</figure>
</div>
<h3 id="re-identifying-the-mutator-allele-on-chromosome-4-in-the-bxds">Re-identifying the mutator allele on chromosome 4 in the BXDs</h3>
<p>We applied our inter-haplotype distance method to 94 BXD RILs (Materials and Methods) with a total of 63,914 <em>de novo</em> germline mutations <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>. Reassuringly, we observed a large peak in cosine distance at a locus on chromosome 4 (Figure <a href="#fig:distance-results">3</a>A; maximum distance of X at marker ID rsYYYYY; position 116.8 Mbp in mm10 coordinates).</p>
<div id="fig:distance-results" class="fignos">
<figure>
<img src="images/Figure%202.png" style="width:7in" alt="Figure 3: Results of inter-haplotype distance scans in the BXD RILs. a) Cosine distances between aggregate de novo mutation spectra on BXD haplotypes (n = 94)with either D or B alleles at approximately 7,500 informative markers. Distance threshold at p = 0.05 was calculated by performing 10,000 permutations of the BXD haplotype mutation data, and is shown as a dotted grey line. One outlier RIL with an extremely high C&gt;A mutation rate (BXD68) was removed prior to running the distance scan. b) Cosine distances between aggregate de novo mutation spectra on BXD haplotypes with either D or B alleles at approximately 7,500 informative markers. Only BXDs with D haplotypes at marker rsYYYY on chromosome 4 were included in the scan. Distance threshold at p = 0.05 was calculated by performing 10,000 permutations of the BXD haplotype mutation data, and is shown as a dotted grey line. c) Fractions of de novo germline mutations in BXDs with either D or B haplotypes at markers rsXXXX and rsYYYY. Total counts of each mutation type were aggregated in groups of BXDs with either of the four possible haplotype combinations, and fractions of each mutation type were calculated in each group separately. Chi-square tests of independence were used to compare counts of individual mutation types between the D-D and D-B groups of BXDs, as well as between the B-D and B-B groups. d) Fractions of de novo germline mutations in Sanger Mouse Genome Project (MGP) strains with either D or B haplotypes at markers rsXXXX and rsYYYY. Total counts of each mutation type were aggregated in groups of MGP strains with either of the four possible haplotype combinations, and fractions of each mutation type were calculated in each group separately. Chi-square tests of independence were used to compare counts of individual mutation types between the D-D and D-B groups of BXDs, as well as between the B-D and B-B groups." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>Results of inter-haplotype distance scans in the BXD RILs.</strong> <strong>a)</strong> Cosine distances between aggregate <em>de novo</em> mutation spectra on BXD haplotypes (n = 94)with either <em>D</em> or <em>B</em> alleles at approximately 7,500 informative markers. Distance threshold at <span class="math inline">\(p = 0.05\)</span> was calculated by performing 10,000 permutations of the BXD haplotype mutation data, and is shown as a dotted grey line. One outlier RIL with an extremely high C&gt;A mutation rate (BXD68) was removed prior to running the distance scan. <strong>b)</strong> Cosine distances between aggregate <em>de novo</em> mutation spectra on BXD haplotypes with either <em>D</em> or <em>B</em> alleles at approximately 7,500 informative markers. Only BXDs with <em>D</em> haplotypes at marker rsYYYY on chromosome 4 were included in the scan. Distance threshold at <span class="math inline">\(p = 0.05\)</span> was calculated by performing 10,000 permutations of the BXD haplotype mutation data, and is shown as a dotted grey line. <strong>c)</strong> Fractions of <em>de novo</em> germline mutations in BXDs with either <em>D</em> or <em>B</em> haplotypes at markers rsXXXX and rsYYYY. Total counts of each mutation type were aggregated in groups of BXDs with either of the four possible haplotype combinations, and fractions of each mutation type were calculated in each group separately. Chi-square tests of independence were used to compare counts of individual mutation types between the D-D and D-B groups of BXDs, as well as between the B-D and B-B groups. <strong>d)</strong> Fractions of <em>de novo</em> germline mutations in Sanger Mouse Genome Project (MGP) strains with either <em>D</em> or <em>B</em> haplotypes at markers rsXXXX and rsYYYY. Total counts of each mutation type were aggregated in groups of MGP strains with either of the four possible haplotype combinations, and fractions of each mutation type were calculated in each group separately. Chi-square tests of independence were used to compare counts of individual mutation types between the D-D and D-B groups of BXDs, as well as between the B-D and B-B groups.</figcaption>
</figure>
</div>
<p>In a previous analysis, we used quantitative trait locus (QTL) mapping to identify a nearly identical locus on chromosome 4 that was significantly associated with the C&gt;A germline mutation rate in the BXDs <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>. This locus overlaps 21 protein-coding genes that are annotated by the Gene Ontology as being involved in “DNA repair,” but only one of these genes contains non-synonymous differences between the two parental strains: <em>Mutyh</em>. <em>Mutyh</em> encodes a protein involved in the base-excision repair of 8-oxoguanine (8-oxoG), a DNA lesion caused by oxidative damage, and prevents the accumulation of C&gt;A mutations <span class="citation" data-cites="PMID:28551381 PMID:28127763 PMID:17581577">[<a href="#ref-PMID:28551381" role="doc-biblioref"><strong>PMID:28551381?</strong></a>,<a href="#ref-PMID:28127763" role="doc-biblioref"><strong>PMID:28127763?</strong></a>,<a href="#ref-PMID:17581577" role="doc-biblioref"><strong>PMID:17581577?</strong></a>]</span>. C&gt;A germline mutation rates are nearly 50% higher in BXDs that inherited <em>D</em> haplotypes at marker ID rsYYYY than in those that inherited <em>B</em> haplotypes <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.</p>
<h3 id="an-additional-germline-mutator-allele-on-chromosome-6">An additional germline mutator allele on chromosome 6</h3>
<p>After confirming that the inter-haplotype distance method could recover the mutator locus overlapping <em>Mutyh</em>, we asked if our approach could identify additional mutator loci in the BXD. To account for the effects of the large-effect C&gt;A germline mutator locus near <em>Mutyh</em>, we divided the BXD RILs into those with either <em>D</em> (n = X) or <em>B</em> (n = Y) genotypes at rsYYYYY (the marker at which we observed the highest inter-haplotype cosine distance on chromosome 4), and ran a genome-wide distance scan using each group separately (Figure <a href="#fig:distance-results">3</a>B.</p>
<p>Using only the BXDs with <em>B</em> genotypes at the <em>Mutyh</em> mutator locus, we did not observe any genome-wide significant peaks. But using the BXDs with <em>D</em> genotypes at the same locus, we identified a cosine distance peak on chromosome 6 (Figure <a href="#fig:distance-results">3</a>B; maximum distance of X at marker rsYYYYY; position 133.2 Mbp in mm10 coordinates). We queried the region underneath this peak and discovered two genes annotated with the Gene Ontology term “DNA repair”: <em>Setmar</em>, a protein with histone methyltransferase and transposase activity, and remarkably, <em>Ogg1</em>. The latter encodes a key member of the base-exision repair response to oxidative DNA damage, a pathway that also includes <em>Mutyh</em> and a related gene, <em>Mth1</em>. Both <em>Setmar</em> and <em>Ogg1</em> harbor fixed nonsynonymous differences between the C57BL/6J and DBA/2J parental strains (Table <a href="#tbl:missense-diffs">2</a>).</p>
<div id="tbl:missense-diffs" class="tablenos">
<table id="tbl:missense-diffs">
<caption><span>Table 2:</span> Summary of nonsynonymous differences between C57BL/6J and DBA/2J haplotypes in DNA repair genes near the mutator locus on chromosome 6. </caption>
<thead>
<tr class="header">
<th>Gene name</th>
<th>Amino acid change</th>
<th>Position in GRCm38/mm10 coordinates</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><em>Ogg1</em></td>
<td>p.Ala95Thr</td>
<td>chr6:</td>
</tr>
<tr class="even">
<td><em>Setmar</em></td>
<td>p.Leu103Phe</td>
<td>chr6:</td>
</tr>
<tr class="odd">
<td><em>Setmar</em></td>
<td>p.Ser273Arg</td>
<td>chr6:</td>
</tr>
</tbody>
</table>
</div>
<p>We also considered the possibility that expression quantitative trait loci (eQTLs), rather than nonsynonymous mutations, were responsible for the C&gt;A mutator phenotype linked to the locus on chromosome 6. Using GeneNetwork <span class="citation" data-cites="PMID:27933521">[<a href="#ref-PMID:27933521" role="doc-biblioref"><strong>PMID:27933521?</strong></a>]</span>, we mapped cis-eQTLs for <em>Ogg1</em> in a number of tissues, including hematopoetic stem cells, kidney, and spleen. BXD genotypes at the cosine distance peak on chromosome 6 were significantly associated with <em>Ogg1</em> expression in many tissues, and <em>D</em> genotypes were nearly always associated with decreased gene expression (Table <a href="#tbl:eqtl-results">3</a>). We also queried a previously published collection of eQTLs derived from Diversity Outbred (DO) mouse embryonic stem cell (mESCs) expression data <span class="citation" data-cites="1F8NSczsP">[<a href="#ref-1F8NSczsP" role="doc-biblioref">17</a>]</span>, but did not find any significant eQTLs for <em>Ogg1</em>.</p>
<div id="tbl:eqtl-results" class="tablenos">
<table id="tbl:eqtl-results" style="width:100%;">
<caption><span>Table 3:</span> Presence of absence of cis-eQTLs for <em>Ogg1</em> in various tissues identified using GeneNetwork. </caption>
<colgroup>
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
</colgroup>
<thead>
<tr class="header">
<th>Tissue name</th>
<th># BXDs with expression data</th>
<th>Top significant marker</th>
<th>LRS at top significant marker</th>
<th>Significant LRS threshold</th>
<th>Additive effect of D allele on expression</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Kidney</td>
<td>53</td>
<td><code>rsm10000004188</code></td>
<td><strong>56.41</strong></td>
<td>17.80</td>
<td>-0.18</td>
</tr>
<tr class="even">
<td>Gastrointestinal</td>
<td>46</td>
<td><code>rsm10000003441</code></td>
<td><strong>30.10</strong></td>
<td>16.21</td>
<td>-0.081</td>
</tr>
<tr class="odd">
<td>Hematopoetic stem cells</td>
<td>22</td>
<td>-</td>
<td>-</td>
<td>16.43</td>
<td>-</td>
</tr>
<tr class="even">
<td>Hematopoetic progenitor cells</td>
<td>23</td>
<td>-</td>
<td>-</td>
<td>18.27</td>
<td>-</td>
</tr>
<tr class="odd">
<td>Spleen</td>
<td>79</td>
<td><code>rsm10000003418</code></td>
<td><strong>17.72</strong></td>
<td>17.49</td>
<td>-0.056</td>
</tr>
<tr class="even">
<td>Liver</td>
<td>50</td>
<td><code>rsm10000004188</code></td>
<td><strong>52.27</strong></td>
<td>18.81</td>
<td>-0.155</td>
</tr>
<tr class="odd">
<td>Heart</td>
<td>73</td>
<td>-</td>
<td>-</td>
<td>16.12</td>
<td>-</td>
</tr>
<tr class="even">
<td>Eye</td>
<td>87</td>
<td><code>rsm10000004194</code></td>
<td><strong>22.66</strong></td>
<td>17.20</td>
<td>0.087</td>
</tr>
</tbody>
</table>
</div>
<h3 id="evidence-of-epistasis-between-germline-mutator-alleles">Evidence of epistasis between germline mutator alleles</h3>
<p>We next compared the mutation spectra of BXDs with either <em>B</em> or <em>D</em> genotypes at the mutator loci on chromosomes 4 and 6. We observed that C&gt;A germline mutation fractions in BXDs with <em>D</em> alleles at both mutator loci were significantly higher than C&gt;A fractions in BXDs with <em>D</em> alleles at either locus alone (Figure <a href="#fig:distance-results">3</a>C). However, compared to BXDs with <em>B</em> alleles at the chromosome 6 mutator locus, those with <em>D</em> alleles did not exhibit significantly higher C&gt;A mutation fractions, indicating that the effects of the chromosome 6 mutator locus depend on the presence of a <em>D</em> allele at the chromosome 4 locus (Figure <a href="#fig:distance-results">3</a>C).</p>
<p>To explore the effects of the two mutator loci in other inbred laboratory mice, we also compared the germline mutation spectra of Sanger Mouse Genomes Project (MGP) strains. Dumont <span class="citation" data-cites="PMID:30753674">[<a href="#ref-PMID:30753674" role="doc-biblioref"><strong>PMID:30753674?</strong></a>]</span> previously identified private germline mutations in 29 inbred laboratory strains; these private variants likely represent recent <em>de novo</em> germline mutations. Only two of the MGP strains possess <em>D</em> genotypes at both the chromosome 4 and chromosome 6 mutator loci: DBA/1J and DBA/2J. Compared to strains with <em>B</em> alleles at both mutator loci, those with <em>D</em> alleles at both mutator loci exhibit significantly higher C&gt;A germline mutation fractions (<span class="math inline">\(p = 3.5\mathrm{e}{-8}\)</span>, Figure <a href="#fig:distance-results">3</a>D). MGP strains with <em>D</em> alleles at both mutator loci appear to have higher C&gt;A mutation fractions than those with <em>D</em> alleles at either locus alone (Figure <a href="#fig:distance-results">3</a>D), but this difference is not significant (<span class="math inline">\(p = 0.16\)</span>). Therefore, given the smaller number of MGP strains with <em>de novo</em> germline mutation data, we are unable to confirm the signal of epistasis observed in the BXDs.</p>
<h3 id="the-candidate-mutator-alleles-are-present-in-wild-mice">The candidate mutator alleles are present in wild mice</h3>
<p>To determine whether the candidate mutator alleles on chromosome 6 were segregating in natural populations of mice, we queried previously published sequencing data generated from 67 wild-derived mice <span class="citation" data-cites="PMID:27622383">[<a href="#ref-PMID:27622383" role="doc-biblioref"><strong>PMID:27622383?</strong></a>]</span>. These data include three subspecies of <em>Mus musculus</em>, as well as the outgroup <em>Mus spretus</em>. We found that every <em>D</em> allele in <em>Setmar</em> or <em>Ogg1</em> was segregating in at least one population of <em>Mus</em> (Figure <a href="#fig:wild-afs">4</a>). Notably, the nonsynonymous p.Ala95Thr mutation was present at approximately 25% allele frequency in <em>Mus musculus domesticus</em>, and fixed in all other subspecies.</p>
<div id="fig:wild-afs" class="fignos">
<figure>
<img src="images/Figure%203.png" style="width:7in" alt="Figure 4: Frequency of candidate mutator alleles in wild mice. We queried a VCF file containing variant calls from 67 wild-derived Mus samples for each of the three fixed nonsynonymous differences between C57BL/6J and DBA/2J that were observed in DNA repair genes near the mutator locus on chromosome 6. The species abbreviations are: Ms: Mus spretus; Mmd: Mus musculus domesticus; Mmc: Mus musculus castaneus; Mmm: Mus musculus musculus" />
<figcaption aria-hidden="true"><span>Figure 4:</span> <strong>Frequency of candidate mutator alleles in wild mice.</strong> We queried a VCF file containing variant calls from 67 wild-derived <em>Mus</em> samples for each of the three fixed nonsynonymous differences between C57BL/6J and DBA/2J that were observed in DNA repair genes near the mutator locus on chromosome 6. The species abbreviations are: Ms: <em>Mus spretus</em>; Mmd: <em>Mus musculus domesticus</em>; Mmc: <em>Mus musculus castaneus</em>; Mmm: <em>Mus musculus musculus</em></figcaption>
</figure>
</div>
<h2 id="discussion">Discussion</h2>
<h3 id="using-germline-mutation-spectra-to-identify-mutator-alleles">Using germline mutation spectra to identify mutator alleles</h3>
<p>Germline mutation spectra are a rich source of information about the demographic history of populations, as well as the activity of both exogenous and endogenous sources of mutation throughout time. For example, by analyzing the <span class="math inline">\(3\)</span>-mer mutation spectrum in a collection of human genomes, Harris and Pritchard <span class="citation" data-cites="PMID:28440220">[<a href="#ref-PMID:28440220" role="doc-biblioref"><strong>PMID:28440220?</strong></a>]</span> discovered a “pulse” of TCC&gt;TTC mutation activity in European populations that likely occurred between 15,000 and 2,000 years ago, and perhaps began even earlier <span class="citation" data-cites="PMID:34016747">[<a href="#ref-PMID:34016747" role="doc-biblioref"><strong>PMID:34016747?</strong></a>]</span>).</p>
<p>Within somatic tissues, mutation spectra can also be used to uncover the mutational processes active in particular populations of cells <span class="citation" data-cites="PMID:23945592">[<a href="#ref-PMID:23945592" role="doc-biblioref"><strong>PMID:23945592?</strong></a>]</span>. New computational methods have been developed to extract “mutational signatures” from large databases of somatic mutations in cancer <span class="citation" data-cites="ibrvarsm">[<a href="#ref-ibrvarsm" role="doc-biblioref">18</a>]</span>. These signatures, which describe the relative frequency of each <span class="math inline">\(3\)</span>-mer mutation type, can often be precisely attributed to chemotherapeutic agents, exposures to environmental mutagens, or loss-of-function mutations in genes encoding DNA repair or replication proteins <span class="citation" data-cites="PMID:23945592 PMID:31740835 PMID:27811275">[<a href="#ref-PMID:23945592" role="doc-biblioref"><strong>PMID:23945592?</strong></a>,<a href="#ref-PMID:31740835" role="doc-biblioref"><strong>PMID:31740835?</strong></a>,<a href="#ref-PMID:27811275" role="doc-biblioref"><strong>PMID:27811275?</strong></a>]</span>.</p>
<p>Although a germline mutator allele should increase the absolute count of mutations on a linked haplotype, our results demonstrate that its effects can be more easily detectable by examining mutation <em>spectra</em> instead. For example, <em>D</em> alleles at the mutator locus on chromosome 6 augment the C&gt;A mutation rate by a factor of approximately 1.2 (Figure <a href="#fig:distance-results">3</a>). Since C&gt;A mutations comprise approximately 10% of all germline mutations to begin with, <em>D</em> alleles only increase the overall germline mutation rate by about 2%. Given the depth of information that can be encoded in the mutation spectrum, we expect that mutation spectra can be further exploited to discover genetic modifiers of the mutation rate in other study systems, as well.</p>
<h3 id="epistasis-between-germline-mutator-alleles">Epistasis between germline mutator alleles</h3>
<p>Our results also reveal evidence of epistasis between mammalian germline mutator alleles for the first time. BXDs with <em>D</em> alleles at the mutator locus on chromosome 6 only exhibit elevated C&gt;A mutation rates if they also carry <em>D</em> alleles at the previously-identified <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span> mutator locus on chromosome 4. And BXDs with <em>D</em> alleles at both loci have significantly higher C&gt;A germline mutation rates than lines with <em>D</em> alleles at only one mutator locus alone (Figure <a href="#fig:distance-results">3</a>C). This raises the exciting possibility that epistasis between mutator alleles has contributed to the evolution of germline mutation rates and spectra in mammalian genomes.</p>
<p>Importantly, we note that we observed epistasis between germline mutator alleles in an unnatural population; the BXDs were inbred by brother-sister mating in a highly controlled laboratory environment that attenuated the effects of natural selection on all but the most deleterious alleles <span class="citation" data-cites="LF8DCRcP">[<a href="#ref-LF8DCRcP" role="doc-biblioref">1</a>]</span>. Howver, when we queried wild mouse genomes for for the three nonsynonymous DNA repair mutations overlapping the chromosome 6 mutator locus, we found all three at high frequency in <em>Mus musculus domesticus</em>, the strain from which C57BL/6J and DBA/2J derive most of their genomes <span class="citation" data-cites="PMID:17660819">[<a href="#ref-PMID:17660819" role="doc-biblioref"><strong>PMID:17660819?</strong></a>]</span>. Since the <em>D</em> mutator haplotype on chromosome 6 does not appear to increase the C&gt;A germline mutation rate on its own (even in a homozygous state), we hypothesize that similar alleles may be at intermediate or high frequency in other natural populations.</p>
<h3 id="causal-variants-underlying-the-mutator-allele">Causal variants underlying the mutator allele</h3>
<p>We discovered three nonsynonymous fixed differences in DNA repair genes between C57BL/6J and DBA/2J under the C&gt;A mutator locus on chromosome 6 (Table <a href="#tbl:missense-diffs">2</a>). One of these mutations affects <em>Ogg1</em>, a protein-coding gene that participates in the base-excision repair of 8-oxoguanine (8-oxoG), a DNA lesion produced by oxidative DNA damage <span class="citation" data-cites="PMID:17581577">[<a href="#ref-PMID:17581577" role="doc-biblioref"><strong>PMID:17581577?</strong></a>]</span>. Both missense mutations and loss-of-heterozygosity in <em>Ogg1</em> have been associated with initiation and progression of various types of human cancer <span class="citation" data-cites="PMID:22829015 PMID:10987279 PMID:9662341">[<a href="#ref-PMID:22829015" role="doc-biblioref"><strong>PMID:22829015?</strong></a>,<a href="#ref-PMID:10987279" role="doc-biblioref"><strong>PMID:10987279?</strong></a>,<a href="#ref-PMID:9662341" role="doc-biblioref"><strong>PMID:9662341?</strong></a>]</span>. Unrepaired 8-oxoG lesions can also lead to C&gt;A mutations, and copy-number losses of either <em>Ogg1</em> or <em>Mutyh</em> are linked to elevated rates of spontaneous C&gt;A mutation in human neuroblastoma <span class="citation" data-cites="8vP4Q1ja">[<a href="#ref-8vP4Q1ja" role="doc-biblioref">19</a>]</span>. Given these various lines of evidence, we believe that <em>Ogg1</em> is the most likely candidate gene to explain the additional C&gt;A mutator phenotype in the BXDs, but it remains unclear whether the p.Ala95Thr missense mutation (Table <a href="#tbl:missense-diffs">2</a>) is the causal allele. We hypothesized that missense mutations in <em>Mutyh</em> were responsible for the large-effect C&gt;A mutator phenotype we previously observed in the BXDs <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>. However, using high-quality long-read assemblies of inbred laboratory strains, another group recently identified a ~5 kbp mobile element insertion (MEI) within the first intron of <em>Mutyh</em> <span class="citation" data-cites="DAiplkDB">[<a href="#ref-DAiplkDB" role="doc-biblioref">20</a>]</span> that is present on <em>D</em> haplotypes and absent from <em>B</em> haplotypes. The MEI is associated with significantly reduced expression of <em>Mutyh</em> in laboratory strains, and may underlie the previous C&gt;A germline mutator phenotype in the BXDs. In light of this new evidence, we cannot discount the possibility that eQTLs associated with decreased expression of <em>Ogg1</em> are responsible for the C&gt;A mutator phenotype we observed in this study (Table <a href="#tbl:eqtl-results">3</a>).</p>
<h3 id="discovering-mutator-alleles-in-other-systems">Discovering mutator alleles in other systems</h3>
<p>Numerous lines of evidence suggest that mutator alleles contribute to variation in mutation rates and spectra across the tree of life. In two natural isolates of <em>Saccharomyces cerevisiae</em>, nonsynonymous variation in <em>OGG1</em> causes a substantial increase in the C&gt;A <em>de novo</em> mutation rate <span class="citation" data-cites="PMID:34523420">[<a href="#ref-PMID:34523420" role="doc-biblioref"><strong>PMID:34523420?</strong></a>]</span>. A recent analysis suggested that mutator alleles and/or environmental mutagens have shaped mutation rate evolution in human genomes <span class="citation" data-cites="q8rJCwGH">[<a href="#ref-q8rJCwGH" role="doc-biblioref">21</a>]</span>, and more generally, the mutation spectrum has evolved rapidly during great ape evolution, potentially due to the effects of <em>trans</em>-acting mutation rate modifiers <span class="citation" data-cites="PMID:33983415">[<a href="#ref-PMID:33983415" role="doc-biblioref"><strong>PMID:33983415?</strong></a>]</span>. The heritability of paternal <em>de novo</em> mutation counts in the human germline has also been estimated to be between 10 and 20%, demonstrating a contribution of genetic factors to germline mutation rates <span class="citation" data-cites="WwxkZzW7">[<a href="#ref-WwxkZzW7" role="doc-biblioref">22</a>]</span>). However, mutator discovery remains challenging in mammalian genomes.</p>
<p>What conditions must be met in order to detect a germline mutator allele? Presumably, one must have access to many haplotypes, each with a reasonably large number of <em>de novo</em> germline mutations that remain linked to the mutator allele(s) that caused them. Recently, thousands of human pedigrees have been sequenced in an effort to precisely estimate the rate of human <em>de novo</em> germline mutation <span class="citation" data-cites="PMID:31549960 PMID:28959963">[<a href="#ref-PMID:31549960" role="doc-biblioref"><strong>PMID:31549960?</strong></a>,<a href="#ref-PMID:28959963" role="doc-biblioref"><strong>PMID:28959963?</strong></a>]</span>. Selection on germline mutator alleles will likely prevent large-effect mutators from reaching high allele frequencies; however, if multiple mutators are active in a particular population, it becomes much more likely that a subset will be detectable by sequencing human trios <span class="citation" data-cites="PMID:35666194">[<a href="#ref-PMID:35666194" role="doc-biblioref"><strong>PMID:35666194?</strong></a>]</span>. Current estimates of power to detect germline mutators in human pedigrees generally assume that mutators affect all mutation types equally, and that methods for mutator discovery will rely on identifying haplotypes with excess total mutation counts <span class="citation" data-cites="PMID:35666194">[<a href="#ref-PMID:35666194" role="doc-biblioref"><strong>PMID:35666194?</strong></a>]</span>. However, our results in the BXD suggest that germline mutators often exert their effects on a small number of <span class="math inline">\(k\)</span>-mer mutation types, and may be far more amenable to detection by analyzing mutation spectra instead.</p>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-LF8DCRcP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>Spontaneous Mutation Accumulation Studies in Evolutionary Genetics</strong> <div class="csl-block">Daniel L Halligan, Peter D Keightley</div> <em>Annual Review of Ecology, Evolution, and Systematics</em> (2009-12-01) <a href="https://doi.org/dvrjz8">https://doi.org/dvrjz8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1146/annurev.ecolsys.39.110707.173437">10.1146/annurev.ecolsys.39.110707.173437</a></div></div>
</div>
<div id="ref-Bp0bGkXP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>Array programming with NumPy</strong> <div class="csl-block">Charles R Harris, KJarrod Millman, Stéfan J van der Walt, Ralf Gommers, Pauli Virtanen, David Cournapeau, Eric Wieser, Julian Taylor, Sebastian Berg, Nathaniel J Smith, … Travis E Oliphant</div> <em>Nature</em> (2020-09) <a href="https://doi.org/ghbzf2">https://doi.org/ghbzf2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-020-2649-2">https://doi.org/10.1038/s41586-020-2649-2</a></div></div>
</div>
<div id="ref-8YNFNGgR" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>pandas-dev/pandas: Pandas</strong> <div class="csl-block">The pandas development team</div> <em>Zenodo</em> (2023-02-20) <a href="https://doi.org/ggt8bh">https://doi.org/ggt8bh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.5281/zenodo.3509134">https://doi.org/10.5281/zenodo.3509134</a></div></div>
</div>
<div id="ref-1026Gxdsi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>Matplotlib: A 2D Graphics Environment</strong> <div class="csl-block">John D Hunter</div> <em>Computing in Science &amp;amp; Engineering</em> (2007) <a href="https://doi.org/drbjhg">https://doi.org/drbjhg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1109/mcse.2007.55">10.1109/mcse.2007.55</a></div></div>
</div>
<div id="ref-lF0Fr9LL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>Scikit-learn: Machine Learning in Python</strong> <div class="csl-block">Fabian Pedregosa, Gaël Varoquaux, Alexandre Gramfort, Vincent Michel, Bertrand Thirion, Olivier Grisel, Mathieu Blondel, Peter Prettenhofer, Ron Weiss, Vincent Dubourg, … Édouard Duchesnay</div> <em>Journal of Machine Learning Research</em> (2011) <a href="http://jmlr.org/papers/v12/pedregosa11a.html">http://jmlr.org/papers/v12/pedregosa11a.html</a></div>
</div>
<div id="ref-34sZyTFd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>pandera: Statistical Data Validation of Pandas Dataframes</strong> <div class="csl-block">Niels Bantilan</div> <em>Proceedings of the Python in Science Conference</em> (2020) <a href="https://doi.org/grr54q">https://doi.org/grr54q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.25080/majora-342d178e-010">10.25080/majora-342d178e-010</a></div></div>
</div>
<div id="ref-fsZYWnTE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>seaborn: statistical data visualization</strong> <div class="csl-block">Michael L Waskom</div> <em>Journal of Open Source Software</em> (2021-04-06) <a href="https://doi.org/gjqn3g">https://doi.org/gjqn3g</a> <div class="csl-block">DOI: <a href="https://doi.org/10.21105/joss.03021">https://doi.org/10.21105/joss.03021</a></div></div>
</div>
<div id="ref-MBYIZnrX" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>Numba: a LLVM-based Python JIT compiler</strong> <div class="csl-block">Siu Kwan Lam, Antoine Pitrou, Stanley Seibert</div> <em>Proceedings of the Second Workshop on the LLVM Compiler Infrastructure in HPC</em> (2015-11-15) <a href="https://doi.org/gf3nks">https://doi.org/gf3nks</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1145/2833157.2833162">https://doi.org/10.1145/2833157.2833162</a> · ISBN: 9781450340052</div></div>
</div>
<div id="ref-14uVnBReQ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><a href="https://github.com/quinlan-lab/proj-mutator-mapping">https://github.com/quinlan-lab/proj-mutator-mapping</a></div>
</div>
<div id="ref-alBakI22" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>Sustainable data analysis with Snakemake</strong> <div class="csl-block">Felix Mölder, Kim Philipp Jablonski, Brice Letcher, Michael B Hall, Christopher H Tomkins-Tinch, Vanessa Sochat, Jan Forster, Soohyun Lee, Sven O Twardziok, Alexander Kanitz, … Johannes Köster</div> <em>F1000Research</em> (2021-01-18) <a href="https://doi.org/gjjkwv">https://doi.org/gjjkwv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.12688/f1000research.29032.1">https://doi.org/10.12688/f1000research.29032.1</a></div></div>
</div>
<div id="ref-TYYBVYN3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>A natural mutator allele shapes mutation spectrum variation in mice</strong> <div class="csl-block">Tom Sasani</div> (2023-01-24) <a href="https://github.com/tomsasani/bxd_mutator_manuscript">https://github.com/tomsasani/bxd_mutator_manuscript</a></div>
</div>
<div id="ref-z6H1lPG4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>tomsasani/bxd_mutator_manuscript: Final figure generation updates prior to publication</strong> <div class="csl-block">Tom Sasani</div> <em>Zenodo</em> (2022-02-01) <a href="https://doi.org/grrwv8">https://doi.org/grrwv8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.5281/zenodo.5941048">10.5281/zenodo.5941048</a></div></div>
</div>
<div id="ref-t8fRZlGg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>BXD Genotype / WebQTL</strong> <a href="https://gn1.genenetwork.org/dbdoc/BXDGeno.html">https://gn1.genenetwork.org/dbdoc/BXDGeno.html</a></div>
</div>
<div id="ref-8Z2SLpxu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>Genetics and Probability in Animal Breeding Experiments</strong> <a href="https://link.springer.com/book/10.1007/978-1-349-04904-2">https://link.springer.com/book/10.1007/978-1-349-04904-2</a></div>
</div>
<div id="ref-1FskMy3ZE" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>Sustainable data analysis with Snakemake.</strong> <div class="csl-block">Felix Mölder, Kim Philipp Jablonski, Brice Letcher, Michael B Hall, Christopher H Tomkins-Tinch, Vanessa Sochat, Jan Forster, Soohyun Lee, Sven O Twardziok, Alexander Kanitz, … Johannes Köster</div> <em>F1000Research</em> (2021-01-18) <a href="https://www.ncbi.nlm.nih.gov/pubmed/34035898">https://www.ncbi.nlm.nih.gov/pubmed/34035898</a> <div class="csl-block">DOI: <a href="https://doi.org/10.12688/f1000research.29032.2">10.12688/f1000research.29032.2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34035898">34035898</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8114187">PMC8114187</a></div></div>
</div>
<div id="ref-V7hHut0M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>ENA Browser</strong> <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB45429">https://www.ebi.ac.uk/ena/browser/view/PRJEB45429</a></div>
</div>
<div id="ref-1F8NSczsP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>Mapping the Effects of Genetic Variation on Chromatin State and Gene Expression Reveals Loci That Control Ground State Pluripotency.</strong> <div class="csl-block">Daniel A Skelly, Anne Czechanski, Candice Byers, Selcan Aydin, Catrina Spruce, Chris Olivier, Kwangbom Choi, Daniel M Gatti, Narayanan Raghupathy, Gregory R Keele, … Laura G Reinholdt</div> <em>Cell stem cell</em> (2020-08-13) <a href="https://www.ncbi.nlm.nih.gov/pubmed/32795400">https://www.ncbi.nlm.nih.gov/pubmed/32795400</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.stem.2020.07.005">10.1016/j.stem.2020.07.005</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32795400">32795400</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7484384">PMC7484384</a></div></div>
</div>
<div id="ref-ibrvarsm" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>Uncovering novel mutational signatures by </strong> <div class="csl-block">SMAshiqul Islam, Marcos Díaz-Gay, Yang Wu, Mark Barnes, Raviteja Vangara, Erik N Bergstrom, Yudou He, Mike Vella, Jingwei Wang, Jon W Teague, … Ludmil B Alexandrov</div> <em>Cell genomics</em> (2022-11-09) <a href="https://www.ncbi.nlm.nih.gov/pubmed/36388765">https://www.ncbi.nlm.nih.gov/pubmed/36388765</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.xgen.2022.100179">10.1016/j.xgen.2022.100179</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/36388765">36388765</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC9646490">PMC9646490</a></div></div>
</div>
<div id="ref-8vP4Q1ja" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>Defects in 8-oxo-guanine repair pathway cause high frequency of C &amp;gt; A substitutions in neuroblastoma</strong> <div class="csl-block">Marlinde L van den Boogaard, Rurika Oka, Anne Hakkert, Linda Schild, Marli E Ebus, Michael R van Gerven, Danny A Zwijnenburg, Piet Molenaar, Lieke L Hoyng, MEmmy M Dolman, … Jan J Molenaar</div> <em>Proceedings of the National Academy of Sciences</em> (2021-09-03) <a href="https://doi.org/grtcs9">https://doi.org/grtcs9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.2007898118">10.1073/pnas.2007898118</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34479993">34479993</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8433536">PMC8433536</a></div></div>
</div>
<div id="ref-DAiplkDB" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">20. </div><div class="csl-right-inline"><strong>Resolution of structural variation in diverse mouse genomes reveals chromatin remodeling due to transposable elements</strong> <div class="csl-block">Ardian Ferraj, Peter A Audano, Parithi Balachandran, Anne Czechanski, Jacob I Flores, Alexander A Radecki, Varun Mosur, David S Gordon, Isha A Walawalkar, Evan E Eichler, … Christine R Beck</div> <em>Cold Spring Harbor Laboratory</em> (2022-09-27) <a href="https://doi.org/grtctb">https://doi.org/grtctb</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2022.09.26.509577">10.1101/2022.09.26.509577</a></div></div>
</div>
<div id="ref-q8rJCwGH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">21. </div><div class="csl-right-inline"><strong>Limited role of generation time changes in driving the evolution of mutation spectrum in humans</strong> <div class="csl-block">Ziyue Gao, Yulin Zhang, Nathan Cramer, Molly Przeworski, Priya Moorjani</div> <em>bioRxiv</em> (2023-01-13) <a href="https://doi.org/grr525">https://doi.org/grr525</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2022.06.17.496622">https://doi.org/10.1101/2022.06.17.496622</a></div></div>
</div>
<div id="ref-WwxkZzW7" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">22. </div><div class="csl-right-inline"><strong>Heritability of de novo germline mutation reveals a contribution from paternal but not maternal genetic factors</strong> <div class="csl-block">Seongwon Hwang, Matthew DC Neville, Genomics England Research Consortium, Felix R Day, Aylwyn Scally</div> <em>bioRxiv</em> (2022-12-17) <a href="https://doi.org/grr526">https://doi.org/grr526</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2022.12.17.520885">https://doi.org/10.1101/2022.12.17.520885</a></div></div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Helvetica", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 12pt;
      line-height: 1.5;
      max-width: 8.5in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
