## Discussion

### Epistasis between germline mutator alleles

We have identified a locus on chromosome 6 that amplifies a C>A germline mutator phenotype in the BXDs, a population of recombinant inbred lines derived from the laboratory mice DBA/2J and C57BL/6J. 
DBA/2J (*D*) alleles at this new locus had no detectable effect on C>A mutation rates in mice that harbored "wild-type" C57BL/6J (*B*) alleles at a previously discovered mutator locus on chromosome 4 [@PMID:35545679]; moreover, mice with *D* alleles at both loci had even higher mutation rates that those with *D* alleles at either locus alone. 
Epistatic interactions between mutator alleles have been previously documented in yeast [@PMID:8964477;@PMID:22567994;PMID:16492773] and in human cell lines [@PMID:35859169], but never to our knowledge in a whole-animal context. 

Importantly, we discovered epistasis between germline mutator alleles in an unnatural population of model organisms that have been inbred by brother-sister mating in a highly controlled laboratory environment [@PMID]. 
This breeding setup likely attenuated the effects of natural selection on all but the most deleterious alleles [@doi:10.1146/annurev.ecolsys.39.110707.173437], and may have facilitated the fixation of large-effect mutator alleles that would be less common in natural, outbreeding populations. 
We have not conclusively fine-mapped the chromosome 6 QTL, but the best candidate causal variant is a nonsynonymous mutation in the DNA glycosylase *Mbd4* that was not observed in wild mouse populations. 
Our results demonstrate that multiple mutator alleles have spontaneously arisen during the evolutionary history of laboratory mice, supporting the hypothesis that mutation pressure can rapidly augment germline mutation rates and that purifying selection is required to keep mutation rates low. 

### *Mbd4* may influence the penetrance of *Mutyh*-dependent DNA damage by regulating apoptosis

Two DNA repair genes overlap the C>A mutator locus on chromosome 6 and also contain nonsynonymous differences between the C57BL/6J and DBA/2J founder strains: *Ogg1*, a glycosylase that excises the oxidative DNA lesion 8-oxoguanine (8-oxoG) [@PMID:17581577], and *Mbd4*, a glycosylase that can remove mispaired thymine nucleotides opposite spontaneously deaminated CpGs. 
Both genes have been previously linked to somatic mutator phenotypes and carcinogenesis: missense mutations and loss-of-heterozygosity in *Ogg1* have been associated with increased risk of human cancer [@PMID:22829015;@PMID:9662341], and copy-number losses of either *Ogg1* or *Mutyh* are linked to elevated rates of spontaneous C>A mutation in human neuroblastoma [@doi:10.1073/pnas.2007898118]. 
Bi-allelic LOF mutations in human *MBD4* underlie a neoplastic syndrome that closely mimics forms of familial adenomatous polyposis caused by LOF mutations in *MUTYH* [@PMID:35460607]. 
A recent study also identified a homozygous frameshift mutation in the primate homolog of *MBD4* that causes a hypermutator phenotype in the maternal germline [@doi:10.1101/2023.03.27.534460].
This phenotype primarily comprises C>T mutations at CpG and CpA sites, a mutation signature that has previously been associated with loss-of-function (LOF) mutations in *Mbd4* [@PMID:12130785]. 

Although *Ogg1* is a member of the same base-excision repair pathway as *Mutyh* (the protein-coding gene we previously implicated as harboring mutator alleles at the locus on chromosome 4), a number of lines of evidence suggest that the p.Asp129Asn missense mutation in *Mbd4* is the more compelling candidate mutator allele. 
Unlike the *Ogg1* p.Thr95Ala mutation, p.Asp129Asn occurs at an amino acid residue that is well-conserved across mammalian species, resides within an annotated protein domain (the *Mbd4* methyl-CpG binding domain), and is predicted to be deleterious by *in silico* tools like SIFT [@PMID:12824425] (Table @tbl:nonsyn-diffs). 
A missense mutation that affects the homologous amino acid in humans (p.Asp142Gly in GRCh38/hg38) is also present on a single haplotype in the Genome Aggregation Database (gnomAD) [@PMID:32461654] and is predicted by SIFT and Polyphen [@PMID:20354512] to be "deleterious" and "probably_damaging" in human genomes, respectively. 

Loss-of-function (LOF) mutations in *Mbd4* are not typically associated with C>A mutator phenotypes. 
Instead, *Mbd4* deficiency is usually implicated in C>T mutagenesis at CpG sites; we did not detect an excess of C>T mutations in BXDs with *D* alleles at the chromosome 6 mutator locus.
However, loss of function mutations in *Mbd4* are known to exacerbate the effects of exogenous DNA damage agents.
For example, mouse embryonic fibroblasts that harbor homozygous knockouts of *Mbd4* fail to undergo apoptosis following treatment with a number of chemotherapeutics and mutagenic compounds [@PMID:14614141]. 
Most of these exogenous mutagens cause DNA damage that is normally repaired by mismatch repair (MMR) machinery, but murine intestinal cells with biallelic *Mbd4* LOF mutations also showed a reduced apoptotic response to gamma irradiation, which is repaired independently of the MMR gene *Mlh1* [@PMID:14562041]. 
Homozygous loss of *Mbd4* function also leads to accelerated intestinal tumor formation in mice that harbor an *Apc* allele that predisposes them to intestinal neoplasia [@PMID:12130785], and mice with biallelic truncations of the *Mbd4* coding sequence exhibit modestly increased mutation rates in colon cancer cell lines, including increased C>A mutation rates in certain lines [@PMID:17285135].

Together, these lines of evidence suggest that *Mbd4* can modulate sensitivity to many types of exogenous mutagens, potentially through its role in determining whether cells harboring DNA damage should undergo apoptosis [@PMID:14614141;@PMID:14562041]. 
We speculate that in mice with deficient 8-oxoguanine repair &mdash; caused by a mutator allele in *Mutyh*, for example &mdash; reactive oxygen species (ROS) may lead to accumulated DNA damage and elevated C>A mutation rates in the germline. 
If these germ cells harbor wild-type copies of *Mbd4*, they might be able to trigger apoptosis and mitigate the effects of the *Mutyh* mutator allele; however, mice with loss-of-function mutations in both *Mbd4* and *Mutyh* would accumulate even higher levels of germline DNA damage and experience germline mutation rates. 

Although we are unable to conclusively determine that either *Mbd4* or *Ogg1* harbors the causal variant underlying the observed C>A mutator phenotype, we anticipate that future experimental evidence will help pinpoint a causal allele and provide insight into the mechanism of epistasis between mutator alleles in the BXDs.

### Potential roles of structural variation and mobile elements as mutator alleles

Although we argue above that *Mbd4* and *Ogg1* are the best candidate genes to explain the augmented C>A mutator phenotype in a subset of the BXDs, we cannot conclusively determine that either the p.Asp129Asn or p.Thr95Ala missense mutation is a causal allele. 
We previously hypothesized that *Mutyh* missense mutations on *D* haplotypes were responsible for the large-effect C>A mutator phenotype we observed in the BXDs [@PMID:35545679]. 

Using recently published long-read assemblies of several inbred laboratory mouse strains, we were able to able to test several structural variants and 
mobile element insertions for association with the chromosome 6 mutator phenotype. In the case of the chromosome 4 mutator locus, these long-read 
assemblies revealed that the mutator phenotype is potentially caused by a ~5 kbp mobile element insertion (MEI) within the first intron of *Mutyh* 
[@doi:10.1016/j.xgen.2023.100291], which is associated with significantly reduced expression of *Mutyh* in embryonic stem cells. In contrast, we found no similarly compelling evidence that structural variation might underly the chromsome 6 mutator locus. 

### Strengths and limitations of the inter-haplotype distance approach

Our inter-haplotype distance (IHD) approach was able to identify a mutator allele that escaped notice using quantitative trait locus (QTL) mapping. 
To more systematically compare the relative power of IHD and QTL mapping, we performed simulations under a variety of possible parameter regimes. 
Overall, we found that IHD and QTL mapping have similar power to detect mutator alleles in biallelic recombinant inbred lines (RILs) in which we've identified tens or hundreds of *de novo* germline mutations (Figure @fig:ihd_vs_qtl_power).
Nonetheless, only IHD was able to discover the mutator locus on chromosome 6 in the BXDs, demonstrating that it outperforms QTL mapping in certain experimental systems and can reveal previously undiscovered signals of mutator alleles. 
Because the BXDs were generated in six breeding epochs over a period of nearly 40 years, the oldest lines have accumulated many more mutations than the younger lines; these younger RILs also have much noisier mutation spectra as a result.
While QTL mapping approaches weight the phenotypic measurements of each sample equally, IHD compares the *aggregate* mutation spectra of haplotypes at every locus, a property that likely increased its power to detect mutators in the BXD dataset.

Another benefit of the IHD approach is that it obviates the need to perform separate association tests for for every possible $k$-mer mutation type, and therefore the need to adjust significance thresholds for multiple tests.
Since IHD compares the complete mutation spectrum between haplotypes that carry either allele at a site, it would also be well-powered to detect a mutator allele that exerted a coordinated effect on multiple $k$-mer mutation types (e.g., increased the rates of both C>T and C>A mutations). 

However, the IHD method suffers a handful of drawbacks when compared to QTL mapping.
Popular QTL mapping methods (such as R/qtl2 [@PMID:30591514]) use linear models to test associations between genotypes and phenotypes, enabling the inclusion of additive and interactive covariates, as well as kinship matrices, in QTL scans.
Although we have included simple methods to account for inter-sample relatedness in the IHD approach, they are not as robust or flexible as similar methods in QTL mapping software.
Additionally, the IHD method assumes that mutator alleles affect a subset of $k$-mer mutation types; if a mutator allele increased the rates of all mutation types equally on haplotypes that carried it, IHD would be unable to detect it.

Our simulations predict that the IHD method will enjoy its greatest power advantage over QTL mapping in large datasets with very sparse sampling of de novo mutations per individual. For example, if we are able to measure just 10 mutations in each of 10,000 lineages, we predict that the IHD will have X-fold higher power to detect mutators that elevate the mutation rate just Y-fold. 

### Discovering mutator alleles in other experimental systems

Our discovery of a second BXD mutator allele underscores the power of recombinant inbred lines (RILs) as a resource for dissecting the genetic architecture of germline mutation rates. 
Large populations of RILs exist for many model organisms, and we anticipate that as whole-genome sequencing becomes cheaper and cheaper, the IHD method could be useful for future mutator allele discovery outside of the BXDs.
At the same time, RILs are a finite resource that require enormous investments of time and labor to construct. 
If germline mutator alleles are only detectable in these highly unusual experimental populations, we are unlikely to discover more than a small fraction of the mutator alleles that may exist in nature. 

Fortunately, the statistical approach introduced in this paper is readily adaptable to datasets beyond RILs. 
Thousands of human pedigrees have been sequenced in an effort to precisely estimate the rate of human *de novo* germline mutation [@PMID:31549960;@PMID:28959963;@PMID:29700473], and as pedigree sequencing has become a more common step in the diagnosis of many congenital disorders, pedigree datasets are growing on a daily basis. 
Large pedigree datasets are an example of a regime in which IHD could enjoy high power: by pooling sparse mutation counts across many individuals who share the same candidate mutator allele, even a subtle mutator allele signal could rise above the noise.

Selection on germline mutator alleles will likely prevent large-effect mutators from reaching high allele frequencies; however, a subset may be detectable by sequencing a sufficient number of human trios [@PMID:35666194]. 
Previous estimates of power to detect germline mutators in human pedigrees have assumed that mutators affect all mutation types equally, and that methods for mutator discovery will rely on identifying haplotypes with excess total mutation counts [@PMID:35666194]. 
Our results in the BXDs suggest that germline mutators often exert their effects on a small number of mutation types, and may be far more amenable to 
detection by analyzing mutation spectra instead. 