## Discussion

### Epistasis between germline mutator alleles

We have identified a QTL on chromosome 6 that amplifies the germline mutator phenotype caused by \emph{Mutyh} deficiency in certain strains of laboratory 
mice. This new QTL has no detectable effect on the "wild-type" C57BL/6J background, but it further increases the C>A mutation rate in mice with a 
previously discovered 
mutator allele on chromosome 4 [@PMID:35545679]. Epistatic interactions between mutator alleles have been previously documented in the yeast 
\emph{Saccharomyces cerevisiae} [@PMID:8964477; @PMID: 22567994;@PMID:16492773;] and in gene-edited human cell 
lines [@PMID:35859169], 
but never to our knowledge in a whole-animal context. 

Importantly, we discovered this epistatic interaction between germline mutator alleles in an unnatural population of model organisms that have been 
inbred by brother-sister mating 
in a highly controlled laboratory environment  
[@doi:10.1146/annurev.ecolsys.39.110707.173437]. This breeding setup likely attenuated the effects of natural selection on all but the most deleterious 
alleles, possibly facilitating the fixation of large-effect mutator alleles that may be less common in natural, outbreeding mammalian populations. We have 
not conclusively fine-mapped the chromosome 6 QTL, but the best candidate causal variant is a nonsynonymous variant that has not been detected in wild 
mouse populations. Regardless of the origin of the chromosome 6 QTL, our results demonstrate that multiple mutator alleles have spontaneously arisen 
during the evolutionary history of laboratory 
mice, supporting the hypothesis that mutational pressure can increase germline mutation rates fairly quickly and that purifying selection is required to 
keep mutation rates low. 

### Evidence for *Mbd4* as the causal gene underlying the chromosome 6 mutator locus

Two DNA repair genes overlap the C>A mutator locus on chromosome 6 and also contain nonsynonymous differences between the C57BL/6J 
and 
DBA/2J founder strains: *Ogg1*, a glycosylase that excises the oxidative DNA lesion 8-oxoguanine (8-oxoG) [@PMID:17581577], and *Mbd4*, a glycosylase that 
can bind to methylated CpG sites and remove mispaired thymine nucleotides opposite spontaneously deaminated CpGs. Both genes have been previously linked 
to somatic mutator phenotypes and carcinogenesis: missense mutations and loss-of-heterozygosity in *Ogg1* have been associated with increased risk of 
human cancer [@PMID:22829015;@PMID:9662341], and copy-
number losses of either *Ogg1* or *Mutyh* are linked to elevated rates of spontaneous C>A mutation in human neuroblastoma [@doi:10.1073/pnas.2007898118]. 
In addition, bi-allelic LOF mutations in human *MBD4* underlie a neoplastic syndrome that closely mimics forms of familial adenomatous polyposis caused by 
LOF mutations in *MUTYH* [@PMID:35460607]. A recent study identified a homozygous frameshift mutation in the primate homolog of *MBD4* that causes a hypermutator phenotype in the maternal germline 
[@doi:10.1101/2023.03.27.534460].
This phenotype primarily comprises C>T mutations at CpG and CpA sites, a mutation signature that has previously been associated with loss-of-function 
(LOF) mutations in *Mbd4* [@PMID:12130785]. 

Although *Ogg1* is a member of the same base-excision repair pathway as *Mutyh* (the protein-coding gene we previously implicated as harboring mutator 
alleles at the locus on chromosome 4), a number of lines of evidence suggest that the p.Asp129Asn missense mutation in *Mbd4* is the more compelling 
candidate mutator allele. 
Unlike the *Ogg1* p.Thr95Ala mutation, p.Asp129Asn occurs at an amino acid residue that is well-conserved across mammalian species, resides within an 
annotated protein domain (the *Mbd4* methyl-CpG binding domain), and is predicted to be deleterious by *in silico* tools like SIFT [@PMID:12824425] (Table 
@tbl:nonsyn-diffs). 
A missense mutation that affects the homologous amino acid in humans (p.Asp142Gly in GRCh38/hg38) is present on a single haplotype in the Genome 
Aggregation Database (gnomAD) [@PMID:32461654] and is predicted by SIFT and Polyphen [@PMID:20354512] to be "deleterious" and "probably_damaging" in human 
genomes, respectively. 

Loss of *Mbd4* function is not typically associated with a C>A mutator phenotype. Instead, *Mbd4* defects are usually implicated in C>T mutagenesis 
at CpG 
sites, and we detected no association of this mutation type with the chromosome 6 QTL. However, loss of function mutations in *Mbd4* are known to 
exacerbate the effects of exogenous DNA damage agents--mouse embryonic fibroblasts that harbor homozygous knockout mutations in *Mbd4* fail to undergo 
apoptosis following treatment with a number of 
chemotherapeutics and mutagenic compounds [@PMID:14614141]. 
Most of these exogenous mutagens cause DNA damage that is normally repaired by mismatch repair (MMR) machinery, but murine intestinal cells with biallelic 
*Mbd4* knockout mutations also show a reduced apoptotic response to gamma irradiation, which is repaired independently of the MMR gene *Mlh1* 
[@PMID:14562041]. 
Homozygous loss of *Mbd4* function also leads to accelerated intestinal tumor formation in mice that harbor an *Apc* allele that predisposes them to 
intestinal 
neoplasia [@PMID:12130785], and mice with biallelic truncations of the *Mbd4* coding sequence exhibit modestly increased mutation rates in colon cancer 
cell lines, including increased C>A mutation rates in certain lines [@PMID:17285135].

Together, these lines of evidence suggest that *Mbd4* modulates the accumulation of DNA damage outside the CpG sites to which it binds, perhaps through 
its role in determining whether damaged cells should undergo apoptosis [@PMID:14614141;@PMID:14562041]. We speculate that in mice with defective 8-
oxoguanine repair, where reactive oxygen species tend to damage germ cell DNA, *Mbd4* might play a role in determining how often these damaged germ cells 
undergo apoptosis. If *Mbd4* is able to recognize germ cells whose DNA has been damaged as a result of *Mutyh* deficiency and target those cells for 
apoptosis, the effect of the *Mutyh* mutator allele maybe be buffered in a way that will not be possible in mice where *Mbd4* and *Mutyh* are both impaired. 

Ultimately, we are unable to conclusively determine that either *Mbd4* or *Ogg1* harbors the causal variant underlying the observed C>A mutator phenotype. 
We anticipate that future experimental evidence will help pinpoint a causal allele and provide insight into the mechanism of epistasis between alleles in the BXDs.

### Potential roles of structural variation and mobile elements as mutator alleles

Although we believe that *Mbd4* and *Ogg1* are the most likely candidate genes to explain the additional C>A mutator phenotype in the BXDs, we cannot conclusively determine that either the p.Asp129Asn or p.Thr95Ala missense mutation is a causal allele. 
We previously hypothesized that *Mutyh* missense mutations on *D* haplotypes were responsible for the large-effect C>A mutator phenotype we observed in the BXDs [@PMID:35545679]. 
Using high-quality long-read assemblies of inbred laboratory strains, another group recently identified a ~5 kbp mobile element insertion (MEI) within the first intron of *Mutyh* [@doi:10.1016/j.xgen.2023.100291] that is present on *D* haplotypes and absent from *B* haplotypes. 
This MEI is associated with significantly reduced expression of *Mutyh* in embryonic stem cells from laboratory strains, and may in fact underlie the previous C>A germline mutator phenotype in the BXDs. 
Although we did not find compelling evidence that structural variants were responsible for the novel C>A mutator phenotype observed in this study, it remains plausible that large SVs or MEIs contribute to mutation spectrum evolution in mammalian genomes.

### Strengths and limitations of the inter-haplotype distance approach

Using simulated data, we found that both inter-haplotype distance (IHD) and QTL mapping had similar power to detect mutators that augment the rates of specific 1-mer and 3-mer mutation types (Figure @fig:ihd_vs_qtl_power).
Nonetheless, only IHD was able to discover the mutator locus on chromosome 6 in the BXDs, demonstrating that it outperforms QTL mapping in certain experimental systems and can reveal previously undiscovered signals of mutator alleles.
One benefit of the IHD approach is that it obviates the need to perform separate association tests for for every possible $k$-mer mutation type, and therefore the need to adjust significance thresholds for multiple tests.
Since IHD compares the complete mutation spectrum between haplotypes that carry either allele at a site, it would also be well-powered to detect a mutator allele that exerted a coordinated effect on multiple $k$-mer mutation types (e.g., increased the rates of both C>T and C>A mutations).

However, the IHD method suffers a handful of drawbacks when compared to QTL mapping.
Popular QTL mapping methods (such as R/qtl2 [@PMID:30591514]) use linear models to test associations between genotypes and phenotypes, enabling the inclusion of additive and interactive covariates, as well as kinship matrices, in QTL scans.
Although we have included simple methods to account for inter-sample relatedness in the IHD approach, they are not as robust or flexible as similar methods in QTL mapping software.
Additionally, the IHD method assumes that mutator alleles affect a subset of $k$-mer mutation types.
If a mutator allele increased the rates of all mutation types equally on haplotypes that carried it, IHD would be unable to detect it.


### Discovering mutator alleles in other systems

Mutator alleles likely contribute to variation in mutation rates and spectra across the tree of life. 
In two natural isolates of *Saccharomyces cerevisiae*, nonsynonymous variation in *OGG1* causes a substantial increase in the C>A *de novo* mutation rate [@PMID:34523420]. 
Recent analyses have suggested that mutator alleles and/or environmental mutagens have shaped mutation rate evolution both in human genomes [@doi:10.1101/2022.06.17.496622] and more broadly during great ape evolution [@PMID:33983415]. 
The heritability of paternal *de novo* mutation counts in the human germline has also been estimated to be between 10 and 20%, demonstrating a contribution of genetic factors to germline mutation rates [@doi:10.1101/2022.12.17.520885]. 
However, mutator discovery remains challenging in mammalian genomes.

Thousands of human pedigrees have been sequenced in an effort to precisely estimate the rate of human *de novo* germline mutation [@PMID:31549960;@PMID:28959963;@PMID:29700473]. 
Selection on germline mutator alleles will likely prevent large-effect mutators from reaching high allele frequencies; however, a subset may be detectable by sequencing a sufficient number of human trios [@PMID:35666194]. 
Current estimates of power to detect germline mutators in human pedigrees generally assume that mutators affect all mutation types equally, and that methods for mutator discovery will rely on identifying haplotypes with excess total mutation counts [@PMID:35666194]. 
Our results in the BXDs suggest that germline mutators often exert their effects on a small number of $k$-mer mutation types, and may be far more amenable to detection by analyzing mutation spectra instead.


<!-- ### Using germline mutation spectra to identify mutator alleles

Germline mutation spectra are a rich source of information about the demographic history of populations, as well as the activity of both exogenous and endogenous sources of mutation throughout time. For example, by analyzing the 3-mer mutation spectrum in a collection of human genomes, Harris and Pritchard [@PMID:28440220] discovered a "pulse" of TCC>TTC mutation activity in European populations that likely occurred between 15,000 and 2,000 years ago, and perhaps began even earlier [@PMID:34016747]).

Within somatic tissues, mutation spectra can also be used to uncover the mutational processes active in particular populations of cells [@PMID:23945592]. New computational methods have been developed to extract "mutational signatures" from large databases of somatic mutations in cancer [@PMID:36388765]. These signatures, which describe the relative frequency of each 3-mer mutation type, can often be precisely attributed to chemotherapeutic agents, exposures to environmental mutagens, or loss-of-function mutations in genes encoding DNA repair or replication proteins [@PMID:23945592;@PMID:31740835;@PMID:27811275]. 

Although a germline mutator allele should increase the absolute count of mutations on a linked haplotype, our results demonstrate that its effects can be more easily detectable by examining mutation *spectra* instead. For example, *D* alleles at the mutator locus on chromosome 6 augment the C>A mutation rate by a factor of approximately 1.2 (Figure {@fig:distance-results}). Since C>A mutations comprise approximately 10% of all germline mutations to begin with, *D* alleles only increase the overall germline mutation rate by about 2%. Given the depth of information that can be encoded in the mutation spectrum, we expect that mutation spectra can be further exploited to discover genetic modifiers of the mutation rate in other study systems, as well.  -->
