<!DOCTYPE html>
<html xmlns="http://www.w3.org/1999/xhtml" lang="en-US" xml:lang="en-US">
<head>
  <meta charset="utf-8" />
  <meta name="generator" content="pandoc" />
  <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes" />
  <meta name="author" content="Thomas A. Sasani" />
  <meta name="author" content="Aaron R. Quinlan" />
  <meta name="author" content="Kelley Harris" />
  <meta name="dcterms.date" content="2023-09-06" />
  <meta name="keywords" content="markdown, publishing, manubot" />
  <title>Epistasis between mutator alleles contributes to germline mutation rate variability in laboratory mice</title>
  <style>
    code{white-space: pre-wrap;}
    span.smallcaps{font-variant: small-caps;}
    div.columns{display: flex; gap: min(4vw, 1.5em);}
    div.column{flex: auto; overflow-x: auto;}
    div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
    ul.task-list{list-style: none;}
    ul.task-list li input[type="checkbox"] {
      width: 0.8em;
      margin: 0 0.8em 0.2em -1.6em;
      vertical-align: middle;
    }
    pre > code.sourceCode { white-space: pre; position: relative; }
    pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
    pre > code.sourceCode > span:empty { height: 1.2em; }
    .sourceCode { overflow: visible; }
    code.sourceCode > span { color: inherit; text-decoration: inherit; }
    div.sourceCode { margin: 1em 0; }
    pre.sourceCode { margin: 0; }
    @media screen {
    div.sourceCode { overflow: auto; }
    }
    @media print {
    pre > code.sourceCode { white-space: pre-wrap; }
    pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
    }
    pre.numberSource code
      { counter-reset: source-line 0; }
    pre.numberSource code > span
      { position: relative; left: -4em; counter-increment: source-line; }
    pre.numberSource code > span > a:first-child::before
      { content: counter(source-line);
        position: relative; left: -1em; text-align: right; vertical-align: baseline;
        border: none; display: inline-block;
        -webkit-touch-callout: none; -webkit-user-select: none;
        -khtml-user-select: none; -moz-user-select: none;
        -ms-user-select: none; user-select: none;
        padding: 0 4px; width: 4em;
        color: #aaaaaa;
      }
    pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
    div.sourceCode
      {   }
    @media screen {
    pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
    }
    code span.al { color: #ff0000; font-weight: bold; } /* Alert */
    code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
    code span.at { color: #7d9029; } /* Attribute */
    code span.bn { color: #40a070; } /* BaseN */
    code span.bu { color: #008000; } /* BuiltIn */
    code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
    code span.ch { color: #4070a0; } /* Char */
    code span.cn { color: #880000; } /* Constant */
    code span.co { color: #60a0b0; font-style: italic; } /* Comment */
    code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
    code span.do { color: #ba2121; font-style: italic; } /* Documentation */
    code span.dt { color: #902000; } /* DataType */
    code span.dv { color: #40a070; } /* DecVal */
    code span.er { color: #ff0000; font-weight: bold; } /* Error */
    code span.ex { } /* Extension */
    code span.fl { color: #40a070; } /* Float */
    code span.fu { color: #06287e; } /* Function */
    code span.im { color: #008000; font-weight: bold; } /* Import */
    code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
    code span.kw { color: #007020; font-weight: bold; } /* Keyword */
    code span.op { color: #666666; } /* Operator */
    code span.ot { color: #007020; } /* Other */
    code span.pp { color: #bc7a00; } /* Preprocessor */
    code span.sc { color: #4070a0; } /* SpecialChar */
    code span.ss { color: #bb6688; } /* SpecialString */
    code span.st { color: #4070a0; } /* String */
    code span.va { color: #19177c; } /* Variable */
    code span.vs { color: #4070a0; } /* VerbatimString */
    code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
    div.csl-bib-body { }
    div.csl-entry {
      clear: both;
    }
    .hanging div.csl-entry {
      margin-left:2em;
      text-indent:-2em;
    }
    div.csl-left-margin {
      min-width:2em;
      float:left;
    }
    div.csl-right-inline {
      margin-left:2em;
      padding-left:1em;
    }
    div.csl-indent {
      margin-left: 2em;
    }
  </style>
  <!--
  Manubot generated metadata rendered from header-includes-template.html.
  Suggest improvements at https://github.com/manubot/manubot/blob/main/manubot/process/header-includes-template.html
  -->
  <meta name="dc.format" content="text/html" />
  <meta property="og:type" content="article" />
  <meta name="dc.title" content="Epistasis between mutator alleles contributes to germline mutation rate variability in laboratory mice" />
  <meta name="citation_title" content="Epistasis between mutator alleles contributes to germline mutation rate variability in laboratory mice" />
  <meta property="og:title" content="Epistasis between mutator alleles contributes to germline mutation rate variability in laboratory mice" />
  <meta property="twitter:title" content="Epistasis between mutator alleles contributes to germline mutation rate variability in laboratory mice" />
  <meta name="dc.date" content="2023-09-06" />
  <meta name="citation_publication_date" content="2023-09-06" />
  <meta property="article:published_time" content="2023-09-06" />
  <meta name="dc.modified" content="2023-09-06T21:23:20+00:00" />
  <meta property="article:modified_time" content="2023-09-06T21:23:20+00:00" />
  <meta name="dc.language" content="en-US" />
  <meta name="citation_language" content="en-US" />
  <meta name="dc.relation.ispartof" content="Manubot" />
  <meta name="dc.publisher" content="Manubot" />
  <meta name="citation_journal_title" content="Manubot" />
  <meta name="citation_technical_report_institution" content="Manubot" />
  <meta name="citation_author" content="Thomas A. Sasani" />
  <meta name="citation_author_institution" content="Department of Human Genetics, University of Utah" />
  <meta name="citation_author_orcid" content="0000-0003-2317-1374" />
  <meta name="twitter:creator" content="@tomsasani" />
  <meta name="citation_author" content="Aaron R. Quinlan" />
  <meta name="citation_author_institution" content="Department of Human Genetics, University of Utah" />
  <meta name="citation_author_institution" content="Department of Biomedical Informatics, University of Utah" />
  <meta name="citation_author_orcid" content="0000-0003-1756-0859" />
  <meta name="twitter:creator" content="@aaronquinlan" />
  <meta name="citation_author" content="Kelley Harris" />
  <meta name="citation_author_institution" content="Department of Genome Sciences, University of Washington" />
  <meta name="citation_author_orcid" content="0000-0003-0302-2523" />
  <meta name="twitter:creator" content="@Kelley__Harris" />
  <link rel="canonical" href="https://quinlan-lab.github.io/mutator-epistasis-manuscript/" />
  <meta property="og:url" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/" />
  <meta property="twitter:url" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/" />
  <meta name="citation_fulltext_html_url" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/" />
  <meta name="citation_pdf_url" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/manuscript.pdf" />
  <link rel="alternate" type="application/pdf" href="https://quinlan-lab.github.io/mutator-epistasis-manuscript/manuscript.pdf" />
  <link rel="alternate" type="text/html" href="https://quinlan-lab.github.io/mutator-epistasis-manuscript/v/b4dbe7866fc8c6823f0f81f3e1685a9f56262aab/" />
  <meta name="manubot_html_url_versioned" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/v/b4dbe7866fc8c6823f0f81f3e1685a9f56262aab/" />
  <meta name="manubot_pdf_url_versioned" content="https://quinlan-lab.github.io/mutator-epistasis-manuscript/v/b4dbe7866fc8c6823f0f81f3e1685a9f56262aab/manuscript.pdf" />
  <meta property="og:type" content="article" />
  <meta property="twitter:card" content="summary_large_image" />
  <link rel="icon" type="image/png" sizes="192x192" href="https://manubot.org/favicon-192x192.png" />
  <link rel="mask-icon" href="https://manubot.org/safari-pinned-tab.svg" color="#ad1457" />
  <meta name="theme-color" content="#ad1457" />
  <!-- end Manubot generated metadata -->
  <!--[if lt IE 9]>
    <script src="//cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv-printshiv.min.js"></script>
  <![endif]-->
</head>
<body>
<header id="title-block-header">
<h1 class="title">Epistasis between mutator alleles contributes to germline mutation rate variability in laboratory mice</h1>
</header>
<p><small><em>
This manuscript
(<a href="https://quinlan-lab.github.io/mutator-epistasis-manuscript/v/b4dbe7866fc8c6823f0f81f3e1685a9f56262aab/">permalink</a>)
was automatically generated
from <a href="https://github.com/quinlan-lab/mutator-epistasis-manuscript/tree/b4dbe7866fc8c6823f0f81f3e1685a9f56262aab">quinlan-lab/mutator-epistasis-manuscript@b4dbe78</a>
on September 6, 2023.
</em></small></p>
<h2 id="authors">Authors</h2>
<ul>
<li><p><strong>Thomas A. Sasani</strong>
<br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-2317-1374">0000-0003-2317-1374</a>
· <img src="images/github.svg" class="inline_icon" width="16" height="16" alt="GitHub icon" />
<a href="https://github.com/tomsasani">tomsasani</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/tomsasani">tomsasani</a>
<br>
<small>
Department of Human Genetics, University of Utah
</small></p></li>
<li><p><strong>Aaron R. Quinlan</strong>
<sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-1756-0859">0000-0003-1756-0859</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/aaronquinlan">aaronquinlan</a>
<br>
<small>
Department of Human Genetics, University of Utah; Department of Biomedical Informatics, University of Utah
· Funded by NIH/NHGRI R01HG012252
</small></p></li>
<li><p><strong>Kelley Harris</strong>
<sup><a href="#correspondence">✉</a></sup><br>
<img src="images/orcid.svg" class="inline_icon" width="16" height="16" alt="ORCID icon" />
<a href="https://orcid.org/0000-0003-0302-2523">0000-0003-0302-2523</a>
· <img src="images/twitter.svg" class="inline_icon" width="16" height="16" alt="Twitter icon" />
<a href="https://twitter.com/Kelley__Harris">Kelley__Harris</a>
<br>
<small>
Department of Genome Sciences, University of Washington
· Funded by NIH/NIGMS R35GM133428; Burroughs Wellcome Career Award at the Scientific Interface; Searle Scholarship; Pew Scholarship; Sloan Fellowship; Allen Discovery Center for Cell Lineage Tracing
</small></p></li>
</ul>
<div id="correspondence">
<p>✉ — Correspondence possible via <a href="https://github.com/quinlan-lab/mutator-epistasis-manuscript/issues">GitHub Issues</a>
or email to
Aaron R. Quinlan &lt;aquinlan@genetics.utah.edu&gt;,
Kelley Harris &lt;harriske@uw.edu&gt;.</p>
</div>
<h2 class="page_break_before" id="abstract">Abstract</h2>
<p>Maintaining germline genome integrity is essential and enormously complex.
Hundreds of proteins are involved in DNA replication and proofreading, and hundreds more are mobilized to repair DNA damage <span class="citation" data-cites="PMID:28485537">[<a href="#ref-PMID:28485537" role="doc-biblioref"><strong>PMID:28485537?</strong></a>]</span>.
While loss-of-function mutations in any of the genes encoding these proteins might lead to elevated mutation rates, <em>mutator alleles</em> have largely eluded detection in mammals.</p>
<p>DNA replication and repair proteins often recognize particular sequence motifs or excise lesions at specific nucleotides.
Thus, we might expect that the spectrum of <em>de novo</em> mutations — that is, the frequency of each individual mutation type (C&gt;T, A&gt;G, etc.) — will differ between genomes that harbor either a mutator or wild-type allele at a given locus.
Previously, we used quantitative trait locus mapping to discover candidate mutator alleles in the DNA repair gene <em>Mutyh</em> that increased the C&gt;A germline mutation rate in a family of inbred mice known as the BXDs <span class="citation" data-cites="PMID:35545679 PMID:33472028">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>,<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>.</p>
<blockquote>
<p>In this study we developed a new method, called “aggregate mutation spectrum distance,” to detect alleles associated with mutation spectrum variation.
By applying this approach to mutation data from the BXDs, we confirmed the presence of the germline mutator locus near <em>Mutyh</em> and discovered an additional C&gt;A mutator locus on chromosome 6 that overlaps <em>Ogg1</em>, a DNA glycosylase involved in the same base-excision repair network as <em>Mutyh</em> <span class="citation" data-cites="PMID:17581577">[<a href="#ref-PMID:17581577" role="doc-biblioref"><strong>PMID:17581577?</strong></a>]</span>.</p>
</blockquote>
<p>The effect of a chromosome 6 mutator allele depended on the presence of a mutator allele near <em>Mutyh</em>, and BXDs with mutator alleles at both loci had even greater numbers of C&gt;A mutations than those with mutator alleles at either locus alone.</p>
<p>Our new methods for analyzing mutation spectra reveal evidence of epistasis between germline mutator alleles, and may be applicable to mutation data from humans and other model organisms.</p>
<h2 id="introduction">Introduction</h2>
<p>Germline mutation rates reflect the complex interplay between DNA proofreading and repair pathways, exogenous sources of DNA damage, and life-history traits.
For example, parental age is an important determinant of mutation rate variability; in many mammalian species, the number of germline <em>de novo</em> mutations observed in offspring increases as a function of paternal and maternal age <span class="citation" data-cites="PMID:28959963 PMID:31549960 PMID:35771663 PMID:32804933 PMID:31492841">[<a href="#ref-PMID:28959963" role="doc-biblioref"><strong>PMID:28959963?</strong></a>,<a href="#ref-PMID:31549960" role="doc-biblioref"><strong>PMID:31549960?</strong></a>,<a href="#ref-PMID:35771663" role="doc-biblioref"><strong>PMID:35771663?</strong></a>,<a href="#ref-PMID:32804933" role="doc-biblioref"><strong>PMID:32804933?</strong></a>,<a href="#ref-PMID:31492841" role="doc-biblioref"><strong>PMID:31492841?</strong></a>]</span>.
Rates of germline mutation accumulation are also variable across human families <span class="citation" data-cites="PMID:26656846 PMID:31549960">[<a href="#ref-PMID:26656846" role="doc-biblioref"><strong>PMID:26656846?</strong></a>,<a href="#ref-PMID:31549960" role="doc-biblioref"><strong>PMID:31549960?</strong></a>]</span>, likely due to either genetic variation or differences in environmental exposures.
Although numerous protein-coding genes contribute to the maintenance of genome integrity, genetic variants that increase germline mutation rates, known as <em>mutator alleles</em>, have proven difficult to discover in mammals.</p>
<p>The dearth of observed germline mutators in mammalian genomes is not necessarily surprising, since alleles that lead to elevated germline mutation rates would likely have deleterious consequences and be purged by negative selection <span class="citation" data-cites="PMID:27739533">[<a href="#ref-PMID:27739533" role="doc-biblioref"><strong>PMID:27739533?</strong></a>]</span>.
Moreover, germline mutation rates are relatively low, and direct mutation rate measurements require whole-genome sequencing data from both parents and their offspring.
As a result, large-scale association studies — which have been used to map the contributions of common genetic variants to many complex traits — are not currently well-powered to investigate the polygenic architecture of germline mutation rates <span class="citation" data-cites="PMID:31964835">[<a href="#ref-PMID:31964835" role="doc-biblioref"><strong>PMID:31964835?</strong></a>]</span>.</p>
<p>Despite these challenges, less traditional strategies have been used to identify a small number of mutator alleles in humans, macaques <span class="citation" data-cites="12CXVVJzy">[<a href="#ref-12CXVVJzy" role="doc-biblioref">1</a>]</span>, and mice.
By focusing on families with rare genetic diseases, a recent study discovered two mutator alleles that led to significantly elevated rates of <em>de novo</em> germline mutation in human genomes <span class="citation" data-cites="PMID:35545669">[<a href="#ref-PMID:35545669" role="doc-biblioref"><strong>PMID:35545669?</strong></a>]</span>.
Another group observed mutator phenotypes in the sperm and somatic tissues of adults who carry cancer-predisposing inherited mutations in the POLE/POLD1 exonucleases <span class="citation" data-cites="PMID:34594041">[<a href="#ref-PMID:34594041" role="doc-biblioref"><strong>PMID:34594041?</strong></a>]</span>.
Candidate mutator loci were also found by identifying human haplotypes from the Thousand Genomes Project with excess counts of derived alleles in genomic windows <span class="citation" data-cites="PMID:28095480">[<a href="#ref-PMID:28095480" role="doc-biblioref"><strong>PMID:28095480?</strong></a>]</span>.</p>
<p>In mice, a germline mutator allele was recently discovered by sequencing a large family of inbred mice <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.
Commonly known as the <u>B</u>X<u>D</u>s, these recombinant inbred lines (RILs) were derived from either F2 or advanced intercrosses of C57<u>B</u>L/6J and <u>D</u>BA/2J, two laboratory strains that exhibit significant differences in their germline mutation spectra <span class="citation" data-cites="PMID:33472028 PMID:30753674">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>,<a href="#ref-PMID:30753674" role="doc-biblioref"><strong>PMID:30753674?</strong></a>]</span>.
The BXDs were maintained via brother-sister mating for up to 180 generations, and each BXD therefore accumulated hundreds or thousands of germline mutations on a nearly-homozygous linear mosaic of parental <u>B</u> and <u>D</u> haplotypes.
Due to their husbandry in a controlled laboratory setting, the BXDs were largely free from confounding by environmental heterogeneity, and the effects of selection on <em>de novo</em> mutations were attenuated by strict inbreeding <span class="citation" data-cites="LF8DCRcP">[<a href="#ref-LF8DCRcP" role="doc-biblioref">2</a>]</span>.</p>
<p>In this previous study, whole-genome sequencing data from the BXD family were used to map a quantitative trait locus (QTL) for the C&gt;A mutation rate <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.
Germline C&gt;A mutation rates were nearly 50% higher in mice with <em>D</em> haplotypes at the QTL, likely due to genetic variation in the DNA glycosylase <em>Mutyh</em> that reduced the efficacy of oxidative DNA damage repair.
Importantly, the QTL did not reach genome-wide significance in a scan for variation in overall germline mutation rates, which were only modestly higher in BXDs with <em>D</em> alleles, demonstrating the utility of mutation spectrum analysis for mutator allele discovery.
Close examination of the mutation spectrum is likely to be broadly useful for detecting mutator alleles, as genes involved in DNA proofreading and repair often recognize particular sequence motifs or excise specific types of DNA lesions <span class="citation" data-cites="PMID:32619789">[<a href="#ref-PMID:32619789" role="doc-biblioref"><strong>PMID:32619789?</strong></a>]</span>.
Mutation spectra are usually defined in terms of <span class="math inline">\(k\)</span>-mer nucleotide context; the 1-mer mutation spectrum, for example, consists of 6 mutation types after collapsing by strand complement (C&gt;T, C&gt;A, C&gt;G, A&gt;T, A&gt;C, A&gt;G), while the 3-mer mutation spectrum contains 96 (each of the 1-mer mutations partitioned by trinucleotide context).</p>
<p>Although mutation spectrum analysis can enable the discovery of mutator alleles that affect the rates of specific mutation types, early implementations of this strategy have suffered from a few drawbacks.
For example, performing association tests on the rates or fractions of every <span class="math inline">\(k\)</span>-mer mutation type can quickly incur a substantial multiple testing burden.
Since germline mutation rates are generally quite low, estimates of <span class="math inline">\(k\)</span>-mer mutation type frequencies from individual samples can also be noisy and imprecise.
We were therefore motivated to develop a statistical method that could overcome the sparsity of <em>de novo</em> mutation spectra, eliminate the need to test each <span class="math inline">\(k\)</span>-mer mutation type separately, and enable sensitive detection of alleles that influence the germline mutation spectrum.</p>
<blockquote>
<p>Here, we present a new mutation spectrum association test, called “aggregate mutation spectrum distance,” that minimizes multiple testing burdens and mitigates the challenges of sparsity in <em>de novo</em> mutation datasets.</p>
</blockquote>
<p>We leverage this method to re-analyze germline mutation data from the BXD family and find compelling evidence for a second mutator allele that was not detected using previous approaches.
The new allele appears to interact epistatically with the mutator that was previously discovered in the BXDs, further augmenting the C&gt;A germline mutation rate in a subset of inbred mice.
Our observation of epistasis suggests that mild DNA repair deficiencies can compound one another, as mutator alleles chip away at the redundant systems that collectively maintain germline integrity.</p>
<h2 id="results">Results</h2>
<h3 id="a-novel-method-for-detecting-mutator-alleles">A novel method for detecting mutator alleles</h3>
<blockquote>
<p>We developed a statistical method, termed “aggregate mutation spectrum distance” (AMSD), to detect loci that are associated with mutation spectrum variation in recombinant inbred lines (RILs) (Figure <a href="#fig:distance-method">1</a>; <em>Materials and Methods</em>).</p>
</blockquote>
<p>Our approach leverages the fact that mutator alleles often leave behind distinct and detectable impressions on the <em>mutation spectrum</em>, even if they increase the overall mutation rate by a relatively small amount.
Given a population of haplotypes, we assume that each has been genotyped at the same collection of biallelic loci and that each harbors <em>de novo</em> mutations which have been partitioned by <span class="math inline">\(k\)</span>-mer context (Figure <a href="#fig:distance-method">1</a>).
At every locus, we calculate a cosine distance between the aggregate mutation spectra of haplotypes that inherited either parental allele.
Using permutation tests, we then identify loci at which those distances are larger than what we’d expect by random chance.
To account for polygenic effects on the mutation process that might be shared between BXDs, we also regress the cosine distance at each marker against the genetic similarity between haplotype groups, and assess significance using the fitted residuals (which we call the “adjusted” cosine distances) (<em>Materials and Methods</em>).</p>
<p>Using simulated data, we find that our method’s power is primarily limited by the initial mutation rate of the <span class="math inline">\(k\)</span>-mer mutation type affected by a mutator allele and the total number of <em>de novo</em> mutations used to detect it (Figure <a href="#fig:simulations">1-figure supplement 1</a>).
Given 100 haplotypes with an average of 500 <em>de novo</em> germline mutations each, AMSD has approximately 90% power to detect a mutator allele that increases the C&gt;A <em>de novo</em> mutation rate by as little as 20%.
However, the approach has less than 20% power to detect a mutator of identical effect size that augments the C&gt;G mutation rate, since C&gt;G mutations are expected to make up a smaller fraction of all <em>de novo</em> germline mutations to begin with.
Simulations also demonstrate that our approach is well-powered to detect large-effect mutator alleles (e.g., those that increase the mutation rate of a specific <span class="math inline">\(k\)</span>-mer by 50%), even with a relatively small number of mutations per haplotype (Figure <a href="#fig:simulations">1-figure supplement 1</a>).
Both AMSD and traditional quantitative trait locus (QTL) mapping have similar power to detect alleles that augment the rates of individual 1-mer mutation types (Figure <a href="#fig:ihd_vs_qtl_power">1-figure supplement 2</a>), but AMSD has a number of potential advantages for mutator allele discovery.</p>
<blockquote>
<p>For example, we find that AMSD is better-powered than QTL mapping when the number of simulated <em>de novo</em> mutations is allowed to vary (by a factor of 20) across haplotypes (Figure <a href="#fig:ihd_vs_qtl_power_variable_counts">1-figure supplement 3</a>).
However, we also caution that many of the parameters used in our simulations are specific to the BXD mice (e.g., numbers of haplotypes, average numbers of mutations, expected allele frequencies at markers), and do not necessarily reflect the power of AMSD on other populations.</p>
</blockquote>
<div id="fig:distance-method" class="fignos">
<figure>
<img src="images/fig-distance-method.png" style="width:7.5in" alt="Figure 1: Overview of aggregate mutation spectrum distance method for discovering mutator alleles. a) A population of four haplotypes has been genotyped at three informative markers (g_1 through g_3); each haplotype also harbors unique de novo germline mutations. In practice, de novo mutations are partitioned by k-mer context; for simplicity in this toy example, de novo mutations are simply classified into two possible mutation types (grey squares represent C&gt;(A/T/G) mutations, while grey triangles represent A&gt;(C/T/G) mutations). b) At each informative marker g_n, we calculate the total number of each mutation type observed on haplotypes that carry either parental allele (i.e., the aggregate mutation spectrum) using all genome-wide de novo mutations. For example, haplotypes with A (orange) genotypes at g_1 carry a total of three “triangle” mutations and five “square” mutations, and haplotypes with B (green) genotypes carry a total of six triangle and two square mutations. We then calculate the cosine distance between the two aggregate mutation spectra, which we call the “aggregate mutation spectrum distance.” Cosine distance can be defined as 1 - \cos(\theta), where \theta is the angle between two vectors; in this case, the two vectors are the two aggregate spectra. We repeat this process for every informative marker g_n. c) To assess the significance of any distance peaks in b), we perform permutation tests. In each of N permutations, we shuffle the haplotype labels associated with the de novo mutation data, run a genome-wide distance scan, and record the maximum cosine distance encountered at any locus in the scan. Finally, we calculate the 1 - p percentile of the distribution of those maximum distances to obtain a genome-wide cosine distance threshold at the specified value of p." />
<figcaption aria-hidden="true"><span>Figure 1:</span> <strong>Overview of aggregate mutation spectrum distance method for discovering mutator alleles.</strong>
<strong>a)</strong> A population of four haplotypes has been genotyped at three informative markers (<span class="math inline">\(g_1\)</span> through <span class="math inline">\(g_3\)</span>); each haplotype also harbors unique <em>de novo</em> germline mutations.
In practice, <em>de novo</em> mutations are partitioned by <span class="math inline">\(k\)</span>-mer context; for simplicity in this toy example, <em>de novo</em> mutations are simply classified into two possible mutation types (grey squares represent C&gt;(A/T/G) mutations, while grey triangles represent A&gt;(C/T/G) mutations).
<strong>b)</strong> At each informative marker <span class="math inline">\(g_n\)</span>, we calculate the total number of each mutation type observed on haplotypes that carry either parental allele (i.e., the aggregate mutation spectrum) using all genome-wide <em>de novo</em> mutations.
For example, haplotypes with <em>A</em> (orange) genotypes at <span class="math inline">\(g_1\)</span> carry a total of three “triangle” mutations and five “square” mutations, and haplotypes with <em>B</em> (green) genotypes carry a total of six triangle and two square mutations.
We then calculate the cosine distance between the two aggregate mutation spectra, which we call the “aggregate mutation spectrum distance.”
Cosine distance can be defined as <span class="math inline">\(1 - \cos(\theta)\)</span>, where <span class="math inline">\(\theta\)</span> is the angle between two vectors; in this case, the two vectors are the two aggregate spectra.
We repeat this process for every informative marker <span class="math inline">\(g_n\)</span>.
<strong>c)</strong> To assess the significance of any distance peaks in b), we perform permutation tests.
In each of <span class="math inline">\(N\)</span> permutations, we shuffle the haplotype labels associated with the <em>de novo</em> mutation data, run a genome-wide distance scan, and record the maximum cosine distance encountered at any locus in the scan.
Finally, we calculate the <span class="math inline">\(1 - p\)</span> percentile of the distribution of those maximum distances to obtain a genome-wide cosine distance threshold at the specified value of <span class="math inline">\(p\)</span>.</figcaption>
</figure>
</div>
<h3 id="re-identifying-a-mutator-allele-on-chromosome-4-in-the-bxds">Re-identifying a mutator allele on chromosome 4 in the BXDs</h3>
<p>We applied our aggregate mutation spectrum distance method to 117 BXDs (<em>Materials and Methods</em>) with a total of 65,552 <em>de novo</em> germline mutations <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.</p>
<blockquote>
<p>Using mutation data that were partitioned by 1-mer nucleotide context, we discovered a locus on chromosome 4 that was significantly associated with mutation spectrum variation (Figure <a href="#fig:distance-results">2</a>a; maximum adjusted cosine distance of 8.57e-3 at marker ID <code>rs27509845</code>; position 118.28 Mbp in GRCm38/mm10 coordinates; 90% bootstrap confidence interval from 109.87 - 118.28 Mbp).</p>
</blockquote>
<div id="fig:distance-results" class="fignos">
<figure>
<img src="images/fig-distance-results.png" style="width:7.5in" alt="Figure 2: Results of aggregate mutation spectrum distance scans in the BXDs. a) Adjusted cosine distances between aggregate 1-mer de novo mutation spectra on BXD haplotypes (n = 117 haplotypes; 65,552 total mutations) with either D or B alleles at 6,987 informative markers. Cosine distance threshold at p = 0.05 was calculated by performing 10,000 permutations of the BXD mutation data, and is shown as a dotted grey line. b) Adjusted cosine distances between aggregate 1-mer de novo mutation spectra on BXD haplotypes with D alleles at rs27509845 (n = 66 haplotypes; 42,171 total mutations) and either D or B alleles at 6,847 informative markers. Cosine distance threshold at p = 0.05 was calculated by performing 10,000 permutations of the BXD mutation data, and is shown as a dotted grey line. c) Adjusted cosine distances between aggregate 1-mer de novo mutation spectra on BXD haplotypes with B alleles at rs27509845 (n = 44 haplotypes; 22,645 total mutations) and either D or B alleles at 6,953 informative markers. Cosine distance threshold at p = 0.05 was calculated by performing 10,000 permutations of the BXD mutation data, and is shown as a dotted grey line." />
<figcaption aria-hidden="true"><span>Figure 2:</span> <strong>Results of aggregate mutation spectrum distance scans in the BXDs.</strong>
<strong>a)</strong> Adjusted cosine distances between aggregate 1-mer <em>de novo</em> mutation spectra on BXD haplotypes (n = 117 haplotypes; 65,552 total mutations) with either <em>D</em> or <em>B</em> alleles at 6,987 informative markers.
Cosine distance threshold at p = 0.05 was calculated by performing 10,000 permutations of the BXD mutation data, and is shown as a dotted grey line.
<strong>b)</strong> Adjusted cosine distances between aggregate 1-mer <em>de novo</em> mutation spectra on BXD haplotypes with <em>D</em> alleles at <code>rs27509845</code> (n = 66 haplotypes; 42,171 total mutations) and either <em>D</em> or <em>B</em> alleles at 6,847 informative markers.
Cosine distance threshold at p = 0.05 was calculated by performing 10,000 permutations of the BXD mutation data, and is shown as a dotted grey line.
<strong>c)</strong> Adjusted cosine distances between aggregate 1-mer <em>de novo</em> mutation spectra on BXD haplotypes with <em>B</em> alleles at <code>rs27509845</code> (n = 44 haplotypes; 22,645 total mutations) and either <em>D</em> or <em>B</em> alleles at 6,953 informative markers.
Cosine distance threshold at p = 0.05 was calculated by performing 10,000 permutations of the BXD mutation data, and is shown as a dotted grey line.</figcaption>
</figure>
</div>
<p>Using quantitative trait locus (QTL) mapping, we previously identified a nearly-identical locus on chromosome 4 that was significantly associated with the C&gt;A germline mutation rate in the BXDs <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.
This locus overlapped 21 protein-coding genes that were annotated by the Gene Ontology as being involved in “DNA repair,” but only one of those genes contained nonsynonymous differences between the two parental strains: <em>Mutyh</em>.
<em>Mutyh</em> encodes a protein involved in the base-excision repair of 8-oxoguanine (8-oxoG), a DNA lesion caused by oxidative damage, and prevents the accumulation of C&gt;A mutations <span class="citation" data-cites="PMID:28551381 PMID:28127763 PMID:17581577">[<a href="#ref-PMID:28551381" role="doc-biblioref"><strong>PMID:28551381?</strong></a>,<a href="#ref-PMID:28127763" role="doc-biblioref"><strong>PMID:28127763?</strong></a>,<a href="#ref-PMID:17581577" role="doc-biblioref"><strong>PMID:17581577?</strong></a>]</span>.
C&gt;A germline mutation fractions are nearly 50% higher in BXDs that inherit <em>D</em> genotypes at marker ID <code>rs27509845</code> (the marker at which we observed the highest adjusted cosine distance on chromosome 4) than in those that inherit <em>B</em> genotypes (Figure <a href="#fig:spectra-comparison">3</a>) <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.</p>
<h3 id="an-additional-germline-mutator-allele-on-chromosome-6">An additional germline mutator allele on chromosome 6</h3>
<p>After confirming that AMSD could recover the mutator locus overlapping <em>Mutyh</em>, we asked if our approach could identify additional mutator loci in the BXDs.
In particular, we were interested in discovering epistatic interactions between alleles at the chromosome 4 locus and mutator alleles elsewhere in the genome.
We hypothesized that such interactions could be detectable by first “conditioning” on the presence of <em>B</em> or <em>D</em> alleles at the mutator locus on chromosome 4, and then running another genome-wide scan for loci associated with mutation spectrum variation.
To account for the effects of the large-effect mutator locus near <em>Mutyh</em>, we divided the BXDs into those with either <em>D</em> (n = 66) or <em>B</em> (n = 44) genotypes at <code>rs27509845</code> (n = 7 BXDs were heterozygous) and ran an aggregate mutation spectrum distance scan using each group separately (Figure <a href="#fig:distance-results">2</a>b-c).</p>
<blockquote>
<p>Using the BXDs with <em>D</em> genotypes at <code>rs27509845</code>, we identified a locus on chromosome 6 that was significantly associated with mutation spectrum variation (Figure <a href="#fig:distance-results">2</a>b; maximum adjusted cosine distance of 2.67e-3 at marker <code>rs46276051</code>; position 111.27 Mbp in GRCm38/mm10 coordinates; 90% bootstrap confidence interval from 95.01 - 114.02 Mbp).
This signal was specific to BXDs with <em>D</em> genotypes at the <code>rs27509845</code> locus, as we did not observe any new mutator loci after performing an AMSD scan using BXDs with <em>B</em> genotypes at <code>rs27509845</code> (Figure <a href="#fig:distance-results">2</a>c), and the peak markers on chromosome 4 and 6 did not exhibit strong linkage disequilibrium (<span class="math inline">\(R^2\)</span> = 4e-5).</p>
</blockquote>
<p>We also performed QTL scans for the fractions of each 1-mer mutation type using the same mutation data, but none produced a genome-wide significant log-odds score at any locus (Figure <a href="#fig:qtl-scans">2-figure supplement 1</a>; <em>Materials and Methods</em>).</p>
<blockquote>
<p>We queried the region surrounding the top marker on chromosome 6 (+/- the 90% bootstrap confidence interval) and discovered 64 protein-coding genes, of which five were annotated with a Gene Ontology <span class="citation" data-cites="PMID:10802651 PMID:33290552">[<a href="#ref-PMID:10802651" role="doc-biblioref"><strong>PMID:10802651?</strong></a>,<a href="#ref-PMID:33290552" role="doc-biblioref"><strong>PMID:33290552?</strong></a>]</span> term related to “DNA repair”: <em>Ddx11</em>, <em>Fancd2</em>, <em>Ogg1</em>, <em>Setmar</em>, and <em>Rad18</em>.
Of these, two harbored nonsynonymous differences between the parental C57BL/6J and DBA/2J strains (Table <a href="#tbl:nonsyn-diffs">1</a>).
<em>Ogg1</em> encodes a key member of the base-excision repair response to oxidative DNA damage (a pathway that also includes <em>Mutyh</em>), and in mice <em>Setmar</em> encodes a SET domain-containing histone methyltransferase; both <em>Ogg1</em> and <em>Setmar</em> are expressed in mouse gonadal cells.</p>
</blockquote>
<div id="tbl:nonsyn-diffs" class="tablenos">
<table id="tbl:nonsyn-diffs" style="width:100%;">
<caption><span>Table 1:</span> Nonsynonymous mutations in DNA repair genes near the chr6 peak </caption>
<colgroup>
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
<col style="width: 14%" />
</colgroup>
<thead>
<tr class="header">
<th>Gene name</th>
<th>Ensembl transcript name</th>
<th>Nucleotide change</th>
<th>Amino acid change</th>
<th>Position in GRCm38/mm10 coordinates</th>
<th>PhyloP conservation score</th>
<th>SIFT prediction</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><em>Setmar</em></td>
<td>ENSMUST00000049246</td>
<td>C&gt;T</td>
<td>p.Leu103Phe</td>
<td>chr6:108,075,853</td>
<td>0.422</td>
<td>0.0 (intolerant/deleterious)</td>
</tr>
<tr class="even">
<td><em>Setmar</em></td>
<td>ENSMUST00000049246</td>
<td>T&gt;G</td>
<td>p.Ser273Arg</td>
<td>chr6:108,076,365</td>
<td>-0.355</td>
<td>0.3 (tolerant/benign)</td>
</tr>
<tr class="odd">
<td><em>Ogg1</em></td>
<td>ENSMUST00000032406</td>
<td>A&gt;G</td>
<td>p.Thr95Ala</td>
<td>chr6:113,328,510</td>
<td>-0.016</td>
<td>0.84 (tolerant/benign)</td>
</tr>
</tbody>
</table>
</div>
<p>We also considered the possibility that expression quantitative trait loci (eQTLs), rather than nonsynonymous mutations, could contribute to the C&gt;A mutator phenotype associated with the locus on chromosome 6.
Using GeneNetwork <span class="citation" data-cites="PMID:27933521">[<a href="#ref-PMID:27933521" role="doc-biblioref"><strong>PMID:27933521?</strong></a>]</span> we mapped eQTLs for the five aforementioned DNA repair genes in a number of tissues, though we did not have access to expression data from germline cells.</p>
<blockquote>
<p>Notably, <em>D</em> alleles near the cosine distance peak on chromosome 6 were significantly associated with decreased <em>Ogg1</em> expression in kidney, liver, hippocampus, and gastrointestinal tissues (Table <a href="#tbl:eqtl-results">supplement 1</a>).
Although these cis-eQTLs are challenging to interpret (given their tissue specificity and our lack of access to germline expression data), the presence of strong-effect cis-eQTLs for <em>Ogg1</em> suggests that the C&gt;A mutator phenotype observed in the BXDs may be mediated by regulatory, rather than protein-altering, variants.</p>
</blockquote>
<blockquote>
<p>Finally, we queried a dataset of structural variants (SVs) identified via high-quality, long-read assembly of inbred laboratory mouse strains <span class="citation" data-cites="8KLgRyLH">[<a href="#ref-8KLgRyLH" role="doc-biblioref">3</a>]</span> and found X large insertions or deletions within the 90% bootstrap confidence interval around the cosine distance peak on chromosome 6.
Of these, X overlapped the exonic sequences of protein-coding genes (Table <a href="#tbl:sv-overlap">supplement 2</a>), though none of the genes has a previously annotated role in DNA repair or replication, or in a pathway that would likely affect germline mutation rates.</p>
</blockquote>
<p>One protein-coding gene that is involved in DNA repair (<em>Rad18</em>) harbored intronic insertions and deletions (Table <a href="#tbl:sv-overlap">supplement 2</a>); however, additional experimental evidence will be needed to probe the functional impact of these SVs.</p>
<h3 id="evidence-of-epistasis-between-germline-mutator-alleles">Evidence of epistasis between germline mutator alleles</h3>
<p>Next, we more precisely characterized the effects of the chromosome 4 and 6 mutator alleles on mutation spectra in the BXDs.
To pinpoint the mutation type(s) that underlied the significant cosine distance peak on chromosome 6, we compared the aggregate counts of each 1-mer mutation type (plus CpG&gt;TpG) on BXD haplotypes with <em>D</em> genotypes at <code>rs27509845</code> and either <em>D</em> or <em>B</em> genotypes at <code>rs46276051</code>.</p>
<blockquote>
<p>We found that C&gt;A mutations were significantly enriched on BXD haplotypes with <em>D</em> genotypes at the chromosome 6 mutator locus, relative to those with <em>B</em> genotypes (<span class="math inline">\(\chi^2\)</span> statistic = 85.36, p = 2.48e-20).</p>
</blockquote>
<p>On average, C&gt;A germline mutation fractions were significantly higher in BXDs with <em>D</em> alleles at both mutator loci than in BXDs with <em>D</em> alleles at either locus alone (Figure <a href="#fig:spectra-comparison">3</a>a and <a href="#fig:spectra-comparison-all">3-figure supplement 1</a>).
Among BXDs with <em>B</em> alleles at the locus overlapping <em>Mutyh</em>, those with <em>D</em> alleles on chromosome 6 did not exhibit significantly elevated C&gt;A mutation fractions (Figure <a href="#fig:spectra-comparison">3</a>a).
After controlling for inbreeding duration, we observed that C&gt;A <em>de novo</em> mutation counts were always highest in BXDs with <em>D</em> alleles at both mutator loci (Figure <a href="#fig:spectra-comparison">3</a>b).
After 100 generations of inbreeding, BXDs with <em>D</em> alleles at both mutator loci were predicted to have 238.9 C&gt;A mutations (95% CI: 231.4 - 246.4), about 20% more than the 199.0 mutations (95% CI: 193.3 - 204.7) predicted in those with <em>D</em> and <em>B</em> alleles at the chromosome 4 and chromosome 6 loci, respectively (Figure <a href="#fig:spectra-comparison">3</a>b).</p>
<p>We also used SigProfilerExtractor <span class="citation" data-cites="PMID:36388765">[<a href="#ref-PMID:36388765" role="doc-biblioref"><strong>PMID:36388765?</strong></a>]</span> to assign the germline mutations in each BXD to single-base substitution (SBS) mutation signatures from the COSMIC catalog <span class="citation" data-cites="PMID:30371878">[<a href="#ref-PMID:30371878" role="doc-biblioref"><strong>PMID:30371878?</strong></a>]</span>.
Mutation signatures often reflect specific exogenous or endogenous sources of DNA damage, and the proportions of mutations attributable to particular SBS signatures can suggest a genetic or environmental etiology.
The SBS1, SBS5, and SBS30 mutation signatures were active in nearly all BXDs, regardless of genotypes at the chromosome 4 and 6 mutator loci (Figure <a href="#fig:spectra-comparison">3</a>c).
However, the SBS18 signature, which is dominated by C&gt;A mutations and likely reflects unrepaired DNA damage from reactive oxygen species, was almost exclusively active in mice with <em>D</em> alleles at the chromosome 4 locus; the highest SBS18 activity was observed in mice with <em>D</em> alleles at both mutator loci (Figure <a href="#fig:spectra-comparison">3</a>c).
SBS18 activity was lowest in mice with <em>D</em> alleles at the chromosome 6 mutator locus alone (Figure <a href="#fig:spectra-comparison">3</a>c), further demonstrating that <em>D</em> alleles at this locus are not sufficient to cause a mutator phenotype.</p>
<p>To more formally test for statistical epistasis, we fit a generalized (Poisson) linear model predicting counts of C&gt;A mutations in each BXD as a function of genotypes at <code>rs27509845</code> and <code>rs46276051</code> (the markers with the largest adjusted cosine distance at the two mutator loci); the model also accounted for differences in inbreeding duration and sequencing coverage between the BXDs (<em>Materials and Methods</em>).</p>
<blockquote>
<p>A model that included an interaction term between genotypes at the two markers fit the data significantly better than a model including only additive effects (p = 7.92e-7; <em>Materials and Methods</em>), indicating that the combined effects of <em>D</em> genotypes at both loci exceeded the sum of marginal effects of <em>D</em> genotypes at either locus alone.</p>
</blockquote>
<div id="fig:spectra-comparison" class="fignos">
<figure>
<img src="images/fig-spectra-comparison.png" style="width:7.5in" alt="Figure 3: BXD mutation spectra are affected by alleles at both mutator loci. a) C&gt;A de novo germline mutation fractions in BXDs with either D or B genotypes at markers rs27509845 (chr4 peak) and rs46276051 (chr6 peak). Distributions of C&gt;A mutation fractions were compared with two-sided Mann-Whitney U-tests; annotated p-values are uncorrected. B-B vs. B-D comparison: U-statistic = 149.0, p = 7.58e-2; B-D vs D-D comparison: U-statistic = 21.0, p = 2.61e-8; D-B vs D-D comparison: U-statistic = 232.5, p = 6.99e-5. b) The count of C&gt;A de novo germline mutations in each BXD was plotted against the number of generations for which it was inbred. Lines represent predicted C&gt;A counts in each haplotype group from a generalized linear model (Poisson family, identity link), and shading around each line represents the 95% confidence interval. c) Germline mutations in each BXD were assigned to COSMIC SBS mutation signatures using SigProfilerExtractor [PMID:36388765?]. After grouping BXDs by their genotypes at rs27509845 and rs46276051, we calculated the fraction of mutations in each group that was attributed to each signature. The proposed etiologies of each mutation signature are: SBS1 (spontaneous deamination of methylated cytosine nucleotides at CpG contexts), SBS5 (unknown, clock-like signature), SBS18 (damage by reactive oxygen species, related to SBS36 and defective base-excision repair due to loss-of-function mutations in MUTYH), and SBS30 (defective base-excision repair due to NTHL1 mutations)." />
<figcaption aria-hidden="true"><span>Figure 3:</span> <strong>BXD mutation spectra are affected by alleles at both mutator loci.</strong>
<strong>a)</strong> C&gt;A <em>de novo</em> germline mutation fractions in BXDs with either <em>D</em> or <em>B</em> genotypes at markers <code>rs27509845</code> (chr4 peak) and <code>rs46276051</code> (chr6 peak). Distributions of C&gt;A mutation fractions were compared with two-sided Mann-Whitney U-tests; annotated p-values are uncorrected. <code>B-B</code> vs. <code>B-D</code> comparison: U-statistic = 149.0, p = 7.58e-2; <code>B-D</code> vs <code>D-D</code> comparison: U-statistic = 21.0, p = 2.61e-8; <code>D-B</code> vs <code>D-D</code> comparison: U-statistic = 232.5, p = 6.99e-5.
<strong>b)</strong> The count of C&gt;A <em>de novo</em> germline mutations in each BXD was plotted against the number of generations for which it was inbred.
Lines represent predicted C&gt;A counts in each haplotype group from a generalized linear model (Poisson family, identity link), and shading around each line represents the 95% confidence interval.
<strong>c)</strong> Germline mutations in each BXD were assigned to COSMIC SBS mutation signatures using SigProfilerExtractor <span class="citation" data-cites="PMID:36388765">[<a href="#ref-PMID:36388765" role="doc-biblioref"><strong>PMID:36388765?</strong></a>]</span>. After grouping BXDs by their genotypes at <code>rs27509845</code> and <code>rs46276051</code>, we calculated the fraction of mutations in each group that was attributed to each signature. The proposed etiologies of each mutation signature are: SBS1 (spontaneous deamination of methylated cytosine nucleotides at CpG contexts), SBS5 (unknown, clock-like signature), SBS18 (damage by reactive oxygen species, related to SBS36 and defective base-excision repair due to loss-of-function mutations in MUTYH), and SBS30 (defective base-excision repair due to NTHL1 mutations).</figcaption>
</figure>
</div>
<p>To explore the effects of the two mutator loci in other inbred laboratory mice, we also compared the germline mutation spectra of Sanger Mouse Genomes Project (MGP) strains <span class="citation" data-cites="PMID:21921910">[<a href="#ref-PMID:21921910" role="doc-biblioref"><strong>PMID:21921910?</strong></a>]</span>.
Dumont <span class="citation" data-cites="PMID:30753674">[<a href="#ref-PMID:30753674" role="doc-biblioref"><strong>PMID:30753674?</strong></a>]</span> previously identified germline mutations that were private to each of the 29 MGP strains; these private variants likely represent recent <em>de novo</em> mutations (Figure <a href="#fig:spectra-comparison-mgp">3—figure supplement 2</a>).
Only two of the MGP strains possess <em>D</em> genotypes at both the chromosome 4 and chromosome 6 mutator loci: DBA/1J and DBA/2J.
As before, we tested for epistasis in the MGP strains by fitting two linear models predicting C&gt;A mutation counts as a function of genotypes at the two mutator loci.
A model incorporating an interaction term did not fit the MGP data significantly better than a model with additive effects alone (p = 0.806), so we are unable to confirm the signal of epistasis; however, this may be due to the smaller number of MGP strains with <em>de novo</em> germline mutation data.</p>
<h3 id="some-of-the-candidate-mutator-alleles-are-segregating-in-wild-mice">Some of the candidate mutator alleles are segregating in wild mice</h3>
<p>To determine whether the candidate mutator alleles on chromosome 6 were segregating in natural populations, we queried previously published sequencing data generated from 67 wild-derived mice <span class="citation" data-cites="PMID:27622383">[<a href="#ref-PMID:27622383" role="doc-biblioref"><strong>PMID:27622383?</strong></a>]</span>.
These data include three subspecies of <em>Mus musculus</em>, as well as the outgroup <em>Mus spretus</em>.
We found that the <em>Ogg1</em> <em>D</em> allele was segregating at an allele frequency of 0.259 in <em>Mus musculus domesticus</em>, the species from which C57BL/6J and DBA/2J derive the majority of their genomes <span class="citation" data-cites="PMID:17660819">[<a href="#ref-PMID:17660819" role="doc-biblioref"><strong>PMID:17660819?</strong></a>]</span>, and was fixed in <em>Mus musculus musculus</em>, <em>Mus musculus castaneus</em>, and the outgroup <em>Mus spretus</em> (Figure <a href="#fig:wild-afs">3-figure supplement 3</a>).
The <em>Setmar</em> p.Ser273Arg <em>D</em> allele was also present at an allele frequency of 0.37 in <em>Mus musculus domesticus</em>, while <em>D</em> alleles at the <em>Setmar</em> p.Leu103Phe variant were not observed in any wild <em>Mus musculus domesticus</em> animals.</p>
<h2 id="discussion">Discussion</h2>
<h3 id="epistasis-between-germline-mutator-alleles">Epistasis between germline mutator alleles</h3>
<p>We have identified a locus on chromosome 6 that amplifies a C&gt;A germline mutator phenotype in the BXDs, a family of inbred mice derived from the laboratory strains DBA/2J and C57BL/6J.
DBA/2J (<em>D</em>) alleles at this locus have no significant effect on C&gt;A mutation rates in mice that also harbor “wild-type” C57BL/6J (<em>B</em>) alleles at a previously discovered mutator locus on chromosome 4 <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.
However, mice with <em>D</em> alleles at <em>both</em> loci have even higher mutation rates than those with <em>D</em> alleles at the chromosome 4 mutator locus alone (Figure <a href="#fig:spectra-comparison">3</a>).
Epistatic interactions between mutator alleles have been previously documented in yeast <span class="citation" data-cites="PMID:16492773">[<a href="#ref-PMID:16492773" role="doc-biblioref"><strong>PMID:16492773?</strong></a>]</span> and in human cell lines <span class="citation" data-cites="PMID:35859169">[<a href="#ref-PMID:35859169" role="doc-biblioref"><strong>PMID:35859169?</strong></a>]</span>, but never to our knowledge in a whole-animal context.</p>
<p>Importantly, we discovered epistasis between germline mutator alleles in an unnatural population of model organisms that have been inbred by brother-sister mating in a highly controlled laboratory environment <span class="citation" data-cites="PMID:33472028">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>.
This breeding setup has likely attenuated the effects of natural selection on all but the most deleterious alleles <span class="citation" data-cites="LF8DCRcP">[<a href="#ref-LF8DCRcP" role="doc-biblioref">2</a>]</span>, and may have facilitated the fixation of large-effect mutator alleles that would be less common in wild, outbreeding populations.</p>
<blockquote>
<p>We have not conclusively fine-mapped the chromosome 6 mutator locus to a causal variant, but we argue that nonsynonymous or regulatory variants in the DNA glycosylase <em>Ogg1</em> are the best candidates.</p>
</blockquote>
<!-- Our results demonstrate that multiple mutator alleles have spontaneously arisen during the evolutionary history of inbred laboratory mice, supporting the hypothesis that purifying selection is required to keep mutation rates low.  -->
<p>Our finding suggests that mutational pressure can cause mutation rates to rise in just a few generations of relaxed selection in captivity.
This hypothesis is corroborated by the recent discovery of a large-effect mutator allele in a rhesus macaque research colony <span class="citation" data-cites="12CXVVJzy">[<a href="#ref-12CXVVJzy" role="doc-biblioref">1</a>]</span>, as well as the observation that domesticated animals tend to have higher mutation rates than those in the wild <span class="citation" data-cites="PMID:36859541">[<a href="#ref-PMID:36859541" role="doc-biblioref"><strong>PMID:36859541?</strong></a>]</span>.</p>
<h3 id="protein-coding-genes-that-may-underlie-the-chromosome-6-mutator-locus">Protein-coding genes that may underlie the chromosome 6 mutator locus</h3>
<blockquote>
<p>Five protein-coding genes involved in DNA repair overlap the C&gt;A mutator locus on chromosome 6: <em>Ogg1</em>, a glycosylase that excises the oxidative DNA lesion 8-oxoguanine (8-oxoG) <span class="citation" data-cites="PMID:17581577">[<a href="#ref-PMID:17581577" role="doc-biblioref"><strong>PMID:17581577?</strong></a>]</span>, <em>Setmar</em>, a histone methyltransferase involved in non-homologous end joining (NHEJ) of double-stranded breaks (DSBs) <span class="citation" data-cites="PMID:21187428 PMID:16332963">[<a href="#ref-PMID:21187428" role="doc-biblioref"><strong>PMID:21187428?</strong></a>,<a href="#ref-PMID:16332963" role="doc-biblioref"><strong>PMID:16332963?</strong></a>]</span>, <em>Fancd2</em>, <em>Rad18</em>, and <em>Ddx11</em>.</p>
</blockquote>
<p>Although we are unable to conclusively determine that one or more of these genes harbors a causal variant underlying the observed C&gt;A mutator phenotype, we believe that <em>Ogg1</em> is the most plausible candidate.</p>
<blockquote>
<p><em>Ogg1</em> is a member of the same base-excision repair pathway as <em>Mutyh</em> (the gene that likely underlies the chromosome 4 mutator locus), contains a nonsynonymous fixed difference between the C57BL/6J and DBA/2J parental strains, and appears to be regulated by cis-eQTLs across a number of tissues within the BXD cohort.</p>
</blockquote>
<p>The C57BL/6J and DBA/2J <em>Setmar</em> coding sequences differ by two missense variants (Table <a href="#tbl:nonsyn-diffs">1</a>), one of which is predicted to be deleterious by <em>in silico</em> tools.
The primate <em>SETMAR</em> ortholog is involved in NHEJ of double-strand breaks, but its role in DNA repair appears to depend on the function of both a SET methyltransferase domain and a <em>Mariner</em>-family transposase domain <span class="citation" data-cites="PMID:16332963 PMID:24573677 PMID:21491884">[<a href="#ref-PMID:16332963" role="doc-biblioref"><strong>PMID:16332963?</strong></a>,<a href="#ref-PMID:24573677" role="doc-biblioref"><strong>PMID:24573677?</strong></a>,<a href="#ref-PMID:21491884" role="doc-biblioref"><strong>PMID:21491884?</strong></a>]</span>.</p>
<blockquote>
<p>Since the murine <em>Setmar</em> ortholog lacks the latter element, and because primate <em>SETMAR</em> is involved in a DNA repair process that is not expected to affect the rate of C&gt;A mutations, we believe it is unlikely to underlie the epistatic interaction between the chromosome 4 and 6 mutator loci in the BXDs (<em>Supplementary Information</em>).</p>
</blockquote>
<p>Moreover, we did not observe any significant cis-eQTLs for <em>Setmar</em> across a variety of tissues in the BXD cohort (Table <a href="#tbl:eqtl-results">supplement 1</a>).</p>
<blockquote>
<p>None of the remaining DNA repair genes (<em>Fancd2</em>, <em>Ddx11</em>, or <em>Rad18</em>) contains a nonsynonymous fixed difference between the C57BL/6J and DBA/2J parental strains, and none appear to be regulated by cis-eQTLs that would feasibly lead to a germline C&gt;A mutator phenotype (Table <a href="#tbl:eqtl-results">supplement 1</a>); the only significant cis-eQTL we observed was for <em>Fancd2</em> in gastrointestinal tissue, at which <em>D</em> alleles actually led to <em>increased</em> expression.</p>
</blockquote>
<!-- We also note that *FANCD2* deficiency in human cancer can lead to increased rates of *de novo* structural variation [@PMID:36450981]. -->
<h3 id="an-ogg1-mutator-allele-might-impair-the-excision-of-8-oxoguanine-lesions">An <em>Ogg1</em> mutator allele might impair the excision of 8-oxoguanine lesions</h3>
<p><em>Ogg1</em> is a member of the same base-excision repair (BER) pathway as <em>Mutyh</em>, the protein-coding gene we previously implicated as harboring mutator alleles at the locus on chromosome 4 <span class="citation" data-cites="PMID:17581577">[<a href="#ref-PMID:17581577" role="doc-biblioref"><strong>PMID:17581577?</strong></a>]</span>.
Each of these genes has a distinct role in the BER response to oxidative DNA damage, and thereby the prevention of C&gt;A mutations <span class="citation" data-cites="PMID:28963982 PMID:24732879">[<a href="#ref-PMID:28963982" role="doc-biblioref"><strong>PMID:28963982?</strong></a>,<a href="#ref-PMID:24732879" role="doc-biblioref"><strong>PMID:24732879?</strong></a>]</span>.
Following damage by reactive oxygen species, <em>Ogg1</em> is able to recognize and remove 8-oxoguanine lesions that are base-paired with cytosine nucleotides; once 8-oxoG is excised, other members of the BER pathway are mobilized to restore a proper G:C base pair at the site.
If an 8-oxoG lesion is not removed before the cell enters S-phase, adenine can be mis-incorporated opposite 8-oxoG during DNA replication <span class="citation" data-cites="PMID:28963982">[<a href="#ref-PMID:28963982" role="doc-biblioref"><strong>PMID:28963982?</strong></a>]</span>.
If this occurs, <em>Mutyh</em> can excise the mispaired adenine, leaving a one-nucleotide gap that is processed and filled with a cytosine by other BER proteins.
The resulting C:8-oxoG base pair can then be “returned” to <em>Ogg1</em> for excision and repair.
Defects in the BER response to oxidative damage lead to significantly elevated rates of C&gt;A mutation.
For example, triple-knockout (KO) mice lacking <em>Ogg1</em>, <em>Mutyh</em>, and <em>Mth1</em> (which encodes an enzyme that prevents 8-oxo-dGTP from being incorporated during DNA synthesis <span class="citation" data-cites="PMID:8226881">[<a href="#ref-PMID:8226881" role="doc-biblioref"><strong>PMID:8226881?</strong></a>]</span>) accumulate a 100-fold excess of 8-oxoG in their gonadal cells <span class="citation" data-cites="PMID:24732879">[<a href="#ref-PMID:24732879" role="doc-biblioref"><strong>PMID:24732879?</strong></a>]</span>.
Almost 99% of <em>de novo</em> germline mutations in the <em>Ogg1/Mutyh/Mth1</em> triple KO mice are C&gt;A transversions, demonstrating the clear role of 8-oxoG repair in preventing C&gt;A mutation.
Additionally, missense mutations and loss-of-heterozygosity in <em>Ogg1</em> have been associated with increased risk of human cancer <span class="citation" data-cites="PMID:22829015 PMID:9662341">[<a href="#ref-PMID:22829015" role="doc-biblioref"><strong>PMID:22829015?</strong></a>,<a href="#ref-PMID:9662341" role="doc-biblioref"><strong>PMID:9662341?</strong></a>]</span>, and copy-number losses of either <em>Ogg1</em> or <em>Mutyh</em> are linked to elevated rates of spontaneous C&gt;A mutation in human neuroblastoma <span class="citation" data-cites="8vP4Q1ja">[<a href="#ref-8vP4Q1ja" role="doc-biblioref">4</a>]</span>.</p>
<h4 id="nonsynonymous-mutations-may-underlie-the-chromosome-6-mutator-phenotype">Nonsynonymous mutations may underlie the chromosome 6 mutator phenotype</h4>
<p>The p.Thr95Ala <em>Ogg1</em> missense variant is not predicted to be deleterious by the <em>in silico</em> tool SIFT <span class="citation" data-cites="PMID:12824425">[<a href="#ref-PMID:12824425" role="doc-biblioref"><strong>PMID:12824425?</strong></a>]</span>, and occurs at a nucleotide that is not particularly well-conserved across mammalian species (Table <a href="#tbl:nonsyn-diffs">1</a>).
We also observe that the <em>D</em> allele at p.Thr95Ala is segregating at an allele frequency of approximately 26% among wild-derived <em>Mus musculus domesticus</em> animals, and is fixed in other wild populations of <em>Mus musculus musculus</em>, <em>Mus musculus castaneus</em>, and <em>Mus spretus</em> .
Although we would expect <em>a priori</em> that <em>Ogg1</em> deficiency should lead to increased 8-oxoG accumulation and elevated C&gt;A mutation rates, these lines of evidence suggest that p.Thr95Ala is not highly deleterious on its own, and might only exert a detectable effect on the BER gene network when <em>Mutyh</em> function is also impaired.
It is also possible that <em>D</em> alleles at <em>Ogg1</em> lead to a very subtle increase in C&gt;A mutation rates, and we are simply underpowered to detect such a small mutation rate effect in the BXDs.</p>
<!-- Overall, it is challenging to predict the functional consequences of the *Ogg1* p.Thr95Ala variant, and as we discuss below, the p.Asp129Asn missense mutation in *Mbd4* may be an even more compelling candidate mutator allele. -->
<!-- #### *Mbd4* may buffer the effects of *Mutyh* mutator alleles by triggering apoptosis

Unlike the *Ogg1* p.Thr95Ala mutation, the p.Asp129Asn variant in *Mbd4* resides within an annotated protein domain (the *Mbd4* methyl-CpG binding domain), occurs at a nucleotide and amino acid residue that are both well-conserved, and is predicted to be deleterious by SIFT [@PMID:12824425] (Table @tbl:nonsyn-diffs). 
A missense mutation that affects the homologous amino acid in humans (p.Asp142Gly in GRCh38/hg38) is also present on a single haplotype in the Genome Aggregation Database (gnomAD) [@PMID:32461654] and is predicted by SIFT and Polyphen [@PMID:20354512] to be "deleterious" and "probably_damaging" in human genomes, respectively. 

One puzzling observation is that loss-of-function mutations in *Mbd4* are not typically associated with C>A mutator phenotypes. 
Instead, *Mbd4* deficiency is usually implicated in C>T mutagenesis at CpG sites, and we did not detect an excess of C>T mutations in BXDs with *D* alleles at the chromosome 6 mutator locus (Figure @fig:spectra-comparison-all).
However, loss of function mutations in *Mbd4* have also been shown to exacerbate the effects of exogenous DNA damage agents.
For example, mouse embryonic fibroblasts that harbor homozygous *Mbd4* knockouts fail to undergo apoptosis following treatment with a number of chemotherapeutics and mutagenic compounds [@PMID:14614141]. 
Most of these exogenous mutagens cause DNA damage that is normally repaired by mismatch repair (MMR) machinery, but murine intestinal cells with biallelic *Mbd4* LOF mutations also showed a reduced apoptotic response to gamma irradiation, which is repaired independently of the MMR gene *Mlh1* [@PMID:14562041]. 
Homozygous loss of *Mbd4* function also leads to accelerated intestinal tumor formation in mice that harbor an *Apc* allele that predisposes them to intestinal neoplasia [@PMID:12130785], and mice with biallelic truncations of the *Mbd4* coding sequence exhibit modestly increased mutation rates in colon cancer cell lines, including increased C>A mutation rates in certain lines [@PMID:17285135].

Together, these lines of evidence suggest that *Mbd4* can modulate sensitivity to many types of exogenous mutagens, potentially through its role in determining whether cells harboring DNA damage should undergo apoptosis [@PMID:14614141;@PMID:14562041]. 
We speculate that in mice with deficient 8-oxoguanine repair &mdash; caused by a mutator allele in *Mutyh*, for example &mdash; reactive oxygen species (ROS) could cause accumulation of DNA damage in the germline. 
If those germ cells harbor fully functional copies of *Mbd4*, they might be able to trigger apoptosis and partially mitigate the effects of a *Mutyh* mutator allele.
However, mice with reduced activity of both *Mbd4* and *Mutyh* may have a reduced ability to initiate cell death in response to DNA damage; as a result, their germ cells may accumulate even higher levels of ROS-mediated damage, leading to substantially elevated germline C>A mutation rates. 

Our model makes the testable prediction that mice with *D* haplotypes at *Mbd4* should have elevated sensitivity to a variety of mutagenic agents. 
We thus anticipate that future experimental work will be able to more conclusively establish a mechanistic explanation for the epistatic interaction between mutator loci described in this paper.  -->
<h4 id="no-indication-of-causal-structural-variation-or-mobile-element-insertions-near-the-chromosome-6-mutator-locus">No indication of causal structural variation or mobile element insertions near the chromosome 6 mutator locus</h4>
<p>Although we argue above that <em>Ogg1</em> is likely the the best candidate gene to explain the new BXD C&gt;A mutator phenotype, we cannot conclusively determine that the p.Thr95Ala missense mutation is a causal allele.
We previously hypothesized that <em>Mutyh</em> missense mutations on <em>D</em> haplotypes were responsible for the large-effect C&gt;A mutator phenotype we observed in the BXDs <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.
However, subsequent long-read assemblies of several inbred laboratory mouse strains revealed that this mutator phenotype might be caused by a ~5 kbp mobile element insertion (MEI) within the first intron of <em>Mutyh</em> <span class="citation" data-cites="8KLgRyLH">[<a href="#ref-8KLgRyLH" role="doc-biblioref">3</a>]</span>, which is associated with significantly reduced expression of <em>Mutyh</em> in embryonic stem cells.
We queried the new high-quality assemblies for evidence of mobile elements or other large structural variants (SVs) in the region surrounding the mutator locus on chromosome 6, but found no similarly compelling evidence that either SVs or MEIs might underlie the mutator phenotype described in this study.</p>
<h4 id="expression-quantitative-trait-loci-eqtls-might-mediate-germline-mutator-phenotypes-in-the-bxds">Expression quantitative trait loci (eQTLs) might mediate germline mutator phenotypes in the BXDs</h4>
<p>We observed strong-effect cis-eQTLs for <em>Ogg1</em> expression across a number of tissues in the BXDs (Table <a href="#tbl:eqtl-results">supplement 1</a>).
In each of these tissue types, <em>D</em> genotypes were associated with decreased expression of <em>Ogg1</em>.</p>
<blockquote>
<p>As mentioned above, new evidence from long-read genome assemblies has demonstrated that an intronic mobile element insertion in <em>Mutyh</em> may be responsible for decreased <em>Mutyh</em> expression, and therefore higher C&gt;A mutation rates, in BXDs with <em>D</em> haplotypes at the chromosome 4 mutator locus <span class="citation" data-cites="8KLgRyLH">[<a href="#ref-8KLgRyLH" role="doc-biblioref">3</a>]</span>.</p>
</blockquote>
<blockquote>
<p>Taken together, these results raise the exciting possibility that the mutator loci on both chromosome 4 and chromosome 6 lead to increased C&gt;A mutation rates by lowering the expression of DNA repair genes in the same base-excision repair network.</p>
</blockquote>
<h3 id="strengths-and-limitations-of-the-aggregate-mutation-spectrum-distance-approach">Strengths and limitations of the aggregate mutation spectrum distance approach</h3>
<p>Our aggregate mutation spectrum distance (AMSD) approach was able to identify a mutator allele that escaped notice using quantitative trait locus (QTL) mapping.
To more systematically compare the power of AMSD and QTL mapping, we performed simulations under a variety of possible parameter regimes.
Overall, we found that AMSD and QTL mapping have similar power to detect mutator alleles on haplotypes that each harbor tens or hundreds of <em>de novo</em> germline mutations (Figure <a href="#fig:ihd_vs_qtl_power">1-figure supplement 2</a>).
Nonetheless, only AMSD was able to discover the mutator locus on chromosome 6 in the BXDs, demonstrating that it outperforms QTL mapping in certain experimental systems.
For example, simulations demonstrate that AMSD enjoys greater power than QTL mapping when haplotypes carry variable numbers of mutations that can be leveraged for mutator mapping (Figure <a href="#fig:ihd_vs_qtl_power_variable_counts">1-figure supplement 3</a>).
Because the BXDs were generated in six breeding epochs over a period of nearly 40 years, the oldest lines have accumulated orders of magnitude more mutations than the youngest lines; these younger BXDs have much noisier mutation spectra as a result.
While approaches for QTL mapping typically weight the phenotypic measurements of each sample equally, AMSD compares the <em>aggregate</em> mutation spectra of haplotypes at every locus, a property that likely increased its power to detect mutators in the BXD dataset.</p>
<p>Another benefit of the AMSD approach is that it obviates the need to perform separate association tests for every possible <span class="math inline">\(k\)</span>-mer mutation type, and therefore the need to adjust significance thresholds for multiple tests.
Since AMSD compares the complete mutation spectrum between haplotypes that carry either allele at a site, it would also be well-powered to detect a mutator allele that exerted a coordinated effect on multiple <span class="math inline">\(k\)</span>-mer mutation types (e.g., increased the rates of both C&gt;T and C&gt;A mutations).</p>
<!-- Finally, we find that IHD outperforms QTL mapping when simulated mutators are present at allele frequencies that substantially deviate from 50% ().
This feature of inter-haplotype distance may prove especially useful when mapping mutator alleles in natural, outbreeding populations, in which the effects of negative selection will likely keep mutators at relatively low population frequencies.  -->
<p>However, the AMSD method suffers a handful of drawbacks when compared to QTL mapping.
Popular QTL mapping methods (such as R/qtl2 <span class="citation" data-cites="PMID:30591514">[<a href="#ref-PMID:30591514" role="doc-biblioref"><strong>PMID:30591514?</strong></a>]</span>) use linear models to test associations between genotypes and phenotypes, enabling the inclusion of additive and interactive covariates, as well as kinship matrices, in QTL scans.
Although we have developed methods to account for inter-sample relatedness in the AMSD approach (<em>Materials and Methods</em>), they are not as flexible as similar methods in QTL mapping software.
Additionally, the AMSD method assumes that mutator alleles affect a subset of <span class="math inline">\(k\)</span>-mer mutation types; if a mutator allele increased the rates of all mutation types equally on haplotypes that carried it, AMSD would be unable to detect it.</p>
<h3 id="discovering-mutator-alleles-in-other-experimental-systems">Discovering mutator alleles in other experimental systems</h3>
<p>Our discovery of a second BXD mutator allele underscores the power of recombinant inbred lines (RILs) as a resource for dissecting the genetic architecture of germline mutation rates.
Large populations of RILs exist for many model organisms, and we anticipate that as whole-genome sequencing becomes cheaper and cheaper, the AMSD method could be useful for future mutator allele discovery outside of the BXDs.
At the same time, RILs are a finite resource that require enormous investments of time and labor to construct.
If germline mutator alleles are only detectable in these highly unusual experimental populations, we are unlikely to discover more than a small fraction of the mutator alleles that may exist in nature.</p>
<p>Fortunately, the approach introduced in this paper is readily adaptable to datasets beyond RILs.
Thousands of human pedigrees have been sequenced in an effort to precisely estimate the rate of human <em>de novo</em> germline mutation <span class="citation" data-cites="PMID:31549960 PMID:28959963 PMID:29700473">[<a href="#ref-PMID:31549960" role="doc-biblioref"><strong>PMID:31549960?</strong></a>,<a href="#ref-PMID:28959963" role="doc-biblioref"><strong>PMID:28959963?</strong></a>,<a href="#ref-PMID:29700473" role="doc-biblioref"><strong>PMID:29700473?</strong></a>]</span>, and as family sequencing has become a more common step in the diagnosis of many congenital disorders, these datasets are growing on a daily basis.
Large cohorts of two- or three-generation families are an example of a regime in which AMSD could enjoy high power; by pooling sparse mutation counts across many individuals who share the same candidate mutator allele, even a subtle mutator signal could potentially rise above the noise of <em>de novo</em> germline mutation rate estimates.</p>
<blockquote>
<p>We note, however, that the aggregate mutation spectrum distance approach will require modification before it can be successfully applied to cohorts of outbred, sexually-reproducing individuals.
AMSD assumes that individuals harbor one of two possible genotypes at each marker, and does not yet account for heterozygous genotypes.
As a result, our method is currently applicable only to resources like the BXD RILs, in which individuals have been inbred for sufficiently long that effectively all genotypes are homozgyous.</p>
</blockquote>
<p>Selection on germline mutator alleles will likely prevent large-effect mutators from reaching high allele frequencies, but a subset may be detectable by sequencing a sufficient number of human trios <span class="citation" data-cites="PMID:35666194">[<a href="#ref-PMID:35666194" role="doc-biblioref"><strong>PMID:35666194?</strong></a>]</span>.
Since germline mutators often seem to exert their effects on a small number of mutation types, mutation spectrum analyses may have greater power to detect the genes that underlie heritable mutation rate variation, even if each gene has only a modest effect on the overall mutation rate per generation.</p>
<h2 id="materials-and-methods">Materials and Methods</h2>
<h3 id="identifying-de-novo-germline-mutations-in-the-bxds">Identifying <em>de novo</em> germline mutations in the BXDs</h3>
<p>The BXD resource currently comprises a total of 152 recombinant inbred lines (RILs).
BXDs were derived from either F2 or advanced intercrosses, and subsequently inbred by brother-sister mating for up to 180 generations <span class="citation" data-cites="PMID:33472028">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>.
BXDs were generated in distinct breeding “epochs,” which were each initiated with a distinct cross of C57BL/6J and DBA/2J parents; epochs 1, 2, 4, and 6 were derived from F2 crosses, while epochs 3 and 5 were derived from advanced intercrosses <span class="citation" data-cites="PMID:33472028">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>.
Previously, we analyzed whole-genome sequencing data from the BXDs and identified candidate <em>de novo</em> germline mutations in each line <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.
A detailed description of the methods used for DNA extraction, sequencing, alignment, and variant processing, as well as the characteristics of the <em>de novo</em> mutations, are available in a previous manuscript <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.</p>
<p>Briefly, we identified private single-nucleotide mutations in each BXD that were absent from all other BXDs, as well as from the C57BL/6J and DBA/2J parents. We required each private variant to be meet the following criteria:</p>
<ul>
<li><p>genotyped as either homozygous or heterozygous for the alternate allele, with at least 90% of sequencing reads supporting the alternate allele</p></li>
<li><p>supported by at least 10 sequencing reads</p></li>
<li><p>Phred-scaled genotype quality of at least 20</p></li>
<li><p>must not overlap regions of the genome annotated as segmental duplications or simple repeats in GRCm38/mm10</p></li>
<li><p>must occur on a parental haplotype that was inherited by at least one other BXD at the same locus; these other BXDs must be homozygous for the reference allele at the variant site</p></li>
</ul>
<h3 id="a-new-approach-to-discover-germline-mutator-alleles">A new approach to discover germline mutator alleles</h3>
<h4 id="calculating-aggregate-mutation-spectrum-distance">Calculating aggregate mutation spectrum distance</h4>
<p>We developed a new approach to discover loci that affect the germline <em>de novo</em> mutation spectrum in biparental RILs (Figure <a href="#fig:distance-method">1</a>).</p>
<p>We assume that a collection of haplotypes has been genotyped at informative markers, and that <em>de novo</em> germline mutations have been identified on each haplotype.</p>
<p>At each informative marker, we divide haplotypes into two groups based on the parental allele that they inherited.
We then compute a <span class="math inline">\(k\)</span>-mer mutation spectrum using the aggregate mutation counts in each haplotype group.
The <span class="math inline">\(k\)</span>-mer mutation spectrum contains the frequency of every possible <span class="math inline">\(k\)</span>-mer mutation type in a collection of mutations, and can be represented as a vector of size <span class="math inline">\(6 \times 4^{k - 1}\)</span> after collapsing by strand complement.
For example, the 1-mer mutation spectrum is a 6-element vector that contains the frequencies of C&gt;T, C&gt;G, C&gt;A, A&gt;G, A&gt;T, and A&gt;C mutations.
Since C&gt;T transitions at CpG nucleotides are often caused by a distinct mechanism (spontaneous deamination of methylated cytosine), we expand the 1-mer mutation spectrum to include a separate category for CpG&gt;TpG mutations <span class="citation" data-cites="PMID:19488047">[<a href="#ref-PMID:19488047" role="doc-biblioref"><strong>PMID:19488047?</strong></a>]</span>.</p>
<p>At each marker, we then compute the cosine distance between the two aggregate spectra.
The cosine distance between two vectors <span class="math inline">\(\mathbf{A}\)</span> and <span class="math inline">\(\mathbf{B}\)</span> is defined as</p>
<p><span class="math display">\[D^C = 1 - \frac{\mathbf{A} \cdot \mathbf{B}}{||\mathbf{A}|| \ ||\mathbf{B}||}\]</span></p>
<p>where <span class="math inline">\(||\mathbf{A}||\)</span> and <span class="math inline">\(||\mathbf{B}||\)</span> are the <span class="math inline">\(L^2\)</span> (or Euclidean) norms of <span class="math inline">\(\mathbf{A}\)</span> and <span class="math inline">\(\mathbf{B}\)</span>, respectively.
The cosine distance metric has a number of favorable properties for comparing mutation spectra.
Since it adjusts for the magnitude of the two input vectors, cosine distance can be used to compare two spectra with unequal total mutation counts (even if those total counts are relatively small).
Additionally, by calculating the cosine distance between mutation spectra, we avoid the need to perform separate comparisons of mutation counts at each individual <span class="math inline">\(k\)</span>-mer mutation type.</p>
<p>Inspired by methods from QTL mapping <span class="citation" data-cites="PMID:7851788 PMID:30591514">[<a href="#ref-PMID:7851788" role="doc-biblioref"><strong>PMID:7851788?</strong></a>,<a href="#ref-PMID:30591514" role="doc-biblioref"><strong>PMID:30591514?</strong></a>]</span>, we use permutation tests to establish genome-wide cosine distance thresholds.
In each of <span class="math inline">\(N\)</span> permutation trials, we randomly shuffle the per-haplotype mutation data such that haplotype labels no longer correspond to the correct mutation counts.
Using the shuffled mutation data, we perform a genome-wide scan as described above, and record the maximum cosine distance observed at any locus.
After <span class="math inline">\(N\)</span> permutations (usually 10,000), we compute the <span class="math inline">\(1 - p\)</span> percentile of the distribution of maximum statistics, and use that percentile value as a genome-wide significance threshold (for example, at <span class="math inline">\(p = 0.05\)</span>).</p>
<h4 id="estimating-confidence-intervals-around-amsd-peaks">Estimating confidence intervals around AMSD peaks</h4>
<blockquote>
<p>If we identified an adjusted cosine distance peak on a particular chromosome, we used a bootstrap resampling approach <span class="citation" data-cites="PMID:8725246">[<a href="#ref-PMID:8725246" role="doc-biblioref"><strong>PMID:8725246?</strong></a>]</span> to estimate confidence intervals.
In each of <span class="math inline">\(N = 10,000\)</span> trials, we resampled the mutation spectrum data and corresponding marker genotypes (on the chromosome of interest) with replacement.
Using those resampled spectra and genotypes, we performed an aggregate mutation spectrum distance scan on the chromosome of interest and recorded the position of the marker with the largest adjusted cosine distance value.
We then defined a 90% confidence interval by finding two marker locations between which 90% of all <span class="math inline">\(N\)</span> bootstrap samples produced a peak cosine distance value.
In other words, we estimated the bounds of the 90% confidence interval by finding the markers that defined the 5th and 95th percentiles of the distribution of maximum adjusted cosine distance values across <span class="math inline">\(N\)</span> bootstrap trials.</p>
</blockquote>
<h4 id="accounting-for-relatedness-between-strains">Accounting for relatedness between strains</h4>
<p>We expect each BXD to derive approximately 50% of its genome from C57BL/6J and 50% from DBA/2J.
As a result, every pair of BXDs will likely have identical genotypes at a fraction of markers.
Pairs of more genetically similar BXDs may also have more similar mutation spectra, potentially due to shared polygenic effects on the mutation process.
Therefore, at a given marker, if the BXDs that inherited <em>D</em> alleles are more genetically dissimilar from those that inherited <em>B</em> alleles (considering all loci throughout the genome in our measurement of genetic similarity), we might expect the aggregate mutation spectra in the two groups to also be more dissimilar.</p>
<p>We implemented a simple approach to account for these potential issues of relatedness.
At each marker <span class="math inline">\(g_i\)</span>, we divide BXD haplotypes into two groups based on the parental allele they inherited.
As before, we first compute the aggregate mutation spectrum in each group of haplotypes and calculate the cosine distance between the two aggregate spectra (<span class="math inline">\(D^{C}_{i}\)</span>).
Then, within each group of haplotypes, we calculate the allele frequency of the <em>D</em> allele at every marker along the genome to obtain a vector of length <span class="math inline">\(n\)</span>, where <span class="math inline">\(n\)</span> is the number of genotyped markers.
To quantify the genetic similarity between the two groups of haplotypes, we calculate the Pearson correlation coefficient <span class="math inline">\(r_i\)</span> between the two vectors of marker-wide <em>D</em> allele frequencies.</p>
<p>Put another way, at every marker <span class="math inline">\(g_i\)</span> along the genome, we divide BXD haplotypes into two groups and compute two metrics: <span class="math inline">\(D^{C}_{i}\)</span> (the cosine distance between the two groups’ aggregate spectra) and <span class="math inline">\(r_i\)</span> (the correlation between genome-wide <em>D</em> allele frequencies in the two groups).
To control for the potential effects of genetic similarity on cosine distances, we regress <span class="math inline">\(\left(D^C_{1}, D^C_{2}, \ldots D^C_{n} \right)\)</span> on <span class="math inline">\(\left( r_1, r_2, \ldots r_n \right)\)</span> for all <span class="math inline">\(n\)</span> markers using an ordinary least-squares model.
We then use the residuals from the fitted model as the “adjusted” cosine distance values for each marker.
If genome-wide genetic similarity between haplotypes perfectly predicts cosine distances at each marker, these residuals will all be 0 (or very close to 0).
If genome-wide genetic similarity has no predictive power, the residuals will simply represent the difference between the observed cosine distance at a single marker and the marker-wide mean of cosine distances.</p>
<h4 id="accounting-for-bxd-population-structure-due-to-breeding-epochs">Accounting for BXD population structure due to breeding epochs</h4>
<p>The current BXD family was generated in six breeding “epochs.”
As discussed previously, each epoch was initiated with a distinct cross of C57BL/6J and DBA/2J parents; BXDs in four of the epochs were generated following F2 crosses of C57BL/6J and DBA/2J, and BXDs in the other two were generated following advanced intercrosses.
Due to this breeding approach the BXD epochs differ from each other in a few important ways.
For example, BXDs derived in epochs 3 and 5 (i.e., from advanced intercross) harbor larger numbers of fixed recombination breakpoints than those from epochs 1, 2, 4, and 6 <span class="citation" data-cites="PMID:33472028">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>.
Although the C57BL/6J and DBA/2J parents used to initialize each epoch were completely inbred, they each possessed a small number unique <em>de novo</em> germline mutations that were subsequently inherited by many of their offspring.
A number of these “epoch-specific” variants have also been linked to phenotypic variation observed between BXDs from different epochs <span class="citation" data-cites="PMID:33472028 PMID:31274109 PMID:30984232 PMID:31057322">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>,<a href="#ref-PMID:31274109" role="doc-biblioref"><strong>PMID:31274109?</strong></a>,<a href="#ref-PMID:30984232" role="doc-biblioref"><strong>PMID:30984232?</strong></a>,<a href="#ref-PMID:31057322" role="doc-biblioref"><strong>PMID:31057322?</strong></a>]</span>.</p>
<p>To account for potential population structure, as well as these epoch-specific effects, we introduced the ability to perform stratified permutation tests in the aggregate mutation spectrum distance approach.
Normally, in each of <em>N</em> permutations we shuffle the per-haplotype mutation spectrum data such that haplotype labels no longer correspond to the correct mutation spectra (i.e., shuffle mutation spectra <em>across</em> epochs).
In the stratified approach, we instead shuffle per-haplotype mutation data <em>within</em> epochs, preserving epoch structure while still enabling mutation spectra permutations.</p>
<blockquote>
<p>We used this epoch-aware approach for all permutation tests presented in this manuscript.</p>
</blockquote>
<h4 id="implementation-and-source-code">Implementation and source code</h4>
<p>The aggregate mutation spectrum distance method was implemented in Python, and relies heavily on the following Python libraries: <code>numpy</code>, <code>pandas</code>, <code>matplotlib</code>, <code>scikit-learn</code>, <code>pandera</code>, <code>seaborn</code>, and <code>numba</code> <span class="citation" data-cites="1LexfAxj 2z2qDwzz 1026Gxdsi lF0Fr9LL 34sZyTFd 169TfvTKZ 189xgFrlN">[<a href="#ref-1LexfAxj" role="doc-biblioref">5</a>,<a href="#ref-2z2qDwzz" role="doc-biblioref">6</a>,<a href="#ref-1026Gxdsi" role="doc-biblioref">7</a>,<a href="#ref-lF0Fr9LL" role="doc-biblioref">8</a>,<a href="#ref-34sZyTFd" role="doc-biblioref">9</a>,<a href="#ref-169TfvTKZ" role="doc-biblioref">10</a>,<a href="#ref-189xgFrlN" role="doc-biblioref">11</a>]</span>.</p>
<p>The code underlying AMSD, as well as documentation of the method, is available on GitHub (https://github.com/quinlan-lab/proj-mutator-mapping). We have also deposited a reproducible Snakemake <span class="citation" data-cites="fVLvtRTc">[<a href="#ref-fVLvtRTc" role="doc-biblioref">12</a>]</span> workflow for running reproducing all analyses and figures presented in the manuscript.</p>
<h3 id="simulations-to-assess-the-power-of-the-aggregate-mutation-spectrum-distance-approach">Simulations to assess the power of the aggregate mutation spectrum distance approach</h3>
<p>We performed a series of simple simulations to estimate our power to detect alleles that affect the germline mutation spectrum using the aggregate mutation spectrum distance method.</p>
<h4 id="simulating-genotypes">Simulating genotypes</h4>
<p>First, we simulate genotypes on a population of haplotypes at a collection of sites.
We define a matrix <span class="math inline">\(G\)</span> of size <span class="math inline">\((s, h)\)</span>, where <span class="math inline">\(s\)</span> is the number of sites and <span class="math inline">\(h\)</span> is the number of haplotypes.
We assume that every site is biallelic, and that the minor allele frequency at every site is 0.5.
For every entry <span class="math inline">\(G_{i,j}\)</span>, we take a single draw from a uniform distribution in the interval <span class="math inline">\([0.0, 1.0)\)</span>.
If the value of that draw is less than 0.5, we assign the value of <span class="math inline">\(G_{i,j}\)</span> to be <span class="math inline">\(1\)</span>.
Otherwise, we assign the value of <span class="math inline">\(G_{i,j}\)</span> to be <span class="math inline">\(0\)</span>.</p>
<h4 id="defining-expected-mutation-type-probabilities">Defining expected mutation type probabilities</h4>
<p>Next, we define a vector of 1-mer mutation probabilities:</p>
<p><span class="math display">\[P = \left( 0.29, \ 0.17, \ 0.12, \ 0.075, \ 0.1, \ 0.075, \ 0.17 \right)\]</span></p>
<p>These probabilities sum to 1 and roughly correspond to the expected frequencies of C&gt;T, CpG&gt;TpG, C&gt;A, C&gt;G, A&gt;T, A&gt;C, and A&gt;G <em>de novo</em> germline mutations in mice, respectively <span class="citation" data-cites="PMID:31492841">[<a href="#ref-PMID:31492841" role="doc-biblioref"><strong>PMID:31492841?</strong></a>]</span>.
If we are simulating the 3-mer mutation spectrum, we modify the vector of mutation probabilities <span class="math inline">\(P\)</span> to be length 96, and assign every 3-mer mutation type a value of <span class="math inline">\(\frac{P_c}{16}\)</span>, where <span class="math inline">\(P_c\)</span> is the probability of the “central” mutation type associated with the 3-mer mutation type.
In other words, each of the 16 possible N<ins>C</ins>N&gt;N<ins>T</ins>N 3-mer mutation types would be assigned a mutation probability of <span class="math inline">\(\frac{P_c}{16} = \frac{0.46}{16} = 0.02875\)</span>.
We then generate a vector of lambda values by scaling the mutation probabilities by the number of mutations we wish to simulate (<span class="math inline">\(m\)</span>):</p>
<p><span class="math display">\[\lambda = Pm\]</span></p>
<p>We also create a second vector of lambda values (<span class="math inline">\(\lambda^{\prime}\)</span>), in which we multiply the <span class="math inline">\(\lambda\)</span> value of a single mutation type by the mutator effect size <span class="math inline">\(e\)</span>.</p>
<blockquote>
<p>Rather than simulate the same mean number of mutations (<span class="math inline">\(m\)</span>) on every haplotype, we also performed a series of simulations in which the mean number of mutations on each haplotype was allowed to vary.
The BXD RILs were inbred for variable numbers of generations, and each BXD therefore accumulated a variable number of <em>de novo</em> germline mutations <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.
To more closely approximate the BXD haplotypes, we performed simulations in which the number of mutations (<span class="math inline">\(m\)</span>) on each haplotype was drawn from a uniform distribution from <span class="math inline">\(m\)</span> to <span class="math inline">\(20m\)</span>.
In other words, we created a vector of mutation counts <span class="math inline">\(M\)</span> containing <span class="math inline">\(h\)</span> evenly-spaced integers from <span class="math inline">\(m\)</span> to <span class="math inline">\(20m\)</span>, where <span class="math inline">\(h\)</span> is the number of simulated haplotypes.
Thus, if we simulated between 100 and 2,000 mutations on 50 haplotypes, the <span class="math inline">\(i\)</span>th entry of <span class="math inline">\(M\)</span> would be <span class="math inline">\(100 + \frac{(2,000 - 100)}{50}i\)</span>.
Each haplotype’s mean number of mutations was then assigned by looking up the haplotype’s index <span class="math inline">\(i\)</span> in <span class="math inline">\(M\)</span>.</p>
</blockquote>
<p>In our simulations, we assume that genotypes at a single site (the “mutator locus”) are associated with variation in the mutation spectrum.
That is, at a single site <span class="math inline">\(s_i\)</span>, all of the haplotypes with <span class="math inline">\(1\)</span> alleles should have elevated rates of a particular mutation type and draw their mutation counts from <span class="math inline">\(\lambda^{\prime}\)</span>, while all of the haplotypes with <span class="math inline">\(0\)</span> alleles should have “wild-type” rates of that mutation type and draw their mutation counts from <span class="math inline">\(\lambda\)</span>.
We therefore pick a random site <span class="math inline">\(s_i\)</span> to be the “mutator locus,” and identify the indices of haplotypes in <span class="math inline">\(G\)</span> that were assigned <span class="math inline">\(1\)</span> alleles at <span class="math inline">\(s_i\)</span>.
We call these indices <span class="math inline">\(h_{mut}\)</span>.</p>
<h4 id="simulating-mutation-spectra">Simulating mutation spectra</h4>
<p>To simulate the mutation spectrum on our toy population of haplotypes, we define a matrix <span class="math inline">\(C\)</span> of size <span class="math inline">\((h, n)\)</span>, where <span class="math inline">\(n = 6 \times 4^{k - 1}\)</span> (or if <span class="math inline">\(k = 1\)</span> and we include CpG&gt;TpG mutations, <span class="math inline">\(6 \times 4^{k - 1} + 1\)</span>).</p>
<p>Then, we populate the matrix <span class="math inline">\(C\)</span> separately for <em>mutator</em> and <em>wild-type</em> haplotypes.
For every row <span class="math inline">\(i\)</span> in the matrix (i.e., for every haplotype), we first ask if <span class="math inline">\(i\)</span> is in <span class="math inline">\(h_{mut}\)</span> (that is, if the haplotype at index <span class="math inline">\(i\)</span> was assigned a <span class="math inline">\(1\)</span> allele at the “mutator locus”).
If so, we set the values of <span class="math inline">\(C_i\)</span> to be the results of a single Poisson draw from <span class="math inline">\(\lambda^{\prime}\)</span>.
If row <span class="math inline">\(i\)</span> is not in <span class="math inline">\(h_{mut}\)</span>, we set the values of <span class="math inline">\(C_i\)</span> to be the results of a single Poisson draw from <span class="math inline">\(\lambda\)</span>.</p>
<h4 id="assessing-power-to-detect-a-simulated-mutator-allele-using-amsd">Assessing power to detect a simulated mutator allele using AMSD</h4>
<p>For each combination of parameters (number of simulated haplotypes, number of simulated markers, mutator effect size, etc.), we run 100 independent trials.
In each trial, we simulate the genotype matrix <span class="math inline">\(G\)</span> and the mutation counts <span class="math inline">\(C\)</span>.
We calculate a “focal” cosine distance as the cosine distance between the aggregate mutation spectra of haplotypes with either genotype at <span class="math inline">\(s_i\)</span> (the site at which we artificially simulated an association between genotypes and mutation spectrum variation).
We then perform an aggregate mutation spectrum distance scan using <span class="math inline">\(N = 1,000\)</span> permutations.
If fewer than 5% of the <span class="math inline">\(N\)</span> permutations produced a cosine distance greater than or equal to the focal distance, we say that the approach successfully identified the mutator allele in that trial.</p>
<h4 id="assessing-power-to-detect-a-simulated-mutator-allele-using-quantitative-trait-locus-qtl-mapping">Assessing power to detect a simulated mutator allele using quantitative trait locus (QTL) mapping</h4>
<p>Using simulated data, we also assessed the power of traditional quantitative trait locus (QTL) mapping to detect a locus associated with mutation spectrum variation.
As described above, we simulated both genotype and mutation spectra for a population of haplotypes under various conditions (number of mutations per haplotype, mutator effect size, etc.).
Using those simulated data, we used R/qtl2 <span class="citation" data-cites="PMID:30591514">[<a href="#ref-PMID:30591514" role="doc-biblioref"><strong>PMID:30591514?</strong></a>]</span> to perform a genome scan for significant QTL as follows; we assume that the simulated genotype markers are evenly spaced (in physical Mbp coordinates) on a single chromosome.
First, we calculate the fraction of each haplotype’s <em>de novo</em> mutations that belong to each of the <span class="math inline">\(6 \times 4^{k-1}\)</span> possible <span class="math inline">\(k\)</span>-mer mutation types.
We then convert the simulated genotypes at each marker to genotype probabilities using the <code>calc_genoprob</code> function in R/qtl2, with <code>map_function = "c-f"</code> and <code>error_prob = 0</code>.
For every <span class="math inline">\(k\)</span>-mer mutation type, we use genotype probabilities and per-haplotype mutation fractions to perform a scan for QTL with the <code>scan1</code> function; to make the results more comparable to those from the AMSD method, we do not include any covariates or kinship matrices in these QTL scans.
We then use the <code>scan1perm</code> function to perform 1,000 permutations of the per-haplotype mutation fractions and calculate log-odds (LOD) thresholds for significance.
We consider the QTL scan to be “successful” if it produces a LOD score above the significance threshold (defined using <span class="math inline">\(\alpha = \frac{0.05}{7}\)</span>) for the marker at which we simulated an association with mutation spectrum variation.</p>
<blockquote>
<p><strong>Note</strong>: In our simulations, we augment the mutation rate of a single <span class="math inline">\(k\)</span>-mer mutation type on haplotypes carrying the simulated mutator allele.
However, in an experimental setting, we would not expect to have <em>a priori</em> knowledge of the mutation type affected by the mutator.
Thus, by using an alpha threshold of 0.05 in our simulations, we would likely over-estimate the power of QTL mapping for detecting the mutator.
Since we would need to perform 7 separate QTL scans (one for each 1-mer mutation type plus CpG&gt;TpG) in an experimental setting, we calculate QTL LOD thresholds at a Bonferroni-corrected alpha value of <span class="math inline">\(\alpha = \frac{0.05}{7}\)</span>.</p>
</blockquote>
<h3 id="applying-the-aggregate-mutation-spectrum-distance-method-to-the-bxds">Applying the aggregate mutation spectrum distance method to the BXDs</h3>
<p>We downloaded previously-generated BXD <em>de novo</em> germline mutation data from the GitHub repository associated with our previous manuscript, which was also archived at Zenodo <span class="citation" data-cites="TYYBVYN3 z6H1lPG4 PMID:35545679">[<a href="#ref-TYYBVYN3" role="doc-biblioref">13</a>,<a href="#ref-z6H1lPG4" role="doc-biblioref">14</a>,<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>, and downloaded a CSV file of BXD genotypes at ~7,300 informative markers from GeneNetwork <span class="citation" data-cites="t8fRZlGg PMID:27933521">[<a href="#ref-t8fRZlGg" role="doc-biblioref">15</a>,<a href="#ref-PMID:27933521" role="doc-biblioref"><strong>PMID:27933521?</strong></a>]</span>.
We also downloaded relevant metadata about each BXD from the manuscript describing the updated BXD resource <span class="citation" data-cites="PMID:33472028">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>.
These files are included in the GitHub repository associated with this manuscript.</p>
<p>As in our previous manuscript <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>, we included mutation data from a subset of the 152 BXDs in our aggregate mutation spectrum distance scans.
Specifically, we removed BXDs that were backcrossed to a C57BL/6J or DBA/2J parent at any point during the inbreeding process (usually, in order to rescue that BXD from inbreeding depression <span class="citation" data-cites="PMID:33472028">[<a href="#ref-PMID:33472028" role="doc-biblioref"><strong>PMID:33472028?</strong></a>]</span>).
We also removed BXD68 from our genome-wide scans, since we previously discovered a hyper-mutator phenotype in that line; the C&gt;A germline mutation rate in BXD68 is over 5 times the population mean, likely due to a private deleterious nonsynonymous mutation in <em>Mutyh</em> <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.
In our previous manuscript, we removed any BXDs that had been inbred for fewer than 20 generations, as it takes approximately 20 generations of strict brother-sister mating for an RIL genome to become &gt;98% homozygous <span class="citation" data-cites="8Z2SLpxu">[<a href="#ref-8Z2SLpxu" role="doc-biblioref">16</a>]</span>.
As a result, any potential mutator allele would almost certainly be either fixed or lost after 20 generations; if fixed, the allele would remain linked to any excess mutations it causes for the duration of subsequent inbreeding.
In other words, the <em>de novo</em> mutations present in the genome of a “young” BXD (i.e., a BXD that was inbred for fewer than 20 generations) would not reflect a mutator allele’s activity as strongly as the mutations present in the genome of a much older BXD.
This presented a challenge when we used quantitative trait locus mapping to discover mutator alleles in our previous manuscript, since the phenotypes (i.e., C&gt;A mutation rates) of young and old BXDs were weighted equally; thus, we simply removed the younger BXDs from our analysis to avoid using their especially noisy mutation spectra.
Since AMSD computes an <em>aggregate</em> mutation spectrum using all BXDs that inherited a particular allele at a locus, and can overcome the sparsity and noise of individual mutation spectra, we chose to include these younger BXDs in our genome-wide scans in this study.</p>
<p>In total, we included 117 BXDs in our genome-wide scans.</p>
<h3 id="identifying-candidate-single-nucleotide-mutator-alleles-overlapping-the-chromosome-6-peak">Identifying candidate single-nucleotide mutator alleles overlapping the chromosome 6 peak</h3>
<p>We investigated the region implicated by our aggregate mutation spectrum distance approach on chromosome 6 by subsetting the joint-genotyped BXD VCF file (European Nucleotide Archive accession PRJEB45429 <span class="citation" data-cites="V7hHut0M">[<a href="#ref-V7hHut0M" role="doc-biblioref">17</a>]</span>) using <code>bcftools</code> <span class="citation" data-cites="PMID:33590861">[<a href="#ref-PMID:33590861" role="doc-biblioref"><strong>PMID:33590861?</strong></a>]</span>.</p>
<blockquote>
<p>We defined the candidate interval surrounding the cosine distance peak on chromosome 6 as the 90% bootstrap confidence interval (extending from approximately 95 Mbp to 114 Mbp).</p>
</blockquote>
<p>To predict the functional impacts of both single-nucleotide variants and indels on splicing, protein structure, etc., we annotated variants in the BXD VCF using the following <code>snpEff</code> <span class="citation" data-cites="PMID:22728672">[<a href="#ref-PMID:22728672" role="doc-biblioref"><strong>PMID:22728672?</strong></a>]</span> command:</p>
<pre><code> java -Xmx16g -jar /path/to/snpeff/jarfile GRCm38.75 /path/to/bxd/vcf &gt; /path/to/uncompressed/output/vcf</code></pre>
<p>and used <code>cyvcf2</code> <span class="citation" data-cites="PMID:28165109">[<a href="#ref-PMID:28165109" role="doc-biblioref"><strong>PMID:28165109?</strong></a>]</span> to iterate over the annotated VCF file in order to identify nonsynonymous fixed differences between the parental C57BL/6J and DBA/2J strains.</p>
<h3 id="identifying-candidate-structural-variant-alleles-overlapping-the-chromosome-6-peak">Identifying candidate structural variant alleles overlapping the chromosome 6 peak</h3>
<p>We downloaded summary VCFs containing insertion, deletion and inversion structural variants (identified via high-quality, long-read assembly of inbred laboratory mouse strains <span class="citation" data-cites="8KLgRyLH">[<a href="#ref-8KLgRyLH" role="doc-biblioref">3</a>]</span>) from the Zenodo link associated with the Ferraj et al. manuscript: https://doi.org/10.5281/zenodo.7644286.</p>
<p>We then downloaded a TSV file containing RefSeq gene predictions in GRCm39/mm39 from the UCSC Table Browser <span class="citation" data-cites="PMID:14681465">[<a href="#ref-PMID:14681465" role="doc-biblioref"><strong>PMID:14681465?</strong></a>]</span>, and used the <code>bx-python</code> library <span class="citation" data-cites="ZjBdGGm0">[<a href="#ref-ZjBdGGm0" role="doc-biblioref">18</a>]</span> to intersect the interval spanned by each structural variant with the intervals spanned by the <code>txStart</code> and <code>txEnd</code> of every RefSeq entry.</p>
<p>We queried all structural variants within the 90% bootstrap confidence interval on chromosome 6.</p>
<h3 id="extracting-mutation-signatures">Extracting mutation signatures</h3>
<p>We used SigProfilerExtractor (v.1.1.21) <span class="citation" data-cites="PMID:30371878">[<a href="#ref-PMID:30371878" role="doc-biblioref"><strong>PMID:30371878?</strong></a>]</span> to extract mutation signatures from the BXD mutation data. After converting the BXD mutation data to the “matrix” input format expected by SigProfilerExtractor, we ran the <code>sigProfilerExtractor</code> method as follows:</p>
<div class="sourceCode" id="cb2"><pre class="sourceCode python"><code class="sourceCode python"><span id="cb2-1"><a href="#cb2-1" aria-hidden="true" tabindex="-1"></a><span class="co"># install the mm10 mouse reference data</span></span>
<span id="cb2-2"><a href="#cb2-2" aria-hidden="true" tabindex="-1"></a>genInstall.install(<span class="st">&#39;mm10&#39;</span>)</span>
<span id="cb2-3"><a href="#cb2-3" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb2-4"><a href="#cb2-4" aria-hidden="true" tabindex="-1"></a><span class="co"># run mutation signature extraction</span></span>
<span id="cb2-5"><a href="#cb2-5" aria-hidden="true" tabindex="-1"></a>sig.sigProfilerExtractor(</span>
<span id="cb2-6"><a href="#cb2-6" aria-hidden="true" tabindex="-1"></a>    <span class="st">&#39;matrix&#39;</span>,</span>
<span id="cb2-7"><a href="#cb2-7" aria-hidden="true" tabindex="-1"></a>    <span class="op">/</span>path<span class="op">/</span>to<span class="op">/</span>output<span class="op">/</span>directory,</span>
<span id="cb2-8"><a href="#cb2-8" aria-hidden="true" tabindex="-1"></a>    <span class="op">/</span>path<span class="op">/</span>to<span class="op">/</span><span class="bu">input</span><span class="op">/</span>mutations,</span>
<span id="cb2-9"><a href="#cb2-9" aria-hidden="true" tabindex="-1"></a>    maximum_signatures<span class="op">=</span><span class="dv">10</span>,</span>
<span id="cb2-10"><a href="#cb2-10" aria-hidden="true" tabindex="-1"></a>    nmf_replicates<span class="op">=</span><span class="dv">100</span>,</span>
<span id="cb2-11"><a href="#cb2-11" aria-hidden="true" tabindex="-1"></a>    opportunity_genome<span class="op">=</span><span class="st">&quot;mm10&quot;</span>,</span>
<span id="cb2-12"><a href="#cb2-12" aria-hidden="true" tabindex="-1"></a>)</span></code></pre></div>
<h3 id="comparing-mutation-spectra-between-mouse-genomes-project-strains">Comparing mutation spectra between Mouse Genomes Project strains</h3>
<p>We downloaded mutation data from a previously published analysis <span class="citation" data-cites="PMID:30753674">[<a href="#ref-PMID:30753674" role="doc-biblioref"><strong>PMID:30753674?</strong></a>]</span> (Supplementary File 1, Excel Table S3) that identified strain-private mutations in 29 strains that were originally whole-genome sequenced as part of the Sanger Mouse Genomes (MGP) project <span class="citation" data-cites="PMID:21921910">[<a href="#ref-PMID:21921910" role="doc-biblioref"><strong>PMID:21921910?</strong></a>]</span>.
When comparing counts of each mutation type between MGP strains that harbored either <em>D</em> or <em>B</em> alleles at the chromosome 4 or chromosome 6 mutator loci, we adjusted mutation counts by the number of callable A, T, C, or G nucleotides in each strain as described previously <span class="citation" data-cites="PMID:35545679">[<a href="#ref-PMID:35545679" role="doc-biblioref"><strong>PMID:35545679?</strong></a>]</span>.</p>
<h3 id="querying-genenetwork-for-eqtls-at-the-mutator-locus">Querying GeneNetwork for eQTLs at the mutator locus</h3>
<p>We used the online GeneNetwork resource <span class="citation" data-cites="PMID:27933521">[<a href="#ref-PMID:27933521" role="doc-biblioref"><strong>PMID:27933521?</strong></a>]</span>, which contains array- and RNA-seq-derived expression measurements in a wide variety of tissues, to find <em>cis</em>-eQTLs for the DNA repair genes we implicated under the cosine distance peak on chromosome 6.
On the GeneNetwork homepage (genenetwork.org), we selected the “BXD Family” <strong>Group</strong> and used the <strong>Type</strong> dropdown menu to select each of the specific expression datasets described in Table <a href="#tbl:eqtl-provenance">2</a>.
In the <strong>Get Any</strong> text box, we then entered the listed gene name and clicked <strong>Search</strong>.
After selecting the appropriate trait ID on the next page, we used the <strong>Mapping Tools</strong> dropdown to run Hayley-Knott regression <span class="citation" data-cites="PMID:16718932">[<a href="#ref-PMID:16718932" role="doc-biblioref"><strong>PMID:16718932?</strong></a>]</span> with default parameters: 1,000 permutations, interval mapping, no cofactors, and WGS-based genotypes (2022).</p>
<blockquote>
<p>If we discovered a significant cis-eQTL for the gene of interest (that is, a locus on chromosome 6 with an LRS greater than or equal to the “significant LRS” genome-wide threshold), we then performed a second genome-wide association test for the trait of interest using GEMMA <span class="citation" data-cites="PMID:2453419">[<a href="#ref-PMID:2453419" role="doc-biblioref"><strong>PMID:2453419?</strong></a>]</span> with the following parameters: WGS-based marker genotypes, a minor allele frequency threshold of 0.05, and leave-one-chromosome-out (LOCO).
By using both Haley-Knott regression and GEMMA, we could first discover loci that exceeded a genome-wide LRS threshold, and then more precisely estimate the effect of those loci on gene expression <span class="citation" data-cites="tW44yAw8">[<a href="#ref-tW44yAw8" role="doc-biblioref">19</a>]</span>.</p>
</blockquote>
<p>The exact names of the expression datasets we used for each tissue are shown in Table <a href="#tbl:eqtl-provenance">2</a> below:</p>
<div id="tbl:eqtl-provenance" class="tablenos">
<table id="tbl:eqtl-provenance">
<caption><span>Table 2:</span> Names of gene expression datasets used for each tissue type on GeneNetwork </caption>
<colgroup>
<col style="width: 50%" />
<col style="width: 50%" />
</colgroup>
<thead>
<tr class="header">
<th>Tissue name</th>
<th>Complete name of GeneNetwork expression data</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Kidney</td>
<td><code>Mouse kidney M430v2 Sex Balanced (Aug06) RMA</code></td>
</tr>
<tr class="even">
<td>Gastrointestinal</td>
<td><code>UTHSC Mouse BXD Gastrointestinal Affy MoGene 1.0 ST Gene Level (Apr14) RMA</code></td>
</tr>
<tr class="odd">
<td>Hematopoetic stem cells</td>
<td><code>UMCG Stem Cells ILM6v1.1 (Apr09) transformed</code></td>
</tr>
<tr class="even">
<td>Spleen</td>
<td><code>UTHSC Affy MoGene 1.0 ST Spleen (Dec10) RMA</code></td>
</tr>
<tr class="odd">
<td>Liver</td>
<td><code>UTHSC BXD Liver RNA-Seq Avg (Oct19) TPM Log2</code></td>
</tr>
<tr class="even">
<td>Heart</td>
<td><code>NHLBI BXD All Ages Heart RNA-Seq (Nov20) TMP Log2 **</code></td>
</tr>
<tr class="odd">
<td>Hippocampus</td>
<td><code>Hippocampus Consortium M430v2 (Jun06) RMA</code></td>
</tr>
</tbody>
</table>
</div>
<h3 id="calculating-the-frequencies-of-candidate-mutator-alleles-in-wild-mice">Calculating the frequencies of candidate mutator alleles in wild mice</h3>
<p>To determine the frequencies of the <em>Ogg1</em> and <em>Setmar</em> nonsynonymous mutations in other populations of mice, we queried a VCF file containing genome-wide variation in 67 wild-derived mice from four species of <em>Mus</em> <span class="citation" data-cites="PMID:27622383">[<a href="#ref-PMID:27622383" role="doc-biblioref"><strong>PMID:27622383?</strong></a>]</span>.
We calculated the allele frequency of each nonsynonymous mutation in each of the four species or subspecies (<em>Mus musculus domesticus</em>, <em>Mus musculus musculus</em>, <em>Mus musculus castaneus</em>, and <em>Mus spretus</em>), including genotypes that met the following criteria:</p>
<ul>
<li><p>supported by at least 10 sequencing reads</p></li>
<li><p>Phred-scaled genotype quality of at least 20</p></li>
</ul>
<h3 id="testing-for-epistasis-between-the-two-mutator-loci">Testing for epistasis between the two mutator loci</h3>
<p>To test for statistical epistasis between the mutator loci on chromosome 4 and chromosome 6, we modeled C&gt;A mutation rates in the BXDs as a function of genotypes at either locus.
Specifically, we tested for statistical interaction between genotypes by fitting a generalized linear model in the R statistical language as follows:</p>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="#cb3-1" aria-hidden="true" tabindex="-1"></a>m1 <span class="ot">&lt;-</span> <span class="fu">glm</span>(Count <span class="sc">~</span> <span class="fu">offset</span>(<span class="fu">log</span>(ADJ_AGE)) <span class="sc">+</span> Genotype_A <span class="sc">*</span> Genotype_B, <span class="at">data =</span> data, <span class="at">family=</span><span class="fu">poisson</span>())</span></code></pre></div>
<p>In this model, <code>Count</code> is the count of C&gt;A <em>de novo</em> mutations observed in each BXD.
<code>ADJ_AGE</code> is the product of the number of “callable” cytosine/guanine nucleotides in each BXD (i.e., the total number of cytosines/guanines covered by at least 10 sequencing reads) and the number of generations for which the BXD was inbred.
We included the logarithm of <code>ADJ_AGE</code> as an “offset” in order to model the response variable as a rate (expressed per base-pair, per generation) rather than an absolute count; the BXDs differ in both their durations of inbreeding and the proportions of their genomes that were sequenced to sufficient depth, which influences the number of mutations we observe in each BXD.
The <code>Genotype_A</code> and <code>Genotype_B</code> terms represent the genotypes of BXDs at markers <code>rs27509845</code> and <code>rs46276051</code> (the markers with peak cosine distances on chromosomes 4 and 6 in the two aggregate mutation spectrum distance scans).
We limited our analysis to the n = 108 BXDs that were homozygous at both sites, allowing us to model genotypes at either locus as binary variables (“B” or “D”).
Using analysis of variance (ANOVA), we then compared the model including an interaction effect to a model including only additive effects:</p>
<div class="sourceCode" id="cb4"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb4-1"><a href="#cb4-1" aria-hidden="true" tabindex="-1"></a>m2 <span class="ot">&lt;-</span> <span class="fu">glm</span>(Count <span class="sc">~</span> <span class="fu">offset</span>(<span class="fu">log</span>(ADJ_AGE)) <span class="sc">+</span> Genotype_A <span class="sc">+</span> Genotype_B, <span class="at">data =</span> data, <span class="at">family=</span><span class="fu">poisson</span>())</span></code></pre></div>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="#cb5-1" aria-hidden="true" tabindex="-1"></a><span class="fu">anova</span>(m1, m2, <span class="at">test=</span><span class="st">&quot;Chisq&quot;</span>)</span></code></pre></div>
<blockquote>
<p>If model <code>m1</code> is a significantly better fit to the data than <code>m2</code>, we can reject the null hypothesis that the effect of <em>D</em> genotypes at both markers is equal to the sum of the marginal effects of <em>D</em> genotypes at either <code>rs27509845</code> or <code>rs46276051</code>.
In other words, if <code>m1</code> is a better fit than <code>m2</code>, then the combined effect of <em>D</em> genotypes at both markers is non-additive, and indicative of statistical epistasis.</p>
</blockquote>
<p>We tested for epistasis in the Sanger Mouse Genomes Project (MGP) strains using a nearly-identical approach.
In this analysis, we fit two models as follows:</p>
<div class="sourceCode" id="cb6"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb6-1"><a href="#cb6-1" aria-hidden="true" tabindex="-1"></a>m1 <span class="ot">&lt;-</span> <span class="fu">glm</span>(Count <span class="sc">~</span> <span class="fu">offset</span>(<span class="fu">log</span>(CALLABLE_C)) <span class="sc">+</span> Genotype_A <span class="sc">*</span> Genotype_B, <span class="at">data =</span> data, <span class="at">family=</span><span class="fu">poisson</span>())</span>
<span id="cb6-2"><a href="#cb6-2" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb6-3"><a href="#cb6-3" aria-hidden="true" tabindex="-1"></a>m2 <span class="ot">&lt;-</span> <span class="fu">glm</span>(Count <span class="sc">~</span> <span class="fu">offset</span>(<span class="fu">log</span>(CALLABLE_C)) <span class="sc">+</span> Genotype_A <span class="sc">+</span> Genotype_B, <span class="at">data =</span> data, <span class="at">family=</span><span class="fu">poisson</span>())</span></code></pre></div>
<p>where <code>Count</code> is the count of strain-private C&gt;A mutations observed in each MGP strain <span class="citation" data-cites="PMID:30753674">[<a href="#ref-PMID:30753674" role="doc-biblioref"><strong>PMID:30753674?</strong></a>]</span>.
The <code>CALLABLE_C</code> term represents the total number of cytosine and guanine nucleotides that were accessible for mutation calling in each strain, and the <code>Genotype_A</code> and <code>Genotype_B</code> terms represent MGP genotypes at the chromosome 4 and chromosome 6 mutator loci, respectively.
We compared the two models using ANOVA as described above.</p>
<h2 id="acknowledgments">Acknowledgments</h2>
<p>We thank Robert W. Williams (University of Tennessee Health Sciences Center) and Don F. Conrad (Oregon Health &amp; Science University) for very helpful comments and feedback on a draft of this manuscript.</p>
<h2 class="page_break_before" id="supplementary-information">Supplementary information</h2>
<h3 id="missense-mutations-in-setmar-are-unlikely-to-contribute-to-epistasis-in-the-bxds">Missense mutations in <em>Setmar</em> are unlikely to contribute to epistasis in the BXDs</h3>
<p>Unlike <em>Ogg1</em>, <em>Setmar</em> does not participate directly in base-excision repair, though its primate ortholog plays an indirect role in the repair of double-stranded DNA breaks via non-homologous end joining (NHEJ).
In anthropoid primates, <em>SETMAR</em> encodes a fusion of two functional domains: a SET domain-containing histone methyltransferase and a transposase domain from the <em>Mariner</em> family (MAR) <span class="citation" data-cites="PMID:16672366">[<a href="#ref-PMID:16672366" role="doc-biblioref"><strong>PMID:16672366?</strong></a>]</span>; the mouse <em>Setmar</em> ortholog only encodes the histone methyltransferase domain.
In human cell lines, <em>SETMAR</em> localizes to induced double-strand breaks (DSBs) and dimethylates nearby H3K36, which promotes the recruitment of DNA repair components involved in NHEJ to the DSB <span class="citation" data-cites="PMID:21187428">[<a href="#ref-PMID:21187428" role="doc-biblioref"><strong>PMID:21187428?</strong></a>]</span>.
There is also evidence that overexpression of <em>SETMAR</em> (also known as <em>Metnase</em>) improves the efficiency of NHEJ <span class="citation" data-cites="PMID:16332963">[<a href="#ref-PMID:16332963" role="doc-biblioref"><strong>PMID:16332963?</strong></a>]</span> and leads to increased cell survival following exposure to ionizing radiation <span class="citation" data-cites="PMID:16332963">[<a href="#ref-PMID:16332963" role="doc-biblioref"><strong>PMID:16332963?</strong></a>]</span>.
Point mutations in either the SET or MAR domains significantly reduced the ability of <em>SETMAR</em> to promote non-homologous end joining and DNA repair <span class="citation" data-cites="PMID:16332963 PMID:24573677 PMID:21491884">[<a href="#ref-PMID:16332963" role="doc-biblioref"><strong>PMID:16332963?</strong></a>,<a href="#ref-PMID:24573677" role="doc-biblioref"><strong>PMID:24573677?</strong></a>,<a href="#ref-PMID:21491884" role="doc-biblioref"><strong>PMID:21491884?</strong></a>]</span>, suggesting that both domains are needed for its role in DNA repair.
Another study found that overexpression of the isolated SET and MAR domains, but not of wild-type <em>SETMAR</em>, had a modest effect on NHEJ repair; overexpression of the SET domain slightly <em>decreased</em> NHEJ repair of a linearized plasmid in human cells, while overexpression of the <em>Mariner</em>-derived domain increased NHEJ relative to controls <span class="citation" data-cites="PMID:31238295">[<a href="#ref-PMID:31238295" role="doc-biblioref"><strong>PMID:31238295?</strong></a>]</span>.</p>
<p>Taken together, these results suggest that both the SET and transposase domains of primate <em>SETMAR</em> are important for <em>SETMAR</em>-mediated DNA repair.
The p.Leu103Phe missense mutation that differentiates C57BL/6J and DBA/2J (Table <a href="#tbl:nonsyn-diffs">1</a>) resides within the <em>Setmar</em> pre-SET domain and occurs at an amino acid residue that is predicted to be deleterious by SIFT <span class="citation" data-cites="PMID:12824425">[<a href="#ref-PMID:12824425" role="doc-biblioref"><strong>PMID:12824425?</strong></a>]</span>.
However, since the mouse <em>Setmar</em> ortholog lacks the <em>Mariner</em>-derived domain, we believe that the the p.Leu103Phe or p.Ser273Arg missense mutations are unlikely to affect C&gt;A mutation rates in the BXDs.
Moreover, we believe that the documented mutator phenotypes associated with <em>Ogg1</em>, as well as that gene’s known role in base-excision repair, make it more likely candidate to underlie the epistatic interaction with <em>Mutyh</em> we observed in this study.</p>
<h3 id="supplementary-figures">Supplementary Figures</h3>
<div id="fig:simulations" class="fignos">
<figure>
<img src="images/fig-power-simulations.png" style="width:7.5in" data-tag="1-figure supplement 1" alt="Figure 1-figure supplement 1: Simulations to assess the power of the aggregate mutation spectrum distance method. In each of 50 trials, we simulated genotypes at 1,000 biallelic loci on a toy population of either 50 or 100 haplotypes as follows. At every locus on every haplotype, we drew a single floating point value from a uniform distribution [0, 1). If that value was less than or equal to 0.5, we set the allele to be “A”; otherwise, we set the allele to be “B”. In each trial, we also simulated de novo germline mutations on the population of haplotypes, such that at a single locus g_i, we augmented the mutation rate of a particular k-mer by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate) on haplotypes carrying “A” alleles. We then applied the aggregate mutation spectrum distance method to these simulated data and asked if the adjusted cosine distance at locus g_i was greater than expected by chance. Given a specific combination of parameters, the y-axis denotes the fraction of 100 trials in which the simulated mutator allele could be detected at a significance threshold of p = 0.05. Shaded areas indicate the standard deviation of that fraction across 100 simulations." />
<figcaption aria-hidden="true"><span>Figure 1-figure supplement 1:</span> <strong>Simulations to assess the power of the aggregate mutation spectrum distance method.</strong>
In each of 50 trials, we simulated genotypes at 1,000 biallelic loci on a toy population of either 50 or 100 haplotypes as follows.
At every locus on every haplotype, we drew a single floating point value from a uniform distribution <span class="math inline">\([0, 1)\)</span>.
If that value was less than or equal to 0.5, we set the allele to be “A”; otherwise, we set the allele to be “B”.
In each trial, we also simulated <em>de novo</em> germline mutations on the population of haplotypes, such that at a single locus <span class="math inline">\(g_i\)</span>, we augmented the mutation rate of a particular <span class="math inline">\(k\)</span>-mer by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate) on haplotypes carrying “A” alleles.
We then applied the aggregate mutation spectrum distance method to these simulated data and asked if the adjusted cosine distance at locus <span class="math inline">\(g_i\)</span> was greater than expected by chance.
Given a specific combination of parameters, the y-axis denotes the fraction of 100 trials in which the simulated mutator allele could be detected at a significance threshold of p = 0.05.
Shaded areas indicate the standard deviation of that fraction across 100 simulations.</figcaption>
</figure>
</div>
<div id="fig:ihd_vs_qtl_power" class="fignos">
<figure>
<img src="images/fig-power-comparison.png" style="width:7.5in" data-tag="1-figure supplement 2" alt="Figure 1-figure supplement 2: Comparing power between the aggregate mutation spectrum distance method and QTL mapping. In each of 50 trials, we simulated genotypes at 1,000 biallelic loci on a toy population of 50 haplotypes as follows. At every locus on every haplotype, we drew a single floating point value from a uniform distribution [0, 1). If that value was less than or equal to 0.5, we set the allele to be “A”; otherwise, we set the allele to be “B”. In each trial, we also simulated de novo germline mutations on the population of haplotypes, such that at a single locus g_i, we augmented the rate of the specified mutation type by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate) on haplotypes carrying “A” alleles. We then applied the aggregate mutation spectrum distance method to these simulated data and asked if the adjusted cosine distance at locus g_i was greater than expected by chance. Similarly, in each trial, we used R/qtl2 to perform a genome scan for QTL and asked if the log-odds score at g_i was greater than expected by chance. Given a specific combination of parameters, the y-axis denotes the fraction of 100 trials in which the simulated mutator allele could be detected at a significance threshold of p = 0.05 (for IHD) or at an alpha of \frac{0.05}{7} (for QTL mapping). Shaded areas indicate the standard deviation of that fraction across 100 simulations." />
<figcaption aria-hidden="true"><span>Figure 1-figure supplement 2:</span> <strong>Comparing power between the aggregate mutation spectrum distance method and QTL mapping.</strong>
In each of 50 trials, we simulated genotypes at 1,000 biallelic loci on a toy population of 50 haplotypes as follows.
At every locus on every haplotype, we drew a single floating point value from a uniform distribution <span class="math inline">\([0, 1)\)</span>.
If that value was less than or equal to 0.5, we set the allele to be “A”; otherwise, we set the allele to be “B”.
In each trial, we also simulated <em>de novo</em> germline mutations on the population of haplotypes, such that at a single locus <span class="math inline">\(g_i\)</span>, we augmented the rate of the specified mutation type by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate) on haplotypes carrying “A” alleles.
We then applied the aggregate mutation spectrum distance method to these simulated data and asked if the adjusted cosine distance at locus <span class="math inline">\(g_i\)</span> was greater than expected by chance.
Similarly, in each trial, we used R/qtl2 to perform a genome scan for QTL and asked if the log-odds score at <span class="math inline">\(g_i\)</span> was greater than expected by chance.
Given a specific combination of parameters, the y-axis denotes the fraction of 100 trials in which the simulated mutator allele could be detected at a significance threshold of p = 0.05 (for IHD) or at an alpha of <span class="math inline">\(\frac{0.05}{7}\)</span> (for QTL mapping).
Shaded areas indicate the standard deviation of that fraction across 100 simulations.</figcaption>
</figure>
</div>
<div id="fig:ihd_vs_qtl_power_variable_counts" class="fignos">
<figure>
<img src="images/fig-power-comparison-variable-counts.png" style="width:7.5in" data-tag="1-figure supplement 3" alt="Figure 1-figure supplement 3: Comparing power between the aggregate mutation spectrum distance method and QTL mapping with variable counts of simulated mutations. In each of 50 trials, we simulated genotypes at 1,000 biallelic loci on a toy population of 50 or 100 haplotypes as follows. At every locus on every haplotype, we drew a single floating point value from a uniform distribution [0, 1). If that value was less than or equal to 0.5, we set the allele to be “A”; otherwise, we set the allele to be “B”. In each trial, we also simulated de novo germline mutations on the population of haplotypes, such that at a single locus g_i, we augmented the rate of the specified mutation type by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate) on haplotypes carrying “A” alleles. To more closely approximate the BXD RILs, the mean number of simulated mutations on each haplotype was allowed to vary by a factor of 20 (see Materials and Methods for more details). We then applied the aggregate mutation spectrum distance method to these simulated data and asked if the adjusted cosine distance at locus g_i was greater than expected by chance. Similarly, in each trial, we used R/qtl2 to perform a genome scan for QTL and asked if the log-odds score at g_i was greater than expected by chance. Given a specific combination of parameters, the y-axis denotes the fraction of 100 trials in which the simulated mutator allele could be detected at a significance threshold of p = 0.05 (for IHD) or at an alpha of \frac{0.05}{7} (for QTL mapping). Shaded areas indicate the standard deviation of that fraction across 100 simulations." />
<figcaption aria-hidden="true"><span>Figure 1-figure supplement 3:</span> <strong>Comparing power between the aggregate mutation spectrum distance method and QTL mapping with variable counts of simulated mutations.</strong>
In each of 50 trials, we simulated genotypes at 1,000 biallelic loci on a toy population of 50 or 100 haplotypes as follows.
At every locus on every haplotype, we drew a single floating point value from a uniform distribution <span class="math inline">\([0, 1)\)</span>.
If that value was less than or equal to 0.5, we set the allele to be “A”; otherwise, we set the allele to be “B”.
In each trial, we also simulated <em>de novo</em> germline mutations on the population of haplotypes, such that at a single locus <span class="math inline">\(g_i\)</span>, we augmented the rate of the specified mutation type by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate) on haplotypes carrying “A” alleles.
To more closely approximate the BXD RILs, the mean number of simulated mutations on each haplotype was allowed to vary by a factor of 20 (see Materials and Methods for more details).
We then applied the aggregate mutation spectrum distance method to these simulated data and asked if the adjusted cosine distance at locus <span class="math inline">\(g_i\)</span> was greater than expected by chance.
Similarly, in each trial, we used R/qtl2 to perform a genome scan for QTL and asked if the log-odds score at <span class="math inline">\(g_i\)</span> was greater than expected by chance.
Given a specific combination of parameters, the y-axis denotes the fraction of 100 trials in which the simulated mutator allele could be detected at a significance threshold of p = 0.05 (for IHD) or at an alpha of <span class="math inline">\(\frac{0.05}{7}\)</span> (for QTL mapping).
Shaded areas indicate the standard deviation of that fraction across 100 simulations.</figcaption>
</figure>
</div>
<div id="fig:ihd_vs_qtl_power_variable_afs" class="fignos">
<figure>
<img src="images/fig-power-comparison-variable-afs.png" style="width:7.5in" data-tag="1-figure supplement 4" alt="Figure 1-figure supplement 4: Comparing power between the aggregate mutation spectrum distance method and QTL mapping with variable marker allele frequencies. In each of 50 trials, we simulated genotypes at 1,000 biallelic loci on a toy population of 100 haplotypes as follows. At every locus on every haplotype, we drew a single floating point value from a uniform distribution [0, 1). If that value was less than or equal to f, we set the allele to be “A”; otherwise, we set the allele to be “B”. We allowed f (the expected frequency of “A” alleles at each marker) to be either 0.1, 0.2, or 0.5 in these simulations; f is denoted as “Allele frequency” in each subplot. In each trial, we also simulated de novo germline mutations on the population of haplotypes, such that at a single locus g_i, we augmented the rate of the specified mutation type by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate) on haplotypes carrying “A” alleles. We then applied the aggregate mutation spectrum distance method to these simulated data and asked if the adjusted cosine distance at locus g_i was greater than expected by chance. Similarly, in each trial, we used R/qtl2 to perform a genome scan for QTL and asked if the log-odds score at g_i was greater than expected by chance. Given a specific combination of parameters, the y-axis denotes the fraction of 100 trials in which the simulated mutator allele could be detected at a significance threshold of p = 0.05 (for IHD) or at an alpha of \frac{0.05}{7} (for QTL mapping). Shaded areas indicate the standard deviation of that fraction across 100 simulations." />
<figcaption aria-hidden="true"><span>Figure 1-figure supplement 4:</span> <strong>Comparing power between the aggregate mutation spectrum distance method and QTL mapping with variable marker allele frequencies.</strong>
In each of 50 trials, we simulated genotypes at 1,000 biallelic loci on a toy population of 100 haplotypes as follows.
At every locus on every haplotype, we drew a single floating point value from a uniform distribution <span class="math inline">\([0, 1)\)</span>.
If that value was less than or equal to <span class="math inline">\(f\)</span>, we set the allele to be “A”; otherwise, we set the allele to be “B”.
We allowed <span class="math inline">\(f\)</span> (the expected frequency of “A” alleles at each marker) to be either 0.1, 0.2, or 0.5 in these simulations; <span class="math inline">\(f\)</span> is denoted as “Allele frequency” in each subplot.
In each trial, we also simulated <em>de novo</em> germline mutations on the population of haplotypes, such that at a single locus <span class="math inline">\(g_i\)</span>, we augmented the rate of the specified mutation type by the specified effect size (an effect size of 1.5 indicates a 50% increase in the mutation rate) on haplotypes carrying “A” alleles.
We then applied the aggregate mutation spectrum distance method to these simulated data and asked if the adjusted cosine distance at locus <span class="math inline">\(g_i\)</span> was greater than expected by chance.
Similarly, in each trial, we used R/qtl2 to perform a genome scan for QTL and asked if the log-odds score at <span class="math inline">\(g_i\)</span> was greater than expected by chance.
Given a specific combination of parameters, the y-axis denotes the fraction of 100 trials in which the simulated mutator allele could be detected at a significance threshold of p = 0.05 (for IHD) or at an alpha of <span class="math inline">\(\frac{0.05}{7}\)</span> (for QTL mapping).
Shaded areas indicate the standard deviation of that fraction across 100 simulations.</figcaption>
</figure>
</div>
<div id="fig:qtl-scans" class="fignos">
<figure>
<img src="images/fig-qtl-scans.png" style="width:7.5in" data-tag="2-figure supplement 1" alt="Figure 2-figure supplement 1: Quantitative trait locus scans for mutation spectrum phenotypes. Using the BXDs with D genotypes at rs27509845 (the marker with the highest cosine distance on chromosome 4; n = 66 BXDs, 42,171 total mutations), we used R/qtl2 to perform QTL scans for the fractions of each 1-mer mutation type. QTL scans also included a kinship matrix (that contained the pairwise genetic similarity between each pair of BXDs, calculated using the leave-one-chromosome-out method) as a random effect term using the kinship keyword argument in the scan1 function. Plots show the log-odds (LOD) score at every genotyped marker in blue; the dotted black line represents the genome-wide LOD significance threshold (established using 1,000 permutations at an alpha of \frac{0.05}{7} to account for the fact that 7 separate association tests were performed.)" />
<figcaption aria-hidden="true"><span>Figure 2-figure supplement 1:</span> <strong>Quantitative trait locus scans for mutation spectrum phenotypes.</strong>
Using the BXDs with D genotypes at <code>rs27509845</code> (the marker with the highest cosine distance on chromosome 4; n = 66 BXDs, 42,171 total mutations), we used R/qtl2 to perform QTL scans for the fractions of each 1-mer mutation type.
QTL scans also included a kinship matrix (that contained the pairwise genetic similarity between each pair of BXDs, calculated using the leave-one-chromosome-out method) as a random effect term using the <code>kinship</code> keyword argument in the <code>scan1</code> function.
Plots show the log-odds (LOD) score at every genotyped marker in blue; the dotted black line represents the genome-wide LOD significance threshold (established using 1,000 permutations at an alpha of <span class="math inline">\(\frac{0.05}{7}\)</span> to account for the fact that 7 separate association tests were performed.)</figcaption>
</figure>
</div>
<div id="fig:spectra-comparison-all" class="fignos">
<figure>
<img src="images/fig-spectra-comparison-bxd-all.png" style="width:7.5in" data-tag="3-figure supplement 1" alt="Figure 3-figure supplement 1: Mutation spectra comparison in BXD strains. Fractions of de novo germline mutations in BXDs with either D or B genotypes at markers rs27509845 and rs46276051, stratified by mutation type." />
<figcaption aria-hidden="true"><span>Figure 3-figure supplement 1:</span> <strong>Mutation spectra comparison in BXD strains.</strong>
Fractions of <em>de novo</em> germline mutations in BXDs with either <em>D</em> or <em>B</em> genotypes at markers <code>rs27509845</code> and <code>rs46276051</code>, stratified by mutation type.</figcaption>
</figure>
</div>
<div id="fig:spectra-comparison-mgp" class="fignos">
<figure>
<img src="images/fig-spectra-comparison-mgp.png" style="width:7.5in" data-tag="3—figure supplement 2" alt="Figure 3—figure supplement 2: Mutation spectra comparison in Sanger Mouse Genomes Project strains. Fractions of de novo germline mutations in Sanger MGP strains with either D or B haplotypes at the chromosome 4 and chromosome 6 mutator loci, stratified by mutation type." />
<figcaption aria-hidden="true"><span>Figure 3—figure supplement 2:</span> <strong>Mutation spectra comparison in Sanger Mouse Genomes Project strains.</strong>
Fractions of <em>de novo</em> germline mutations in Sanger MGP strains with either <em>D</em> or <em>B</em> haplotypes at the chromosome 4 and chromosome 6 mutator loci, stratified by mutation type.</figcaption>
</figure>
</div>
<div id="fig:wild-afs" class="fignos">
<figure>
<img src="images/fig-wild-afs.png" style="width:7.5in" data-tag="3-figure supplement 3" alt="Figure 3-figure supplement 3: Frequency of nonsynonymous DNA repair mutations in wild mice. Alternate allele frequencies of each nonsynonymous DNA repair mutation overlapping the chromosome 6 mutator locus were calculated in populations of wild-derived mice from Harr et al. [PMID:27622383?]. Numbers of mice in each subpopulation are shown in parentheses. Mmc (Mus musculus castaneus), Mmd (Mus musculus domesticus), Mmm (Mus musculus musculus), and Ms (Mus spretus). The Mbd4 p.Asp129Asn mutation was not observed in any wild populations." />
<figcaption aria-hidden="true"><span>Figure 3-figure supplement 3:</span> <strong>Frequency of nonsynonymous DNA repair mutations in wild mice.</strong> Alternate allele frequencies of each nonsynonymous DNA repair mutation overlapping the chromosome 6 mutator locus were calculated in populations of wild-derived mice from Harr et al. <span class="citation" data-cites="PMID:27622383">[<a href="#ref-PMID:27622383" role="doc-biblioref"><strong>PMID:27622383?</strong></a>]</span>. Numbers of mice in each subpopulation are shown in parentheses. <em>Mmc</em> (<em>Mus musculus castaneus</em>), <em>Mmd</em> (<em>Mus musculus domesticus</em>), <em>Mmm</em> (<em>Mus musculus musculus</em>), and <em>Ms</em> (<em>Mus spretus</em>). The <em>Mbd4</em> p.Asp129Asn mutation was not observed in any wild populations.</figcaption>
</figure>
</div>
<h3 id="supplementary-tables">Supplementary Tables</h3>
<div id="tbl:eqtl-results" class="tablenos">
<table id="tbl:eqtl-results" style="width:100%;" data-tag="supplement 1">
<caption><span>Table supplement 1:</span> Significant cis-eQTLs for DNA repair genes in various tissues identified using GeneNetwork. </caption>
<colgroup>
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
<col style="width: 16%" />
</colgroup>
<thead>
<tr class="header">
<th>Gene name</th>
<th>Tissue name</th>
<th># BXDs with expression data</th>
<th>Top significant marker</th>
<th>-log10(p) at top significant marker (GEMMA)</th>
<th>Additive effect of D allele on expression (GEMMA)</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td><em>Ogg1</em></td>
<td>Kidney</td>
<td>53</td>
<td><code>rsm10000004188</code></td>
<td>12.89</td>
<td>-0.180</td>
</tr>
<tr class="even">
<td><em>Ogg1</em></td>
<td>Liver</td>
<td>50</td>
<td><code>rsm10000004188</code></td>
<td>13.57</td>
<td>-0.155</td>
</tr>
<tr class="odd">
<td><em>Ogg1</em></td>
<td>Spleen</td>
<td>79</td>
<td><code>rsm10000003418</code></td>
<td>4.73</td>
<td>-0.056</td>
</tr>
<tr class="even">
<td><em>Ogg1</em></td>
<td>Gastrointestinal</td>
<td>46</td>
<td><code>rs4173870</code></td>
<td>5.43</td>
<td>-0.048</td>
</tr>
<tr class="odd">
<td><em>Fancd2</em></td>
<td>Gastrointestinal</td>
<td>46</td>
<td><code>rsm10000004199</code></td>
<td>8.60</td>
<td>0.133</td>
</tr>
<tr class="even">
<td><em>Ogg1</em></td>
<td>Hippocampus</td>
<td>67</td>
<td><code>rsm10000004188</code></td>
<td>16.50</td>
<td>-0.165</td>
</tr>
<tr class="odd">
<td><em>Rad18</em></td>
<td>Hippocampus</td>
<td>67</td>
<td><code>rsm10000003463</code></td>
<td>6.32</td>
<td>0.068</td>
</tr>
<tr class="even">
<td><em>Setmar</em></td>
<td>Hippocampus</td>
<td>67</td>
<td><code>rs13478947</code></td>
<td>11.03</td>
<td>0.141</td>
</tr>
</tbody>
</table>
</div>
<div id="tbl:sv-overlap" class="tablenos">
<table id="tbl:sv-overlap" data-tag="supplement 2">
<caption><span>Table supplement 2:</span> Large structural variants overlapping protein-coding genes in the mutator locus on chromosome 6 All coordinates are with respect to GRCm39/mm39. </caption>
<thead>
<tr class="header">
<th>SV start</th>
<th>SV end</th>
<th>SV type</th>
<th>Gene name(s)</th>
<th>Overlaps exon?</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>108,437,600</td>
<td>108,437,652</td>
<td>DEL</td>
<td><em>Gm35165</em>, <em>Itpr1</em></td>
<td>Yes</td>
</tr>
<tr class="even">
<td>110,617,754</td>
<td>110,618,347</td>
<td>DEL</td>
<td><em>Gm20387</em></td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>113,110,214</td>
<td>113,110,215</td>
<td>INS</td>
<td><em>Setd5</em></td>
<td>Yes</td>
</tr>
<tr class="even">
<td>116,217,366</td>
<td>116,217,367</td>
<td>INS</td>
<td><em>Gm52873</em>, <em>Washc2</em></td>
<td>Yes</td>
</tr>
<tr class="odd">
<td>112,605,594</td>
<td>112,605,595</td>
<td>INS</td>
<td><em>Rad18</em></td>
<td>No</td>
</tr>
<tr class="even">
<td>112,629,618</td>
<td>112,636,619</td>
<td>DEL</td>
<td><em>Rad18</em></td>
<td>No</td>
</tr>
<tr class="odd">
<td>115,823,860</td>
<td>115,824,076</td>
<td>DEL</td>
<td><em>Mbd4</em></td>
<td>No</td>
</tr>
</tbody>
</table>
</div>
<h2 class="page_break_before" id="references">References</h2>
<!-- Explicitly insert bibliography here -->
<div id="refs" class="references csl-bib-body" role="doc-bibliography">
<div id="ref-12CXVVJzy" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">1. </div><div class="csl-right-inline"><strong>A naturally occurring variant of&lt;i&gt;MBD4&lt;/i&gt;causes maternal germline hypermutation in primates</strong> <div class="csl-block">Alexandra M Stendahl, Rashesh Sanghvi, Samuel Peterson, Karina Ray, Ana C Lima, Raheleh Rahbari, Donald F Conrad</div> <em>Cold Spring Harbor Laboratory</em> (2023-03-29) <a href="https://doi.org/gr3ghz">https://doi.org/gr3ghz</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2023.03.27.534460">10.1101/2023.03.27.534460</a></div></div>
</div>
<div id="ref-LF8DCRcP" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">2. </div><div class="csl-right-inline"><strong>Spontaneous Mutation Accumulation Studies in Evolutionary Genetics</strong> <div class="csl-block">Daniel L Halligan, Peter D Keightley</div> <em>Annual Review of Ecology, Evolution, and Systematics</em> (2009-12-01) <a href="https://doi.org/dvrjz8">https://doi.org/dvrjz8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1146/annurev.ecolsys.39.110707.173437">10.1146/annurev.ecolsys.39.110707.173437</a></div></div>
</div>
<div id="ref-8KLgRyLH" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">3. </div><div class="csl-right-inline"><strong>Resolution of structural variation in diverse mouse genomes reveals chromatin remodeling due to transposable elements</strong> <div class="csl-block">Ardian Ferraj, Peter A Audano, Parithi Balachandran, Anne Czechanski, Jacob I Flores, Alexander A Radecki, Varun Mosur, David S Gordon, Isha A Walawalkar, Evan E Eichler, … Christine R Beck</div> <em>Cell Genomics</em> (2023-05) <a href="https://doi.org/gr3t7b">https://doi.org/gr3t7b</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1016/j.xgen.2023.100291">10.1016/j.xgen.2023.100291</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/37228752">37228752</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10203049">PMC10203049</a></div></div>
</div>
<div id="ref-8vP4Q1ja" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">4. </div><div class="csl-right-inline"><strong>Defects in 8-oxo-guanine repair pathway cause high frequency of C &amp;gt; A substitutions in neuroblastoma</strong> <div class="csl-block">Marlinde L van den Boogaard, Rurika Oka, Anne Hakkert, Linda Schild, Marli E Ebus, Michael R van Gerven, Danny A Zwijnenburg, Piet Molenaar, Lieke L Hoyng, MEmmy M Dolman, … Jan J Molenaar</div> <em>Proceedings of the National Academy of Sciences</em> (2021-09-03) <a href="https://doi.org/grtcs9">https://doi.org/grtcs9</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1073/pnas.2007898118">10.1073/pnas.2007898118</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34479993">34479993</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8433536">PMC8433536</a></div></div>
</div>
<div id="ref-1LexfAxj" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">5. </div><div class="csl-right-inline"><strong>Array programming with NumPy</strong> <div class="csl-block">Charles R Harris, KJarrod Millman, Stéfan J van der Walt, Ralf Gommers, Pauli Virtanen, David Cournapeau, Eric Wieser, Julian Taylor, Sebastian Berg, Nathaniel J Smith, … Travis E Oliphant</div> <em>Nature</em> (2020-09-16) <a href="https://doi.org/ghbzf2">https://doi.org/ghbzf2</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1038/s41586-020-2649-2">10.1038/s41586-020-2649-2</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/32939066">32939066</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC7759461">PMC7759461</a></div></div>
</div>
<div id="ref-2z2qDwzz" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">6. </div><div class="csl-right-inline"><strong>pandas-dev/pandas: Pandas</strong> <div class="csl-block">The Pandas Development Team</div> <em>Zenodo</em> (2023-08-30) <a href="https://doi.org/ggt8bh">https://doi.org/ggt8bh</a> <div class="csl-block">DOI: <a href="https://doi.org/10.5281/zenodo.3509134">10.5281/zenodo.3509134</a></div></div>
</div>
<div id="ref-1026Gxdsi" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">7. </div><div class="csl-right-inline"><strong>Matplotlib: A 2D Graphics Environment</strong> <div class="csl-block">John D Hunter</div> <em>Computing in Science &amp;amp; Engineering</em> (2007) <a href="https://doi.org/drbjhg">https://doi.org/drbjhg</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1109/mcse.2007.55">10.1109/mcse.2007.55</a></div></div>
</div>
<div id="ref-lF0Fr9LL" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">8. </div><div class="csl-right-inline"><strong>Scikit-learn: Machine Learning in Python</strong> <div class="csl-block">Fabian Pedregosa, Gaël Varoquaux, Alexandre Gramfort, Vincent Michel, Bertrand Thirion, Olivier Grisel, Mathieu Blondel, Peter Prettenhofer, Ron Weiss, Vincent Dubourg, … Édouard Duchesnay</div> <em>Journal of Machine Learning Research</em> (2011) <a href="http://jmlr.org/papers/v12/pedregosa11a.html">http://jmlr.org/papers/v12/pedregosa11a.html</a></div>
</div>
<div id="ref-34sZyTFd" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">9. </div><div class="csl-right-inline"><strong>pandera: Statistical Data Validation of Pandas Dataframes</strong> <div class="csl-block">Niels Bantilan</div> <em>Proceedings of the Python in Science Conference</em> (2020) <a href="https://doi.org/grr54q">https://doi.org/grr54q</a> <div class="csl-block">DOI: <a href="https://doi.org/10.25080/majora-342d178e-010">10.25080/majora-342d178e-010</a></div></div>
</div>
<div id="ref-169TfvTKZ" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">10. </div><div class="csl-right-inline"><strong>seaborn: statistical data visualization</strong> <div class="csl-block">Michael Waskom</div> <em>Journal of Open Source Software</em> (2021-04-06) <a href="https://doi.org/gjqn3g">https://doi.org/gjqn3g</a> <div class="csl-block">DOI: <a href="https://doi.org/10.21105/joss.03021">10.21105/joss.03021</a></div></div>
</div>
<div id="ref-189xgFrlN" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">11. </div><div class="csl-right-inline"><strong>Numba</strong> <div class="csl-block">Siu Kwan Lam, Antoine Pitrou, Stanley Seibert</div> <em>Proceedings of the Second Workshop on the LLVM Compiler Infrastructure in HPC</em> (2015-11-15) <a href="https://doi.org/gf3nks">https://doi.org/gf3nks</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1145/2833157.2833162">10.1145/2833157.2833162</a></div></div>
</div>
<div id="ref-fVLvtRTc" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">12. </div><div class="csl-right-inline"><strong>Sustainable data analysis with Snakemake</strong> <div class="csl-block">Felix Mölder, Kim Philipp Jablonski, Brice Letcher, Michael B Hall, Christopher H Tomkins-Tinch, Vanessa Sochat, Jan Forster, Soohyun Lee, Sven O Twardziok, Alexander Kanitz, … Johannes Köster</div> <em>F1000Research</em> (2021-01-18) <a href="https://doi.org/gjjkwv">https://doi.org/gjjkwv</a> <div class="csl-block">DOI: <a href="https://doi.org/10.12688/f1000research.29032.1">10.12688/f1000research.29032.1</a> · PMID: <a href="https://www.ncbi.nlm.nih.gov/pubmed/34035898">34035898</a> · PMCID: <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC8114187">PMC8114187</a></div></div>
</div>
<div id="ref-TYYBVYN3" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">13. </div><div class="csl-right-inline"><strong>A natural mutator allele shapes mutation spectrum variation in mice</strong> <div class="csl-block">Tom Sasani</div> (2023-01-24) <a href="https://github.com/tomsasani/bxd_mutator_manuscript">https://github.com/tomsasani/bxd_mutator_manuscript</a></div>
</div>
<div id="ref-z6H1lPG4" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">14. </div><div class="csl-right-inline"><strong>tomsasani/bxd_mutator_manuscript: Final figure generation updates prior to publication</strong> <div class="csl-block">Tom Sasani</div> <em>Zenodo</em> (2022-02-01) <a href="https://doi.org/grrwv8">https://doi.org/grrwv8</a> <div class="csl-block">DOI: <a href="https://doi.org/10.5281/zenodo.5941048">10.5281/zenodo.5941048</a></div></div>
</div>
<div id="ref-t8fRZlGg" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">15. </div><div class="csl-right-inline"><strong>BXD Genotype / WebQTL</strong> <a href="https://gn1.genenetwork.org/dbdoc/BXDGeno.html">https://gn1.genenetwork.org/dbdoc/BXDGeno.html</a></div>
</div>
<div id="ref-8Z2SLpxu" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">16. </div><div class="csl-right-inline"><strong>Genetics and Probability in Animal Breeding Experiments</strong> <a href="https://link.springer.com/book/10.1007/978-1-349-04904-2">https://link.springer.com/book/10.1007/978-1-349-04904-2</a></div>
</div>
<div id="ref-V7hHut0M" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">17. </div><div class="csl-right-inline"><strong>ENA Browser</strong> <a href="https://www.ebi.ac.uk/ena/browser/view/PRJEB45429">https://www.ebi.ac.uk/ena/browser/view/PRJEB45429</a></div>
</div>
<div id="ref-ZjBdGGm0" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">18. </div><div class="csl-right-inline"><strong>bx-python</strong> <div class="csl-block">Taylor Lab at Johns Hopkins University</div> (2023-08-11) <a href="https://github.com/bxlab/bx-python">https://github.com/bxlab/bx-python</a></div>
</div>
<div id="ref-tW44yAw8" class="csl-entry" role="doc-biblioentry">
<div class="csl-left-margin">19. </div><div class="csl-right-inline"><strong>GeneNetwork: a continuously updated tool for systems genetics analyses</strong> <div class="csl-block">Pamela M Watson, David G Ashbrook</div> <em>Cold Spring Harbor Laboratory</em> (2020-12-24) <a href="https://doi.org/gsnpr5">https://doi.org/gsnpr5</a> <div class="csl-block">DOI: <a href="https://doi.org/10.1101/2020.12.23.424047">10.1101/2020.12.23.424047</a></div></div>
</div>
</div>
<!-- default theme -->

<style>
  /* import google fonts */
  @import url("https://fonts.googleapis.com/css?family=Open+Sans:400,600,700");
  @import url('https://fonts.googleapis.com/css?family=Libre+Franklin:100,200,300,400,600,700');
  @import url('https://fonts.googleapis.com/css?family=Newsreader:100,200,300,400,600,700');
  @import url('https://fonts.googleapis.com/css?family=Roboto:100,200,300,400,600,700');

  /* &family=Assistant:wght@300&family=Josefin+Sans:wght@100&family=Rubik:wght@300&display=swap'); */
  @import url("https://fonts.googleapis.com/css?family=Source+Code+Pro");

  /* -------------------------------------------------- */
  /* global */
  /* -------------------------------------------------- */

  /* all elements */
  * {
    /* force sans-serif font unless specified otherwise */
    font-family: "Open Sans", "Times", sans-serif;

    /* prevent text inflation on some mobile browsers */
    -webkit-text-size-adjust: none !important;
    -moz-text-size-adjust: none !important;
    -o-text-size-adjust: none !important;
    text-size-adjust: none !important;
  }

  @media only screen {
    /* "page" element */
    body {
      position: relative;
      box-sizing: border-box;
      font-size: 13pt;
      line-height: 1.5;
      max-width: 10in;
      margin: 20px auto;
      padding: 40px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* "page" element */
    body {
      padding: 20px;
      margin: 0;
      border-radius: 0;
      border: none;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05) inset;
      background: none;
    }
  }

  /* -------------------------------------------------- */
  /* headings */
  /* -------------------------------------------------- */

  /* all headings */
  h1,
  h2,
  h3,
  h4,
  h5,
  h6 {
    margin: 20px 0;
    padding: 0;
    font-weight: bold;
  }

  /* biggest heading */
  h1 {
    margin: 40px 0;
    text-align: center;
  }

  /* second biggest heading */
  h2 {
    margin-top: 30px;
    padding-bottom: 5px;
    border-bottom: solid 1px #bdbdbd;
  }

  /* heading font sizes */
  h1 {
    font-size: 2em;
  }
  h2 {
    font-size: 1.5em;
  }
  h3 {
    font-size: 1.35em;
  }
  h4 {
    font-size: 1.25em;
  }
  h5 {
    font-size: 1.15em;
  }
  h6 {
    font-size: 1em;
  }

  /* -------------------------------------------------- */
  /* manuscript header */
  /* -------------------------------------------------- */

  /* manuscript title */
  header > h1 {
    margin: 0;
  }

  /* manuscript title caption text (ie "automatically generated on") */
  header + p {
    text-align: center;
    margin-top: 10px;
  }

  /* -------------------------------------------------- */
  /* text elements */
  /* -------------------------------------------------- */

  /* links */
  a {
    color: #2196f3;
    overflow-wrap: break-word;
  }

  /* superscripts and subscripts */
  sub,
  sup {
    /* prevent from affecting line height */
    line-height: 0;
  }

  /* unordered and ordered lists*/
  ul,
  ol {
    padding-left: 20px;
  }

  /* class for styling text semibold */
  .semibold {
    font-weight: 600;
  }

  /* class for styling elements horizontally left aligned */
  .left {
    display: block;
    text-align: left;
    margin-left: auto;
    margin-right: 0;
    justify-content: left;
  }

  /* class for styling elements horizontally centered */
  .center {
    display: block;
    text-align: center;
    margin-left: auto;
    margin-right: auto;
    justify-content: center;
  }

  /* class for styling elements horizontally right aligned */
  .right {
    display: block;
    text-align: right;
    margin-left: 0;
    margin-right: auto;
    justify-content: right;
  }

  /* -------------------------------------------------- */
  /* section elements */
  /* -------------------------------------------------- */

  /* horizontal divider line */
  hr {
    border: none;
    height: 1px;
    background: #bdbdbd;
  }

  /* paragraphs, horizontal dividers, figures, tables, code */
  p,
  hr,
  figure,
  table,
  pre {
    /* treat all as "paragraphs", with consistent vertical margins */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* figures */
  /* -------------------------------------------------- */

  /* figure */
  figure {
    max-width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure caption */
  figcaption {
    padding: 0;
    padding-top: 10px;
  }

  /* figure image element */
  figure > img,
  figure > svg {
    max-width: 100%;
    display: block;
    margin-left: auto;
    margin-right: auto;
  }

  /* figure auto-number */
  img + figcaption > span:first-of-type,
  svg + figcaption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* tables */
  /* -------------------------------------------------- */

  /* table */
  table {
    border-collapse: collapse;
    border-spacing: 0;
    width: 100%;
    margin-left: auto;
    margin-right: auto;
  }

  /* table cells */
  th,
  td {
    border: solid 1px #bdbdbd;
    padding: 10px;
    /* squash table if too wide for page by forcing line breaks */
    overflow-wrap: break-word;
    word-break: break-word;
  }

  /* header row and even rows */
  th,
  tr:nth-child(2n) {
    background-color: #fafafa;
  }

  /* odd rows */
  tr:nth-child(2n + 1) {
    background-color: #ffffff;
  }

  /* table caption */
  caption {
    text-align: left;
    padding: 0;
    padding-bottom: 10px;
  }

  /* table auto-number */
  table > caption > span:first-of-type {
    font-weight: bold;
    margin-right: 5px;
  }

  /* -------------------------------------------------- */
  /* code */
  /* -------------------------------------------------- */

  /* multi-line code block */
  pre {
    padding: 10px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
    break-inside: avoid;
    text-align: left;
  }

  /* inline code, ie code within normal text */
  :not(pre) > code {
    padding: 0 4px;
    background-color: #eeeeee;
    color: #000000;
    border-radius: 5px;
  }

  /* code text */
  /* apply all children, to reach syntax highlighting sub-elements */
  code,
  code * {
    /* force monospace font */
    font-family: "Source Code Pro", "Courier New", monospace;
    font-size: 10;
  }

  /* -------------------------------------------------- */
  /* quotes */
  /* -------------------------------------------------- */

  /* quoted text */
  blockquote {
    margin: 0;
    padding: 0;
    border-left: 4px solid #bdbdbd;
    padding-left: 16px;
    break-inside: avoid;
  }

  /* -------------------------------------------------- */
  /* banners */
  /* -------------------------------------------------- */

  /* info banners */
  .banner {
    box-sizing: border-box;
    display: block;
    position: relative;
    width: 100%;
    margin-top: 20px;
    margin-bottom: 20px;
    padding: 20px;
    text-align: center;
  }

  /* paragraph in banner */
  .banner > p {
    margin: 0;
  }

  /* -------------------------------------------------- */
  /* highlight colors */
  /* -------------------------------------------------- */

  .white {
    background: #ffffff;
  }
  .lightgrey {
    background: #eeeeee;
  }
  .grey {
    background: #757575;
  }
  .darkgrey {
    background: #424242;
  }
  .black {
    background: #000000;
  }
  .lightred {
    background: #ffcdd2;
  }
  .lightyellow {
    background: #ffecb3;
  }
  .lightgreen {
    background: #dcedc8;
  }
  .lightblue {
    background: #e3f2fd;
  }
  .lightpurple {
    background: #f3e5f5;
  }
  .red {
    background: #f44336;
  }
  .orange {
    background: #ff9800;
  }
  .yellow {
    background: #ffeb3b;
  }
  .green {
    background: #4caf50;
  }
  .blue {
    background: #2196f3;
  }
  .purple {
    background: #9c27b0;
  }
  .white,
  .lightgrey,
  .lightred,
  .lightyellow,
  .lightgreen,
  .lightblue,
  .lightpurple,
  .orange,
  .yellow,
  .white a,
  .lightgrey a,
  .lightred a,
  .lightyellow a,
  .lightgreen a,
  .lightblue a,
  .lightpurple a,
  .orange a,
  .yellow a {
    color: #000000;
  }
  .grey,
  .darkgrey,
  .black,
  .red,
  .green,
  .blue,
  .purple,
  .grey a,
  .darkgrey a,
  .black a,
  .red a,
  .green a,
  .blue a,
  .purple a {
    color: #ffffff;
  }

  /* -------------------------------------------------- */
  /* buttons */
  /* -------------------------------------------------- */

  /* class for styling links like buttons */
  .button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    margin: 5px;
    padding: 10px 20px;
    font-size: 0.75em;
    font-weight: 600;
    text-transform: uppercase;
    text-decoration: none;
    letter-spacing: 1px;
    background: none;
    color: #2196f3;
    border: solid 1px #bdbdbd;
    border-radius: 5px;
  }

  /* buttons when hovered */
  .button:hover:not([disabled]),
  .icon_button:hover:not([disabled]) {
    cursor: pointer;
    background: #f5f5f5;
  }

  /* buttons when disabled */
  .button[disabled],
  .icon_button[disabled] {
    opacity: 0.35;
    pointer-events: none;
  }

  /* class for styling buttons containg only single icon */
  .icon_button {
    display: inline-flex;
    justify-content: center;
    align-items: center;
    text-decoration: none;
    margin: 0;
    padding: 0;
    background: none;
    border-radius: 5px;
    border: none;
    width: 20px;
    height: 20px;
    min-width: 20px;
    min-height: 20px;
  }

  /* icon button inner svg image */
  .icon_button > svg {
    height: 16px;
  }

  /* -------------------------------------------------- */
  /* icons */
  /* -------------------------------------------------- */

  /* class for styling icons inline with text */
  .inline_icon {
    height: 1em;
    position: relative;
    top: 0.125em;
  }

  /* -------------------------------------------------- */
  /* references */
  /* -------------------------------------------------- */

  .csl-entry {
    margin-top: 15px;
    margin-bottom: 15px;
  }

  /* -------------------------------------------------- */
  /* print control */
  /* -------------------------------------------------- */

  @media print {
    @page {
      /* suggested printing margin */
      margin: 0.5in;
    }

    /* document and "page" elements */
    html,
    body {
      margin: 0;
      padding: 0;
      width: 100%;
      height: 100%;
    }

    /* "page" element */
    body {
      font-size: 11pt !important;
      line-height: 1.35;
    }

    /* all headings */
    h1,
    h2,
    h3,
    h4,
    h5,
    h6 {
      margin: 15px 0;
    }

    /* figures and tables */
    figure,
    table {
      font-size: 0.85em;
    }

    /* table cells */
    th,
    td {
      padding: 5px;
    }

    /* shrink font awesome icons */
    i.fas,
    i.fab,
    i.far,
    i.fal {
      transform: scale(0.85);
    }

    /* decrease banner margins */
    .banner {
      margin-top: 15px;
      margin-bottom: 15px;
      padding: 15px;
    }

    /* class for centering an element vertically on its own page */
    .page_center {
      margin: auto;
      width: 100%;
      height: 100%;
      display: flex;
      align-items: center;
      vertical-align: middle;
      break-before: page;
      break-after: page;
    }

    /* always insert a page break before the element */
    .page_break_before {
      break-before: page;
    }

    /* always insert a page break after the element */
    .page_break_after {
      break-after: page;
    }

    /* avoid page break before the element */
    .page_break_before_avoid {
      break-before: avoid;
    }

    /* avoid page break after the element */
    .page_break_after_avoid {
      break-after: avoid;
    }

    /* avoid page break inside the element */
    .page_break_inside_avoid {
      break-inside: avoid;
    }
  }

  /* -------------------------------------------------- */
  /* override pandoc css quirks */
  /* -------------------------------------------------- */

  .sourceCode {
    /* prevent unsightly overflow in wide code blocks */
    overflow: auto !important;
  }

  div.sourceCode {
    /* prevent background fill on top-most code block  container */
    background: none !important;
  }

  .sourceCode * {
    /* force consistent line spacing */
    line-height: 1.5 !important;
  }

  div.sourceCode {
    /* style code block margins same as <pre> element */
    margin-top: 20px;
    margin-bottom: 20px;
  }

  /* -------------------------------------------------- */
  /* tablenos */
  /* -------------------------------------------------- */

  /* tablenos wrapper */
  .tablenos {
    width: 100%;
    margin: 20px 0;
  }

  .tablenos > table {
    /* move margins from table to table_wrapper to allow margin collapsing */
    margin: 0;
  }

  @media only screen {
    /* tablenos wrapper */
    .tablenos {
      /* show scrollbar on tables if necessary to prevent overflow */
      overflow-x: auto !important;
    }

    .tablenos th,
    .tablenos td {
      overflow-wrap: unset !important;
      word-break: unset !important;
    }

    /* table in wrapper */
    .tablenos table,
    .tablenos table * {
      /* don't break table words */
      overflow-wrap: normal !important;
    }
  }
</style>
<!-- 
    Plugin Core

    Functions needed for and shared across all first-party plugins.
-->

<script>
  // get element that is target of hash (from link element or url)
  function getHashTarget(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector(`[id="${id}"]`);
    if (!target) return;

    // if figure or table, modify target to get expected element
    if (id.indexOf("fig:") === 0) target = target.querySelector("figure");
    if (id.indexOf("tbl:") === 0) target = target.querySelector("table");

    return target;
  }

  // get position/dimensions of element or viewport
  function getRectInView(element) {
    let rect = {};
    rect.left = 0;
    rect.top = 0;
    rect.right = document.documentElement.clientWidth;
    rect.bottom = document.documentElement.clientHeight;
    let style = {};

    if (element instanceof HTMLElement) {
      rect = element.getBoundingClientRect();
      style = window.getComputedStyle(element);
    }

    const margin = {};
    margin.left = parseFloat(style.marginLeftWidth) || 0;
    margin.top = parseFloat(style.marginTopWidth) || 0;
    margin.right = parseFloat(style.marginRightWidth) || 0;
    margin.bottom = parseFloat(style.marginBottomWidth) || 0;

    const border = {};
    border.left = parseFloat(style.borderLeftWidth) || 0;
    border.top = parseFloat(style.borderTopWidth) || 0;
    border.right = parseFloat(style.borderRightWidth) || 0;
    border.bottom = parseFloat(style.borderBottomWidth) || 0;

    const newRect = {};
    newRect.left = rect.left + margin.left + border.left;
    newRect.top = rect.top + margin.top + border.top;
    newRect.right = rect.right + margin.right + border.right;
    newRect.bottom = rect.bottom + margin.bottom + border.bottom;
    newRect.width = newRect.right - newRect.left;
    newRect.height = newRect.bottom - newRect.top;

    return newRect;
  }

  // get position of element relative to page
  function getRectInPage(element) {
    const rect = getRectInView(element);
    const body = getRectInView(document.body);

    const newRect = {};
    newRect.left = rect.left - body.left;
    newRect.top = rect.top - body.top;
    newRect.right = rect.right - body.left;
    newRect.bottom = rect.bottom - body.top;
    newRect.width = rect.width;
    newRect.height = rect.height;

    return newRect;
  }

  // get closest element before specified element that matches query
  function firstBefore(element, query) {
    while (element && element !== document.body && !element.matches(query))
      element = element.previousElementSibling || element.parentNode;

    return element;
  }

  // check if element is part of collapsed heading
  function isCollapsed(element) {
    while (element && element !== document.body) {
      if (element.dataset.collapsed === "true") return true;
      element = element.parentNode;
    }
    return false;
  }

  // expand any collapsed parent containers of element if necessary
  function expandElement(element) {
    if (isCollapsed(element)) {
      // accordion plugin
      const heading = firstBefore(element, "h2");
      if (heading) heading.click();
      // details/summary HTML element
      const summary = firstBefore(element, "summary");
      if (summary) summary.click();
    }
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);

    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // get list of elements after a start element up to element matching query
  function nextUntil(element, query, exclude) {
    const elements = [];
    while (((element = element.nextElementSibling), element)) {
      if (element.matches(query)) break;
      if (!element.matches(exclude)) elements.push(element);
    }
    return elements;
  }
</script>
<!--
  Accordion Plugin

  Allows sections of content under h2 headings to be collapsible.
-->

<script type="module">
  // whether to always start expanded ('false'), always start collapsed
  // ('true'), or start collapsed when screen small ('auto')
  const startCollapsed = "auto";

  // start script
  function start() {
    // run through each <h2> heading
    const headings = document.querySelectorAll("h2");
    for (const heading of headings) {
      addArrow(heading);

      // start expanded/collapsed based on option
      if (
        startCollapsed === "true" ||
        (startCollapsed === "auto" && isSmallScreen()) ||
        heading.dataset.collapsed === "true"
      )
        collapseHeading(heading);
      else expandElement(heading);
    }

    // attach hash change listener to window
    window.addEventListener("hashchange", onHashChange);
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) goToElement(target);
  }

  // add arrow to heading
  function addArrow(heading) {
    // add arrow button
    const arrow = document.createElement("button");
    arrow.innerHTML = document.querySelector(".icon_angle_down").innerHTML;
    arrow.classList.add("icon_button", "accordion_arrow");
    heading.insertBefore(arrow, heading.firstChild);

    // attach click listener to heading and button
    heading.addEventListener("click", onHeadingClick);
    arrow.addEventListener("click", onArrowClick);
  }

  // determine if on mobile-like device with small screen
  function isSmallScreen() {
    return Math.min(window.innerWidth, window.innerHeight) < 480;
  }

  // when <h2> heading is clicked
  function onHeadingClick(event) {
    // only collapse if <h2> itself is target of click (eg, user did
    // not click on anchor within <h2>)
    if (event.target === this) toggleCollapse(this);
  }

  // when arrow button is clicked
  function onArrowClick() {
    toggleCollapse(this.parentNode);
  }

  // collapse section if expanded, expand if collapsed
  function toggleCollapse(heading) {
    if (heading.dataset.collapsed === "false") collapseHeading(heading);
    else expandElement(heading);
  }

  // elements to exclude from collapse, such as table of contents panel,
  // hypothesis panel, etc
  const exclude = "#toc_panel, div.annotator-frame, #lightbox_overlay";

  // collapse section
  function collapseHeading(heading) {
    heading.setAttribute("data-collapsed", "true");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "true");
  }

  // expand section
  function expandElement(heading) {
    heading.setAttribute("data-collapsed", "false");
    const children = getChildren(heading);
    for (const child of children) child.setAttribute("data-collapsed", "false");
  }

  // get list of elements between this <h2> and next <h2> or <h1>
  // ("children" of the <h2> section)
  function getChildren(heading) {
    return nextUntil(heading, "h2, h1", exclude);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle down icon -->

<template class="icon_angle_down">
  <!-- modified from: https://fontawesome.com/icons/angle-down -->
  <svg width="16" height="16" viewBox="0 0 448 512">
    <path
      fill="currentColor"
      d="M207.029 381.476L12.686 187.132c-9.373-9.373-9.373-24.569 0-33.941l22.667-22.667c9.357-9.357 24.522-9.375 33.901-.04L224 284.505l154.745-154.021c9.379-9.335 24.544-9.317 33.901.04l22.667 22.667c9.373 9.373 9.373 24.569 0 33.941L240.971 381.476c-9.373 9.372-24.569 9.372-33.942 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* accordion arrow button */
    .accordion_arrow {
      margin-right: 10px;
    }

    /* arrow icon when <h2> data-collapsed attribute true */
    h2[data-collapsed="true"] > .accordion_arrow > svg {
      transform: rotate(-90deg);
    }

    /* all elements (except <h2>'s) when data-collapsed attribute true */
    *:not(h2)[data-collapsed="true"] {
      display: none;
    }

    /* accordion arrow button when hovered and <h2>'s when hovered */
    .accordion_arrow:hover,
    h2[data-collapsed="true"]:hover,
    h2[data-collapsed="false"]:hover {
      cursor: pointer;
    }
  }

  /* always hide accordion arrow button on print */
  @media only print {
    .accordion_arrow {
      display: none;
    }
  }
</style>
<!--
  Anchors Plugin

  Adds an anchor next to each of a certain type of element that provides a
  human-readable url to that specific item/position in the document (e.g.
  "manuscript.html#abstract"). It also makes it such that scrolling out of view
  of a target removes its identifier from the url.
-->

<script type="module">
  // which types of elements to add anchors next to, in "document.querySelector"
  // format
  const typesQuery =
    'h1, h2, h3, div[id^="fig:"], div[id^="tbl:"], span[id^="eq:"]';

  // start script
  function start() {
    // add anchor to each element of specified types
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) addAnchor(element);

    // attach scroll listener to window
    window.addEventListener("scroll", onScroll);
  }

  // when window is scrolled
  function onScroll() {
    // if url has hash and user has scrolled out of view of hash
    // target, remove hash from url
    const tolerance = 100;
    const target = getHashTarget();
    if (target) {
      if (
        target.getBoundingClientRect().top > window.innerHeight + tolerance ||
        target.getBoundingClientRect().bottom < 0 - tolerance
      )
        history.pushState(null, null, " ");
    }
  }

  // add anchor to element
  function addAnchor(element) {
    let addTo; // element to add anchor button to

    // if figure or table, modify withId and addTo to get expected
    // elements
    if (element.id.indexOf("fig:") === 0) {
      addTo = element.querySelector("figcaption");
    } else if (element.id.indexOf("tbl:") === 0) {
      addTo = element.querySelector("caption");
    } else if (element.id.indexOf("eq:") === 0) {
      addTo = element.querySelector(".eqnos-number");
    }

    addTo = addTo || element;
    const id = element.id || null;

    // do not add anchor if element doesn't have assigned id.
    // id is generated by pandoc and is assumed to be unique and
    // human-readable
    if (!id) return;

    // create anchor button
    const anchor = document.createElement("a");
    anchor.innerHTML = document.querySelector(".icon_link").innerHTML;
    anchor.title = "Link to this part of the document";
    anchor.classList.add("icon_button", "anchor");
    anchor.dataset.ignore = "true";
    anchor.href = "#" + id;
    addTo.appendChild(anchor);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- link icon -->

<template class="icon_link">
  <!-- modified from: https://fontawesome.com/icons/link -->
  <svg width="16" height="16" viewBox="0 0 512 512">
    <path
      fill="currentColor"
      d="M326.612 185.391c59.747 59.809 58.927 155.698.36 214.59-.11.12-.24.25-.36.37l-67.2 67.2c-59.27 59.27-155.699 59.262-214.96 0-59.27-59.26-59.27-155.7 0-214.96l37.106-37.106c9.84-9.84 26.786-3.3 27.294 10.606.648 17.722 3.826 35.527 9.69 52.721 1.986 5.822.567 12.262-3.783 16.612l-13.087 13.087c-28.026 28.026-28.905 73.66-1.155 101.96 28.024 28.579 74.086 28.749 102.325.51l67.2-67.19c28.191-28.191 28.073-73.757 0-101.83-3.701-3.694-7.429-6.564-10.341-8.569a16.037 16.037 0 0 1-6.947-12.606c-.396-10.567 3.348-21.456 11.698-29.806l21.054-21.055c5.521-5.521 14.182-6.199 20.584-1.731a152.482 152.482 0 0 1 20.522 17.197zM467.547 44.449c-59.261-59.262-155.69-59.27-214.96 0l-67.2 67.2c-.12.12-.25.25-.36.37-58.566 58.892-59.387 154.781.36 214.59a152.454 152.454 0 0 0 20.521 17.196c6.402 4.468 15.064 3.789 20.584-1.731l21.054-21.055c8.35-8.35 12.094-19.239 11.698-29.806a16.037 16.037 0 0 0-6.947-12.606c-2.912-2.005-6.64-4.875-10.341-8.569-28.073-28.073-28.191-73.639 0-101.83l67.2-67.19c28.239-28.239 74.3-28.069 102.325.51 27.75 28.3 26.872 73.934-1.155 101.96l-13.087 13.087c-4.35 4.35-5.769 10.79-3.783 16.612 5.864 17.194 9.042 34.999 9.69 52.721.509 13.906 17.454 20.446 27.294 10.606l37.106-37.106c59.271-59.259 59.271-155.699.001-214.959z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* anchor button */
    .anchor {
      opacity: 0;
      margin-left: 5px;
    }

    /* anchor buttons within <h2>'s */
    h2 .anchor {
      margin-left: 10px;
    }

    /* anchor buttons when hovered/focused and anything containing an anchor button when hovered */
    *:hover > .anchor,
    .anchor:hover,
    .anchor:focus {
      opacity: 1;
    }

    /* anchor button when hovered */
    .anchor:hover {
      cursor: pointer;
    }
  }

  /* always show anchor button on devices with no mouse/hover ability */
  @media (hover: none) {
    .anchor {
      opacity: 1;
    }
  }

  /* always hide anchor button on print */
  @media only print {
    .anchor {
      display: none;
    }
  }
</style>
<!-- 
    Attributes Plugin

    Allows arbitrary HTML attributes to be attached to (almost) any element.
    Place an HTML comment inside or next to the desired element with the content:
    $attribute="value"
-->

<script type="module">
  // start script
  function start() {
    // get list of comments in document
    const comments = findComments();

    for (const comment of comments)
      if (comment.parentElement)
        addAttributes(comment.parentElement, comment.nodeValue.trim());
  }

  // add html attributes to specified element based on string of
  // html attributes and values
  function addAttributes(element, text) {
    // regex's for finding attribute/value pairs in the format of
    // attribute="value" or attribute='value
    const regex2 = /\$([a-zA-Z\-]+)?=\"(.+?)\"/;
    const regex1 = /\$([a-zA-Z\-]+)?=\'(.+?)\'/;

    // loop through attribute/value pairs
    let match;
    while ((match = text.match(regex2) || text.match(regex1))) {
      // get attribute and value from regex capture groups
      let attribute = match[1];
      let value = match[2];

      // remove from string
      text = text.substring(match.index + match[0].length);

      if (!attribute || !value) break;

      // set attribute of parent element
      try {
        element.setAttribute(attribute, value);
      } catch (error) {
        console.log(error);
      }

      // special case for colspan
      if (attribute === "colspan") removeTableCells(element, value);
    }
  }

  // get list of comment elements in document
  function findComments() {
    const comments = [];

    // iterate over comment nodes in document
    function acceptNode(node) {
      return NodeFilter.FILTER_ACCEPT;
    }
    const iterator = document.createNodeIterator(
      document.body,
      NodeFilter.SHOW_COMMENT,
      acceptNode
    );
    let node;
    while ((node = iterator.nextNode())) comments.push(node);

    return comments;
  }

  // remove certain number of cells after specified cell
  function removeTableCells(cell, number) {
    number = parseInt(number);
    if (!number) return;

    // remove elements
    for (; number > 1; number--) {
      if (cell.nextElementSibling) cell.nextElementSibling.remove();
    }
  }

  // start script on DOMContentLoaded instead of load to ensure this plugins
  // runs before other plugins
  window.addEventListener("DOMContentLoaded", start);
</script>
<!--
  Jump to First Plugin

  Adds a button next to each reference entry, figure, and table that jumps the
  page to the first occurrence of a link to that item in the manuscript.
-->

<script type="module">
  // whether to add buttons next to reference entries
  const references = "true";
  // whether to add buttons next to figures
  const figures = "true";
  // whether to add buttons next to tables
  const tables = "true";

  // start script
  function start() {
    if (references !== "false")
      makeButtons(`div[id^="ref-"]`, ".csl-left-margin", "reference");
    if (figures !== "false")
      makeButtons(`div[id^="fig:"]`, "figcaption", "figure");
    if (tables !== "false") makeButtons(`div[id^="tbl:"]`, "caption", "table");
  }

  // when jump button clicked
  function onButtonClick() {
    const first = getFirstOccurrence(this.dataset.id);
    if (!first) return;

    // update url hash so navigating "back" in history will return user to button
    window.location.hash = this.dataset.id;
    // scroll to link
    const timeout = function () {
      goToElement(first, window.innerHeight * 0.5);
    };
    window.setTimeout(timeout, 0);
  }

  // get first occurrence of link to item in document
  function getFirstOccurrence(id) {
    let query = "a";
    query += '[href="#' + id + '"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelector(query);
  }

  // add button next to each reference entry, figure, or table
  function makeButtons(query, containerQuery, subject) {
    const elements = document.querySelectorAll(query);
    for (const element of elements) {
      const id = element.id;
      const buttonContainer = element.querySelector(containerQuery);
      const first = getFirstOccurrence(id);

      // if can't find link to reference or place to put button, ignore
      if (!first || !buttonContainer) continue;

      // make jump button
      let button = document.createElement("button");
      button.classList.add("icon_button", "jump_arrow");
      button.title = `Jump to the first occurrence of this ${subject} in the document`;
      const icon = document.querySelector(".icon_angle_double_up");
      button.innerHTML = icon.innerHTML;
      button.dataset.id = id;
      button.dataset.ignore = "true";
      button.addEventListener("click", onButtonClick);
      buttonContainer.prepend(button);
    }
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- angle double up icon -->

<template class="icon_angle_double_up">
  <!-- modified from: https://fontawesome.com/icons/angle-double-up -->
  <svg width="16" height="16" viewBox="0 0 320 512">
    <path
      fill="currentColor"
      d="M177 255.7l136 136c9.4 9.4 9.4 24.6 0 33.9l-22.6 22.6c-9.4 9.4-24.6 9.4-33.9 0L160 351.9l-96.4 96.4c-9.4 9.4-24.6 9.4-33.9 0L7 425.7c-9.4-9.4-9.4-24.6 0-33.9l136-136c9.4-9.5 24.6-9.5 34-.1zm-34-192L7 199.7c-9.4 9.4-9.4 24.6 0 33.9l22.6 22.6c9.4 9.4 24.6 9.4 33.9 0l96.4-96.4 96.4 96.4c9.4 9.4 24.6 9.4 33.9 0l22.6-22.6c9.4-9.4 9.4-24.6 0-33.9l-136-136c-9.2-9.4-24.4-9.4-33.8 0z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* jump button */
    .jump_arrow {
      position: relative;
      top: 0.125em;
      margin-right: 5px;
    }
  }

  /* always hide jump button on print */
  @media only print {
    .jump_arrow {
      display: none;
    }
  }
</style>
<!-- 
    Lightbox Plugin

    Makes it such that when a user clicks on an image, the image fills the
    screen and the user can pan/drag/zoom the image and navigate between other
    images in the document.
-->

<script type="module">
  // list of possible zoom/scale factors
  const zooms =
    "0.1, 0.25, 0.333333, 0.5, 0.666666, 0.75, 1, 1.25, 1.5, 1.75, 2, 2.5, 3, 3.5, 4, 5, 6, 7, 8";
  // whether to fit image to view ('fit'), display at 100% and shrink if
  // necessary ('shrink'), or always display at 100% ('100')
  const defaultZoom = "fit";
  // whether to zoom in/out toward center of view ('true') or mouse ('false')
  const centerZoom = "false";

  // start script
  function start() {
    // run through each <img> element
    const imgs = document.querySelectorAll("figure > img");
    let count = 1;
    for (const img of imgs) {
      img.classList.add("lightbox_document_img");
      img.dataset.number = count;
      img.dataset.total = imgs.length;
      img.addEventListener("click", openLightbox);
      count++;
    }

    // attach mouse and key listeners to window
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("keyup", onKeyUp);
  }

  // when mouse is moved anywhere in window
  function onWindowMouseMove(event) {
    window.mouseX = event.clientX;
    window.mouseY = event.clientY;
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("lightbox_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("lightbox_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeLightbox();
        break;
    }
  }

  // open lightbox
  function openLightbox() {
    const lightbox = makeLightbox(this);
    if (!lightbox) return;

    blurBody(lightbox);
    document.body.appendChild(lightbox);
  }

  // make lightbox
  function makeLightbox(img) {
    // delete lightbox if it exists, start fresh
    closeLightbox();

    // create screen overlay containing lightbox
    const overlay = document.createElement("div");
    overlay.id = "lightbox_overlay";

    // create image info boxes
    const numberInfo = document.createElement("div");
    const zoomInfo = document.createElement("div");
    numberInfo.id = "lightbox_number_info";
    zoomInfo.id = "lightbox_zoom_info";

    // create container for image
    const imageContainer = document.createElement("div");
    imageContainer.id = "lightbox_image_container";
    const lightboxImg = makeLightboxImg(
      img,
      imageContainer,
      numberInfo,
      zoomInfo
    );
    imageContainer.appendChild(lightboxImg);

    // create bottom container for caption and navigation buttons
    const bottomContainer = document.createElement("div");
    bottomContainer.id = "lightbox_bottom_container";
    const caption = makeCaption(img);
    const prevButton = makePrevButton(img);
    const nextButton = makeNextButton(img);
    bottomContainer.appendChild(prevButton);
    bottomContainer.appendChild(caption);
    bottomContainer.appendChild(nextButton);

    // attach top middle and bottom to overlay
    overlay.appendChild(numberInfo);
    overlay.appendChild(zoomInfo);
    overlay.appendChild(imageContainer);
    overlay.appendChild(bottomContainer);

    return overlay;
  }

  // make <img> object that is intuitively draggable and zoomable
  function makeLightboxImg(sourceImg, container, numberInfoBox, zoomInfoBox) {
    // create copy of source <img>
    const img = sourceImg.cloneNode(true);
    img.classList.remove("lightbox_document_img");
    img.removeAttribute("id");
    img.removeAttribute("width");
    img.removeAttribute("height");
    img.style.position = "unset";
    img.style.margin = "0";
    img.style.padding = "0";
    img.style.width = "";
    img.style.height = "";
    img.style.minWidth = "";
    img.style.minHeight = "";
    img.style.maxWidth = "";
    img.style.maxHeight = "";
    img.id = "lightbox_img";

    // build sorted list of zoomSteps
    const zoomSteps = zooms.split(/[^0-9.]/).map((step) => parseFloat(step));
    zoomSteps.sort((a, b) => a - b);

    // <img> object property variables
    let zoom = 1;
    let translateX = 0;
    let translateY = 0;
    let clickMouseX = undefined;
    let clickMouseY = undefined;
    let clickTranslateX = undefined;
    let clickTranslateY = undefined;

    updateNumberInfo();

    // update image numbers displayed in info box
    function updateNumberInfo() {
      numberInfoBox.innerHTML =
        sourceImg.dataset.number + " of " + sourceImg.dataset.total;
    }

    // update zoom displayed in info box
    function updateZoomInfo() {
      let zoomInfo = zoom * 100;
      if (!Number.isInteger(zoomInfo)) zoomInfo = zoomInfo.toFixed(2);
      zoomInfoBox.innerHTML = zoomInfo + "%";
    }

    // move to closest zoom step above current zoom
    const zoomIn = function () {
      for (const zoomStep of zoomSteps) {
        if (zoomStep > zoom) {
          zoom = zoomStep;
          break;
        }
      }
      updateTransform();
    };

    // move to closest zoom step above current zoom
    const zoomOut = function () {
      zoomSteps.reverse();
      for (const zoomStep of zoomSteps) {
        if (zoomStep < zoom) {
          zoom = zoomStep;
          break;
        }
      }
      zoomSteps.reverse();

      updateTransform();
    };

    // update display of <img> based on scale/translate properties
    const updateTransform = function () {
      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      // get new width/height after scale
      const rect = img.getBoundingClientRect();
      // limit translate
      translateX = Math.max(translateX, -rect.width / 2);
      translateX = Math.min(translateX, rect.width / 2);
      translateY = Math.max(translateY, -rect.height / 2);
      translateY = Math.min(translateY, rect.height / 2);

      // set transform
      img.style.transform =
        "translate(" +
        (translateX || 0) +
        "px," +
        (translateY || 0) +
        "px) scale(" +
        (zoom || 1) +
        ")";

      updateZoomInfo();
    };

    // fit <img> to container
    const fit = function () {
      // no x/y offset, 100% zoom by default
      translateX = 0;
      translateY = 0;
      zoom = 1;

      // widths of <img> and container
      const imgWidth = img.naturalWidth;
      const imgHeight = img.naturalHeight;
      const containerWidth = parseFloat(
        window.getComputedStyle(container).width
      );
      const containerHeight = parseFloat(
        window.getComputedStyle(container).height
      );

      // how much zooming is needed to fit <img> to container
      const xRatio = imgWidth / containerWidth;
      const yRatio = imgHeight / containerHeight;
      const maxRatio = Math.max(xRatio, yRatio);
      const newZoom = 1 / maxRatio;

      // fit <img> to container according to option
      if (defaultZoom === "shrink") {
        if (maxRatio > 1) zoom = newZoom;
      } else if (defaultZoom === "fit") zoom = newZoom;

      updateTransform();
    };

    // when mouse wheel is rolled anywhere in container
    const onContainerWheel = function (event) {
      if (!event) return;

      // let ctrl + mouse wheel to zoom behave as normal
      if (event.ctrlKey) return;

      // prevent normal scroll behavior
      event.preventDefault();
      event.stopPropagation();

      // point around which to scale img
      const viewRect = container.getBoundingClientRect();
      const viewX = (viewRect.left + viewRect.right) / 2;
      const viewY = (viewRect.top + viewRect.bottom) / 2;
      const originX = centerZoom === "true" ? viewX : mouseX;
      const originY = centerZoom === "true" ? viewY : mouseY;

      // get point on image under origin
      const oldRect = img.getBoundingClientRect();
      const oldPercentX = (originX - oldRect.left) / oldRect.width;
      const oldPercentY = (originY - oldRect.top) / oldRect.height;

      // increment/decrement zoom
      if (event.deltaY < 0) zoomIn();
      if (event.deltaY > 0) zoomOut();

      // get offset between previous image point and origin
      const newRect = img.getBoundingClientRect();
      const offsetX = originX - (newRect.left + newRect.width * oldPercentX);
      const offsetY = originY - (newRect.top + newRect.height * oldPercentY);

      // translate image to keep image point under origin
      translateX += offsetX;
      translateY += offsetY;

      // perform translate
      updateTransform();
    };

    // when container is clicked
    function onContainerClick(event) {
      // if container itself is target of click, and not other
      // element above it
      if (event.target === this) closeLightbox();
    }

    // when mouse button is pressed on image
    const onImageMouseDown = function (event) {
      // store original mouse position relative to image
      clickMouseX = window.mouseX;
      clickMouseY = window.mouseY;
      clickTranslateX = translateX;
      clickTranslateY = translateY;
      event.stopPropagation();
      event.preventDefault();
    };

    // when mouse button is released anywhere in window
    const onWindowMouseUp = function (event) {
      // reset original mouse position
      clickMouseX = undefined;
      clickMouseY = undefined;
      clickTranslateX = undefined;
      clickTranslateY = undefined;

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mouseup", onWindowMouseUp);
    };

    // when mouse is moved anywhere in window
    const onWindowMouseMove = function (event) {
      if (
        clickMouseX === undefined ||
        clickMouseY === undefined ||
        clickTranslateX === undefined ||
        clickTranslateY === undefined
      )
        return;

      // offset image based on original and current mouse position
      translateX = clickTranslateX + window.mouseX - clickMouseX;
      translateY = clickTranslateY + window.mouseY - clickMouseY;
      updateTransform();
      event.preventDefault();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("mousemove", onWindowMouseMove);
    };

    // when window is resized
    const onWindowResize = function (event) {
      fit();

      // remove global listener if lightbox removed from document
      if (!document.body.contains(container))
        window.removeEventListener("resize", onWindowResize);
    };

    // attach the necessary event listeners
    img.addEventListener("dblclick", fit);
    img.addEventListener("mousedown", onImageMouseDown);
    container.addEventListener("wheel", onContainerWheel);
    container.addEventListener("mousedown", onContainerClick);
    container.addEventListener("touchstart", onContainerClick);
    window.addEventListener("mouseup", onWindowMouseUp);
    window.addEventListener("mousemove", onWindowMouseMove);
    window.addEventListener("resize", onWindowResize);

    // run fit() after lightbox atttached to document and <img> Loaded
    // so needed container and img dimensions available
    img.addEventListener("load", fit);

    return img;
  }

  // make caption
  function makeCaption(img) {
    const caption = document.createElement("div");
    caption.id = "lightbox_caption";
    const captionSource = img.nextElementSibling;
    if (captionSource.tagName.toLowerCase() === "figcaption") {
      const captionCopy = makeCopy(captionSource);
      caption.innerHTML = captionCopy.innerHTML;
    }

    caption.addEventListener("touchstart", function (event) {
      event.stopPropagation();
    });

    return caption;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // make button to jump to previous image in document
  function makePrevButton(img) {
    const prevButton = document.createElement("button");
    prevButton.id = "lightbox_prev_button";
    prevButton.title = "Jump to the previous image in the document [←]";
    prevButton.classList.add("icon_button", "lightbox_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;

    // attach click listeners to button
    prevButton.addEventListener("click", function () {
      getPrevImg(img).click();
    });

    return prevButton;
  }

  // make button to jump to next image in document
  function makeNextButton(img) {
    const nextButton = document.createElement("button");
    nextButton.id = "lightbox_next_button";
    nextButton.title = "Jump to the next image in the document [→]";
    nextButton.classList.add("icon_button", "lightbox_button");
    nextButton.innerHTML = document.querySelector(
      ".icon_caret_right"
    ).innerHTML;

    // attach click listeners to button
    nextButton.addEventListener("click", function () {
      getNextImg(img).click();
    });

    return nextButton;
  }

  // get previous image in document
  function getPrevImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if < 1
    if (index - 1 >= 0) index--;
    else index = imgs.length - 1;
    return imgs[index];
  }

  // get next image in document
  function getNextImg(img) {
    const imgs = document.querySelectorAll(".lightbox_document_img");

    // find index of provided img
    let index;
    for (index = 0; index < imgs.length; index++) {
      if (imgs[index] === img) break;
    }

    // wrap index to other side if > total
    if (index + 1 <= imgs.length - 1) index++;
    else index = 0;
    return imgs[index];
  }

  // close lightbox
  function closeLightbox() {
    focusBody();

    const lightbox = document.getElementById("lightbox_overlay");
    if (lightbox) lightbox.remove();
  }

  // make all elements behind lightbox non-focusable
  function blurBody(overlay) {
    const all = document.querySelectorAll("*");
    for (const element of all) element.tabIndex = -1;
    document.body.classList.add("body_no_scroll");
  }

  // make all elements focusable again
  function focusBody() {
    const all = document.querySelectorAll("*");
    for (const element of all) element.removeAttribute("tabIndex");
    document.body.classList.remove("body_no_scroll");
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* regular <img> in document when hovered */
    img.lightbox_document_img:hover {
      cursor: pointer;
    }

    .body_no_scroll {
      overflow: hidden !important;
    }

    /* screen overlay */
    #lightbox_overlay {
      display: flex;
      flex-direction: column;
      position: fixed;
      left: 0;
      top: 0;
      right: 0;
      bottom: 0;
      background: rgba(0, 0, 0, 0.75);
      z-index: 3;
    }

    /* middle area containing lightbox image */
    #lightbox_image_container {
      flex-grow: 1;
      display: flex;
      justify-content: center;
      align-items: center;
      overflow: hidden;
      position: relative;
      padding: 20px;
    }

    /* bottom area containing caption */
    #lightbox_bottom_container {
      display: flex;
      justify-content: center;
      align-items: center;
      height: 100px;
      min-height: 100px;
      max-height: 100px;
      background: rgba(0, 0, 0, 0.5);
    }

    /* image number info text box */
    #lightbox_number_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      left: 2px;
      top: 0;
      z-index: 4;
    }

    /* zoom info text box */
    #lightbox_zoom_info {
      position: absolute;
      color: #ffffff;
      font-weight: 600;
      right: 2px;
      top: 0;
      z-index: 4;
    }

    /* copy of image caption */
    #lightbox_caption {
      box-sizing: border-box;
      display: inline-block;
      width: 100%;
      max-height: 100%;
      padding: 10px 0;
      text-align: center;
      overflow-y: auto;
      color: #ffffff;
    }

    /* navigation previous/next button */
    .lightbox_button {
      width: 100px;
      height: 100%;
      min-width: 100px;
      min-height: 100%;
      color: #ffffff;
    }

    /* navigation previous/next button when hovered */
    .lightbox_button:hover {
      background: none !important;
    }

    /* navigation button icon */
    .lightbox_button > svg {
      height: 25px;
    }

    /* figure auto-number */
    #lightbox_caption > span:first-of-type {
      font-weight: bold;
      margin-right: 5px;
    }

    /* lightbox image when hovered */
    #lightbox_img:hover {
      cursor: grab;
    }

    /* lightbox image when grabbed */
    #lightbox_img:active {
      cursor: grabbing;
    }
  }

  /* when on screen < 480px wide */
  @media only screen and (max-width: 480px) {
    /* make navigation buttons skinnier on small screens to make more room for caption text */
    .lightbox_button {
      width: 50px;
      min-width: 50px;
    }
  }

  /* always hide lightbox on print */
  @media only print {
    #lightbox_overlay {
      display: none;
    }
  }
</style>
<!-- 
  Link Highlight Plugin

  Makes it such that when a user hovers or focuses a link, other links that have
  the same target will be highlighted. It also makes it such that when clicking
  a link, the target of the link (eg reference, figure, table) is briefly
  highlighted.
-->

<script type="module">
  // whether to also highlight links that go to external urls
  const externalLinks = "false";
  // whether user must click off to unhighlight instead of just
  // un-hovering
  const clickUnhighlight = "false";
  // whether to also highlight links that are unique
  const highlightUnique = "true";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach mouse and focus listeners to link
      link.addEventListener("mouseenter", onLinkFocus);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("mouseleave", onLinkUnhover);
    }

    // attach click and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("hashchange", onHashChange);

    // run hash change on window load in case user has navigated
    // directly to hash
    onHashChange();
  }

  // when link is focused (tabbed to) or hovered
  function onLinkFocus() {
    highlight(this);
  }

  // when link is unhovered
  function onLinkUnhover() {
    if (clickUnhighlight !== "true") unhighlightAll();
  }

  // when the mouse is clicked anywhere in window
  function onClick(event) {
    unhighlightAll();
  }

  // when hash (eg manuscript.html#introduction) changes
  function onHashChange() {
    const target = getHashTarget();
    if (target) glowElement(target);
  }

  // start glow sequence on an element
  function glowElement(element) {
    const startGlow = function () {
      onGlowEnd();
      element.dataset.glow = "true";
      element.addEventListener("animationend", onGlowEnd);
    };
    const onGlowEnd = function () {
      element.removeAttribute("data-glow");
      element.removeEventListener("animationend", onGlowEnd);
    };
    startGlow();
  }

  // highlight link and all others with same target
  function highlight(link) {
    // force unhighlight all to start fresh
    unhighlightAll();

    // get links with same target
    if (!link) return;
    const sameLinks = getSameLinks(link);

    // if link unique and option is off, exit and don't highlight
    if (sameLinks.length <= 1 && highlightUnique !== "true") return;

    // highlight all same links, and "select" (special highlight) this
    // one
    for (const sameLink of sameLinks) {
      if (sameLink === link) sameLink.setAttribute("data-selected", "true");
      else sameLink.setAttribute("data-highlighted", "true");
    }
  }

  // unhighlight all links
  function unhighlightAll() {
    const links = getLinks();
    for (const link of links) {
      link.setAttribute("data-selected", "false");
      link.setAttribute("data-highlighted", "false");
    }
  }

  // get links with same target
  function getSameLinks(link) {
    const results = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        results.push(otherLink);
    }
    return results;
  }

  // get all links of types we wish to handle
  function getLinks() {
    let query = "a";
    if (externalLinks !== "true") query += '[href^="#"]';
    // exclude buttons, anchor links, toc links, etc
    query += ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    return document.querySelectorAll(query);
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<style>
  @media only screen {
    /* anything with data-highlighted attribute true */
    [data-highlighted="true"] {
      background: #ffeb3b;
    }

    /* anything with data-selected attribute true */
    [data-selected="true"] {
      background: #ff8a65 !important;
    }

    /* animation definition for glow */
    @keyframes highlight_glow {
      0% {
        background: none;
      }
      10% {
        background: #bbdefb;
      }
      100% {
        background: none;
      }
    }

    /* anything with data-glow attribute true */
    [data-glow="true"] {
      animation: highlight_glow 2s;
    }
  }
</style>
<!--
  Table of Contents Plugin

  Provides a "table of contents" (toc) panel on the side of the document that
  allows the user to conveniently navigate between sections of the document.
-->

<script type="module">
  // which types of elements to add links for, in "document.querySelector" format
  const typesQuery = "h1, h2, h3";
  // whether toc starts open. use 'true' or 'false', or 'auto' to
  // use 'true' behavior when screen wide enough and 'false' when not
  const startOpen = "false";
  // whether toc closes when clicking on toc link. use 'true' or
  // 'false', or 'auto' to use 'false' behavior when screen wide
  // enough and 'true' when not
  const clickClose = "auto";
  // if list item is more than this many characters, text will be
  // truncated
  const charLimit = "50";
  // whether or not to show bullets next to each toc item
  const bullets = "false";

  // start script
  function start() {
    // make toc panel and populate with entries (links to document
    // sections)
    const panel = makePanel();
    if (!panel) return;
    makeEntries(panel);
    // attach panel to document after making entries, so 'toc' heading
    // in panel isn't included in toc
    document.body.insertBefore(panel, document.body.firstChild);

    // initial panel state
    if (startOpen === "true" || (startOpen === "auto" && !isSmallScreen()))
      openPanel();
    else closePanel();

    // attach click, scroll, and hash change listeners to window
    window.addEventListener("click", onClick);
    window.addEventListener("scroll", onScroll);
    window.addEventListener("hashchange", onScroll);
    window.addEventListener("keyup", onKeyUp);
    onScroll();

    // add class to push document body down out of way of toc button
    document.body.classList.add("toc_body_nudge");
  }

  // determine if screen wide enough to fit toc panel
  function isSmallScreen() {
    // in default theme:
    // 816px = 8.5in = width of "page" (<body>) element
    // 260px = min width of toc panel (*2 for both sides of <body>)
    return window.innerWidth < 816 + 260 * 2;
  }

  // when mouse is clicked anywhere in window
  function onClick() {
    if (isSmallScreen()) closePanel();
  }

  // when window is scrolled or hash changed
  function onScroll() {
    highlightViewed();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    // close on esc
    if (event.key === "Escape") closePanel();
  }

  // find entry of currently viewed document section in toc and highlight
  function highlightViewed() {
    const firstId = getFirstInView(typesQuery);

    // get toc entries (links), unhighlight all, then highlight viewed
    const list = document.getElementById("toc_list");
    if (!firstId || !list) return;
    const links = list.querySelectorAll("a");
    for (const link of links) link.dataset.viewing = "false";
    const link = list.querySelector('a[href="#' + firstId + '"]');
    if (!link) return;
    link.dataset.viewing = "true";
  }

  // get first or previous toc listed element in top half of view
  function getFirstInView(query) {
    // get all elements matching query and with id
    const elements = document.querySelectorAll(query);
    const elementsWithIds = [];
    for (const element of elements) {
      if (element.id) elementsWithIds.push(element);
    }

    // get first or previous element in top half of view
    for (let i = 0; i < elementsWithIds.length; i++) {
      const element = elementsWithIds[i];
      const prevElement = elementsWithIds[Math.max(0, i - 1)];
      if (element.getBoundingClientRect().top >= 0) {
        if (element.getBoundingClientRect().top < window.innerHeight / 2)
          return element.id;
        else return prevElement.id;
      }
    }
  }

  // make panel
  function makePanel() {
    // create panel
    const panel = document.createElement("div");
    panel.id = "toc_panel";
    if (bullets === "true") panel.dataset.bullets = "true";

    // create header
    const header = document.createElement("div");
    header.id = "toc_header";

    // create toc button
    const button = document.createElement("button");
    button.id = "toc_button";
    button.innerHTML = document.querySelector(".icon_th_list").innerHTML;
    button.title = "Table of Contents";
    button.classList.add("icon_button");

    // create header text
    const text = document.createElement("h4");
    text.innerHTML = "Table of Contents";

    // create container for toc list
    const list = document.createElement("div");
    list.id = "toc_list";

    // attach click listeners
    panel.addEventListener("click", onPanelClick);
    header.addEventListener("click", onHeaderClick);
    button.addEventListener("click", onButtonClick);

    // attach elements
    header.appendChild(button);
    header.appendChild(text);
    panel.appendChild(header);
    panel.appendChild(list);

    return panel;
  }

  // create toc entries (links) to each element of the specified types
  function makeEntries(panel) {
    const elements = document.querySelectorAll(typesQuery);
    for (const element of elements) {
      // do not add link if element doesn't have assigned id
      if (!element.id) continue;

      // create link/list item
      const link = document.createElement("a");
      link.classList.add("toc_link");
      switch (element.tagName.toLowerCase()) {
        case "h1":
          link.dataset.level = "1";
          break;
        case "h2":
          link.dataset.level = "2";
          break;
        case "h3":
          link.dataset.level = "3";
          break;
        case "h4":
          link.dataset.level = "4";
          break;
      }
      link.title = element.innerText;
      let text = element.innerText;
      if (text.length > charLimit) text = text.slice(0, charLimit) + "...";
      link.innerHTML = text;
      link.href = "#" + element.id;
      link.addEventListener("click", onLinkClick);

      // attach link
      panel.querySelector("#toc_list").appendChild(link);
    }
  }

  // when panel is clicked
  function onPanelClick(event) {
    // stop click from propagating to window/document and closing panel
    event.stopPropagation();
  }

  // when header itself is clicked
  function onHeaderClick(event) {
    togglePanel();
  }

  // when button is clicked
  function onButtonClick(event) {
    togglePanel();
    // stop header underneath button from also being clicked
    event.stopPropagation();
  }

  // when link is clicked
  function onLinkClick(event) {
    if (clickClose === "true" || (clickClose === "auto" && isSmallScreen()))
      closePanel();
    else openPanel();
  }

  // open panel if closed, close if opened
  function togglePanel() {
    const panel = document.getElementById("toc_panel");
    if (!panel) return;

    if (panel.dataset.open === "true") closePanel();
    else openPanel();
  }

  // open panel
  function openPanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "true";
  }

  // close panel
  function closePanel() {
    const panel = document.getElementById("toc_panel");
    if (panel) panel.dataset.open = "false";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- th list icon -->

<template class="icon_th_list">
  <!-- modified from: https://fontawesome.com/icons/th-list -->
  <svg width="16" height="16" viewBox="0 0 512 512" tabindex="-1">
    <path
      fill="currentColor"
      d="M96 96c0 26.51-21.49 48-48 48S0 122.51 0 96s21.49-48 48-48 48 21.49 48 48zM48 208c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm0 160c-26.51 0-48 21.49-48 48s21.49 48 48 48 48-21.49 48-48-21.49-48-48-48zm96-236h352c8.837 0 16-7.163 16-16V76c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16zm0 160h352c8.837 0 16-7.163 16-16v-40c0-8.837-7.163-16-16-16H144c-8.837 0-16 7.163-16 16v40c0 8.837 7.163 16 16 16z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* toc panel */
    #toc_panel {
      box-sizing: border-box;
      position: fixed;
      top: 0;
      left: 0;
      background: #ffffff;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      z-index: 2;
    }

    /* toc panel when closed */
    #toc_panel[data-open="false"] {
      min-width: 60px;
      width: 60px;
      height: 60px;
      border-right: solid 1px #bdbdbd;
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc panel when open */
    #toc_panel[data-open="true"] {
      min-width: 260px;
      max-width: 480px;
      /* keep panel edge consistent distance away from "page" edge */
      width: calc(((100vw - 8.5in) / 2) - 30px - 40px);
      bottom: 0;
      border-right: solid 1px #bdbdbd;
    }

    /* toc panel header */
    #toc_header {
      box-sizing: border-box;
      display: flex;
      flex-direction: row;
      align-items: center;
      height: 60px;
      margin: 0;
      padding: 20px;
    }

    /* toc panel header when hovered */
    #toc_header:hover {
      cursor: pointer;
    }

    /* toc panel header when panel open */
    #toc_panel[data-open="true"] > #toc_header {
      border-bottom: solid 1px #bdbdbd;
    }

    /* toc open/close header button */
    #toc_button {
      margin-right: 20px;
    }

    /* hide toc list and header text when closed */
    #toc_panel[data-open="false"] > #toc_header > *:not(#toc_button),
    #toc_panel[data-open="false"] > #toc_list {
      display: none;
    }

    /* toc list of entries */
    #toc_list {
      box-sizing: border-box;
      width: 100%;
      padding: 20px;
      position: absolute;
      top: calc(60px + 1px);
      bottom: 0;
      overflow: auto;
    }

    /* toc entry, link to section in document */
    .toc_link {
      display: block;
      padding: 5px;
      position: relative;
      font-weight: 600;
      text-decoration: none;
    }

    /* toc entry when hovered or when "viewed" */
    .toc_link:hover,
    .toc_link[data-viewing="true"] {
      background: #f5f5f5;
    }

    /* toc entry, level 1 indentation */
    .toc_link[data-level="1"] {
      margin-left: 0;
    }

    /* toc entry, level 2 indentation */
    .toc_link[data-level="2"] {
      margin-left: 20px;
    }

    /* toc entry, level 3 indentation */
    .toc_link[data-level="3"] {
      margin-left: 40px;
    }

    /* toc entry, level 4 indentation */
    .toc_link[data-level="4"] {
      margin-left: 60px;
    }

    /* toc entry bullets */
    #toc_panel[data-bullets="true"] .toc_link[data-level]:before {
      position: absolute;
      left: -15px;
      top: -1px;
      font-size: 1.5em;
    }

    /* toc entry, level 2 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="2"]:before {
      content: "\2022";
    }

    /* toc entry, level 3 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="3"]:before {
      content: "\25AB";
    }

    /* toc entry, level 4 bullet */
    #toc_panel[data-bullets="true"] .toc_link[data-level="4"]:before {
      content: "-";
    }
  }

  /* when on screen < 8.5in wide */
  @media only screen and (max-width: 8.5in) {
    /* push <body> ("page") element down to make room for toc icon */
    .toc_body_nudge {
      padding-top: 60px;
    }

    /* toc icon when panel closed and not hovered */
    #toc_panel[data-open="false"]:not(:hover) {
      background: rgba(255, 255, 255, 0.75);
    }
  }

  /* always hide toc panel on print */
  @media only print {
    #toc_panel {
      display: none;
    }
  }
</style>
<!-- 
  Tooltips Plugin

  Makes it such that when the user hovers or focuses a link to a citation or
  figure, a tooltip appears with a preview of the reference content, along with
  arrows to navigate between instances of the same reference in the document.
-->

<script type="module">
  // whether user must click off to close tooltip instead of just un-hovering
  const clickClose = "false";
  // delay (in ms) between opening and closing tooltip
  const delay = "100";

  // start script
  function start() {
    const links = getLinks();
    for (const link of links) {
      // attach hover and focus listeners to link
      link.addEventListener("mouseover", onLinkHover);
      link.addEventListener("mouseleave", onLinkUnhover);
      link.addEventListener("focus", onLinkFocus);
      link.addEventListener("touchend", onLinkTouch);
    }

    // attach mouse, key, and resize listeners to window
    window.addEventListener("mousedown", onClick);
    window.addEventListener("touchstart", onClick);
    window.addEventListener("keyup", onKeyUp);
    window.addEventListener("resize", onResize);
  }

  // when link is hovered
  function onLinkHover() {
    // function to open tooltip
    const delayOpenTooltip = function () {
      openTooltip(this);
    }.bind(this);

    // run open function after delay
    this.openTooltipTimer = window.setTimeout(delayOpenTooltip, delay);
  }

  // when mouse leaves link
  function onLinkUnhover() {
    // cancel opening tooltip
    window.clearTimeout(this.openTooltipTimer);

    // don't close on unhover if option specifies
    if (clickClose === "true") return;

    // function to close tooltip
    const delayCloseTooltip = function () {
      // if tooltip open and if mouse isn't over tooltip, close
      const tooltip = document.getElementById("tooltip");
      if (tooltip && !tooltip.matches(":hover")) closeTooltip();
    };

    // run close function after delay
    this.closeTooltipTimer = window.setTimeout(delayCloseTooltip, delay);
  }

  // when link is focused (tabbed to)
  function onLinkFocus(event) {
    openTooltip(this);
  }

  // when link is touched on touch screen
  function onLinkTouch(event) {
    // attempt to force hover state on first tap always, and trigger
    // regular link click (and navigation) on second tap
    if (event.target === document.activeElement) event.target.click();
    else {
      document.activeElement.blur();
      event.target.focus();
    }
    if (event.cancelable) event.preventDefault();
    event.stopPropagation();
    return false;
  }

  // when mouse is clicked anywhere in window
  function onClick(event) {
    closeTooltip();
  }

  // when key pressed
  function onKeyUp(event) {
    if (!event || !event.key) return;

    switch (event.key) {
      // trigger click of prev button
      case "ArrowLeft":
        const prevButton = document.getElementById("tooltip_prev_button");
        if (prevButton) prevButton.click();
        break;
      // trigger click of next button
      case "ArrowRight":
        const nextButton = document.getElementById("tooltip_next_button");
        if (nextButton) nextButton.click();
        break;
      // close on esc
      case "Escape":
        closeTooltip();
        break;
    }
  }

  // when window is resized or zoomed
  function onResize() {
    closeTooltip();
  }

  // get all links of types we wish to handle
  function getLinks() {
    const queries = [];
    // exclude buttons, anchor links, toc links, etc
    const exclude =
      ":not(.button):not(.icon_button):not(.anchor):not(.toc_link)";
    queries.push('a[href^="#ref-"]' + exclude); // citation links
    queries.push('a[href^="#fig:"]' + exclude); // figure links
    const query = queries.join(", ");
    return document.querySelectorAll(query);
  }

  // get links with same target, get index of link in set, get total
  // same links
  function getSameLinks(link) {
    const sameLinks = [];
    const links = getLinks();
    for (const otherLink of links) {
      if (otherLink.getAttribute("href") === link.getAttribute("href"))
        sameLinks.push(otherLink);
    }

    return {
      elements: sameLinks,
      index: sameLinks.indexOf(link),
      total: sameLinks.length,
    };
  }

  // open tooltip
  function openTooltip(link) {
    // delete tooltip if it exists, start fresh
    closeTooltip();

    // make tooltip element
    const tooltip = makeTooltip(link);

    // if source couldn't be found and tooltip not made, exit
    if (!tooltip) return;

    // make navbar elements
    const navBar = makeNavBar(link);
    if (navBar) tooltip.firstElementChild.appendChild(navBar);

    // attach tooltip to page
    document.body.appendChild(tooltip);

    // position tooltip
    const position = function () {
      positionTooltip(link);
    };
    position();

    // if tooltip contains images, position again after they've loaded
    const imgs = tooltip.querySelectorAll("img");
    for (const img of imgs) img.addEventListener("load", position);
  }

  // close (delete) tooltip
  function closeTooltip() {
    const tooltip = document.getElementById("tooltip");
    if (tooltip) tooltip.remove();
  }

  // make tooltip
  function makeTooltip(link) {
    // get target element that link points to
    const source = getSource(link);

    // if source can't be found, exit
    if (!source) return;

    // create new tooltip
    const tooltip = document.createElement("div");
    tooltip.id = "tooltip";
    const tooltipContent = document.createElement("div");
    tooltipContent.id = "tooltip_content";
    tooltip.appendChild(tooltipContent);

    // make copy of source node and put in tooltip
    const sourceCopy = makeCopy(source);
    tooltipContent.appendChild(sourceCopy);

    // attach mouse event listeners
    tooltip.addEventListener("click", onTooltipClick);
    tooltip.addEventListener("mousedown", onTooltipClick);
    tooltip.addEventListener("touchstart", onTooltipClick);
    tooltip.addEventListener("mouseleave", onTooltipUnhover);

    // (for interaction with lightbox plugin)
    // transfer click on tooltip copied img to original img
    const sourceImg = source.querySelector("img");
    const sourceCopyImg = sourceCopy.querySelector("img");
    if (sourceImg && sourceCopyImg) {
      const clickImg = function () {
        sourceImg.click();
        closeTooltip();
      };
      sourceCopyImg.addEventListener("click", clickImg);
    }

    return tooltip;
  }

  // make carbon copy of html dom element
  function makeCopy(source) {
    const sourceCopy = source.cloneNode(true);

    // delete elements marked with ignore (eg anchor and jump buttons)
    const deleteFromCopy = sourceCopy.querySelectorAll('[data-ignore="true"]');
    for (const element of deleteFromCopy) element.remove();

    // delete certain element attributes
    const attributes = [
      "id",
      "data-collapsed",
      "data-selected",
      "data-highlighted",
      "data-glow",
      "class",
    ];
    for (const attribute of attributes) {
      sourceCopy.removeAttribute(attribute);
      const elements = sourceCopy.querySelectorAll("[" + attribute + "]");
      for (const element of elements) element.removeAttribute(attribute);
    }

    return sourceCopy;
  }

  // when tooltip is clicked
  function onTooltipClick(event) {
    // when user clicks on tooltip, stop click from transferring
    // outside of tooltip (eg, click off to close tooltip, or eg click
    // off to unhighlight same refs)
    event.stopPropagation();
  }

  // when tooltip is unhovered
  function onTooltipUnhover(event) {
    if (clickClose === "true") return;

    // make sure new mouse/touch/focus no longer over tooltip or any
    // element within it
    const tooltip = document.getElementById("tooltip");
    if (!tooltip) return;
    if (this.contains(event.relatedTarget)) return;

    closeTooltip();
  }

  // make nav bar to go betwen prev/next instances of same reference
  function makeNavBar(link) {
    // find other links to the same source
    const sameLinks = getSameLinks(link);

    // don't show nav bar when singular reference
    if (sameLinks.total <= 1) return;

    // find prev/next links with same target
    const prevLink = getPrevLink(link, sameLinks);
    const nextLink = getNextLink(link, sameLinks);

    // create nav bar
    const navBar = document.createElement("div");
    navBar.id = "tooltip_nav_bar";
    const text = sameLinks.index + 1 + " of " + sameLinks.total;

    // create nav bar prev/next buttons
    const prevButton = document.createElement("button");
    const nextButton = document.createElement("button");
    prevButton.id = "tooltip_prev_button";
    nextButton.id = "tooltip_next_button";
    prevButton.title =
      "Jump to the previous occurence of this item in the document [←]";
    nextButton.title =
      "Jump to the next occurence of this item in the document [→]";
    prevButton.classList.add("icon_button");
    nextButton.classList.add("icon_button");
    prevButton.innerHTML = document.querySelector(".icon_caret_left").innerHTML;
    nextButton.innerHTML =
      document.querySelector(".icon_caret_right").innerHTML;
    navBar.appendChild(prevButton);
    navBar.appendChild(document.createTextNode(text));
    navBar.appendChild(nextButton);

    // attach click listeners to buttons
    prevButton.addEventListener("click", function () {
      onPrevNextClick(link, prevLink);
    });
    nextButton.addEventListener("click", function () {
      onPrevNextClick(link, nextLink);
    });

    return navBar;
  }

  // get previous link with same target
  function getPrevLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if < 1
    let index;
    if (sameLinks.index - 1 >= 0) index = sameLinks.index - 1;
    else index = sameLinks.total - 1;
    return sameLinks.elements[index];
  }

  // get next link with same target
  function getNextLink(link, sameLinks) {
    if (!sameLinks) sameLinks = getSameLinks(link);
    // wrap index to other side if > total
    let index;
    if (sameLinks.index + 1 <= sameLinks.total - 1) index = sameLinks.index + 1;
    else index = 0;
    return sameLinks.elements[index];
  }

  // get element that is target of link or url hash
  function getSource(link) {
    const hash = link ? link.hash : window.location.hash;
    const id = hash.slice(1);
    let target = document.querySelector('[id="' + id + '"]');
    if (!target) return;

    // if ref or figure, modify target to get expected element
    if (id.indexOf("ref-") === 0) target = target.querySelector(":nth-child(2)");
    else if (id.indexOf("fig:") === 0) target = target.querySelector("figure");

    return target;
  }

  // when prev/next arrow button is clicked
  function onPrevNextClick(link, prevNextLink) {
    if (link && prevNextLink)
      goToElement(prevNextLink, window.innerHeight * 0.5);
  }

  // scroll to and focus element
  function goToElement(element, offset) {
    // expand accordion section if collapsed
    expandElement(element);
    const y =
      getRectInView(element).top -
      getRectInView(document.documentElement).top -
      (offset || 0);
    // trigger any function listening for "onscroll" event
    window.dispatchEvent(new Event("scroll"));
    window.scrollTo(0, y);
    document.activeElement.blur();
    element.focus();
  }

  // determine position to place tooltip based on link position in
  // viewport and tooltip size
  function positionTooltip(link, left, top) {
    const tooltipElement = document.getElementById("tooltip");
    if (!tooltipElement) return;

    // get convenient vars for position/dimensions of
    // link/tooltip/page/view
    link = getRectInPage(link);
    const tooltip = getRectInPage(tooltipElement);
    const view = getRectInPage();

    // horizontal positioning
    if (left)
      // use explicit value
      left = left;
    else if (link.left + tooltip.width < view.right)
      // fit tooltip to right of link
      left = link.left;
    else if (link.right - tooltip.width > view.left)
      // fit tooltip to left of link
      left = link.right - tooltip.width;
    // center tooltip in view
    else left = (view.right - view.left) / 2 - tooltip.width / 2;

    // vertical positioning
    if (top)
      // use explicit value
      top = top;
    else if (link.top - tooltip.height > view.top)
      // fit tooltip above link
      top = link.top - tooltip.height;
    else if (link.bottom + tooltip.height < view.bottom)
      // fit tooltip below link
      top = link.bottom;
    else {
      // center tooltip in view
      top = view.top + view.height / 2 - tooltip.height / 2;
      // nudge off of link to left/right if possible
      if (link.right + tooltip.width < view.right) left = link.right;
      else if (link.left - tooltip.width > view.left)
        left = link.left - tooltip.width;
    }

    tooltipElement.style.left = left + "px";
    tooltipElement.style.top = top + "px";
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- caret left icon -->

<template class="icon_caret_left">
  <!-- modified from: https://fontawesome.com/icons/caret-left -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M192 127.338v257.324c0 17.818-21.543 26.741-34.142 14.142L29.196 270.142c-7.81-7.81-7.81-20.474 0-28.284l128.662-128.662c12.599-12.6 34.142-3.676 34.142 14.142z"
    ></path>
  </svg>
</template>

<!-- caret right icon -->

<template class="icon_caret_right">
  <!-- modified from: https://fontawesome.com/icons/caret-right -->
  <svg width="16" height="16" viewBox="0 0 192 512">
    <path
      fill="currentColor"
      d="M0 384.662V127.338c0-17.818 21.543-26.741 34.142-14.142l128.662 128.662c7.81 7.81 7.81 20.474 0 28.284L34.142 398.804C21.543 411.404 0 402.48 0 384.662z"
    ></path>
  </svg>
</template>

<style>
  @media only screen {
    /* tooltip container */
    #tooltip {
      position: absolute;
      width: 50%;
      min-width: 240px;
      max-width: 75%;
      z-index: 1;
    }

    /* tooltip content */
    #tooltip_content {
      margin-bottom: 5px;
      padding: 20px;
      border-radius: 5px;
      border: solid 1px #bdbdbd;
      box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
      background: #ffffff;
      overflow-wrap: break-word;
    }

    /* tooltip copy of paragraphs and figures */
    #tooltip_content > p,
    #tooltip_content > figure {
      margin: 0;
      max-height: 320px;
      overflow-y: auto;
    }

    /* tooltip copy of <img> */
    #tooltip_content > figure > img,
    #tooltip_content > figure > svg {
      max-height: 260px;
    }

    /* navigation bar */
    #tooltip_nav_bar {
      margin-top: 10px;
      text-align: center;
    }

    /* navigation bar previous/next buton */
    #tooltip_nav_bar > .icon_button {
      position: relative;
      top: 3px;
    }

    /* navigation bar previous button */
    #tooltip_nav_bar > .icon_button:first-of-type {
      margin-right: 5px;
    }

    /* navigation bar next button */
    #tooltip_nav_bar > .icon_button:last-of-type {
      margin-left: 5px;
    }
  }

  /* always hide tooltip on print */
  @media only print {
    #tooltip {
      display: none;
    }
  }
</style>
<!--
  Analytics Plugin (third-party) 
  
  Copy and paste code from Google Analytics or similar service here.
-->
<!-- 
  Annotations Plugin

  Allows public annotation of the  manuscript. See https://web.hypothes.is/.
-->

<script type="module">
  // configuration
  window.hypothesisConfig = function () {
    return {
      branding: {
        accentColor: "#2196f3",
        appBackgroundColor: "#f8f8f8",
        ctaBackgroundColor: "#f8f8f8",
        ctaTextColor: "#000000",
        selectionFontFamily: "Open Sans, Helvetica, sans serif",
        annotationFontFamily: "Open Sans, Helvetica, sans serif",
      },
    };
  };

  // hypothesis client script
  const embed = "https://hypothes.is/embed.js";
  // hypothesis annotation count query url
  const query = "https://api.hypothes.is/api/search?limit=0&url=";

  // start script
  function start() {
    const button = makeButton();
    document.body.insertBefore(button, document.body.firstChild);
    insertCount(button);
  }

  // make button
  function makeButton() {
    // create button
    const button = document.createElement("button");
    button.id = "hypothesis_button";
    button.innerHTML = document.querySelector(".icon_hypothesis").innerHTML;
    button.title = "Hypothesis annotations";
    button.classList.add("icon_button");

    function onClick(event) {
      onButtonClick(event, button);
    }

    // attach click listeners
    button.addEventListener("click", onClick);

    return button;
  }

  // insert annotations count
  async function insertCount(button) {
    // get annotation count from Hypothesis based on url
    let count = "-";
    try {
      const canonical = document.querySelector('link[rel="canonical"]');
      const location = window.location;
      const url = encodeURIComponent((canonical || location).href);
      const response = await fetch(query + url);
      const json = await response.json();
      count = json.total || "-";
    } catch (error) {
      console.log(error);
    }

    // put count into button
    const counter = document.createElement("span");
    counter.id = "hypothesis_count";
    counter.innerHTML = count;
    button.title = "View " + count + " Hypothesis annotations";
    button.append(counter);
  }

  // when button is clicked
  function onButtonClick(event, button) {
    const script = document.createElement("script");
    script.src = embed;
    document.body.append(script);
    button.remove();
  }

  // start script when document is finished loading
  window.addEventListener("load", start);
</script>

<!-- hypothesis icon -->

<template class="icon_hypothesis">
  <!-- modified from: https://simpleicons.org/icons/hypothesis.svg / https://git.io/Jf1VB -->
  <svg width="16" height="16" viewBox="0 0 24 24" tabindex="-1">
    <path
      fill="currentColor"
      d="M3.43 0C2.5 0 1.72 .768 1.72 1.72V18.86C1.72 19.8 2.5 20.57 3.43 20.57H9.38L12 24L14.62 20.57H20.57C21.5 20.57 22.29 19.8 22.29 18.86V1.72C22.29 .77 21.5 0 20.57 0H3.43M5.14 3.43H7.72V9.43S8.58 7.72 10.28 7.72C12 7.72 13.74 8.57 13.74 11.24V17.14H11.16V12C11.16 10.61 10.28 10.07 9.43 10.29C8.57 10.5 7.72 11.41 7.72 13.29V17.14H5.14V3.43M18 13.72C18.95 13.72 19.72 14.5 19.72 15.42A1.71 1.71 0 0 1 18 17.13A1.71 1.71 0 0 1 16.29 15.42C16.29 14.5 17.05 13.71 18 13.71Z"
      tabindex="-1"
    ></path>
  </svg>
</template>

<style>
  /* hypothesis activation button */
  #hypothesis_button {
    box-sizing: border-box;
    position: fixed;
    top: 0;
    right: 0;
    width: 60px;
    height: 60px;
    background: #ffffff;
    border-radius: 0;
    border-left: solid 1px #bdbdbd;
    border-bottom: solid 1px #bdbdbd;
    box-shadow: 0 0 20px rgba(0, 0, 0, 0.05);
    z-index: 2;
  }

  /* hypothesis button svg */
  #hypothesis_button > svg {
    position: relative;
    top: -4px;
  }

  /* hypothesis annotation count */
  #hypothesis_count {
    position: absolute;
    left: 0;
    right: 0;
    bottom: 5px;
  }

  /* side panel */
  .annotator-frame {
    width: 280px !important;
  }

  /* match highlight color to rest of theme */
  .annotator-highlights-always-on .annotator-hl {
    background-color: #ffeb3b !important;
  }

  /* match focused color to rest of theme */
  .annotator-hl.annotator-hl-focused {
    background-color: #ff8a65 !important;
  }

  /* match bucket bar color to rest of theme */
  .annotator-bucket-bar {
    background: #f5f5f5 !important;
  }

  /* always hide button, toolbar, and tooltip on print */
  @media only print {
    #hypothesis_button {
      display: none;
    }

    .annotator-frame {
      display: none !important;
    }

    hypothesis-adder {
      display: none !important;
    }
  }
</style>
<!-- 
  Mathjax Plugin (third-party) 

  Allows the proper rendering of math/equations written in LaTeX.
  See https://www.mathjax.org/.
-->

<script type="text/x-mathjax-config">
  // configuration
  MathJax.Hub.Config({
    "CommonHTML": { linebreaks: { automatic: true } },
    "HTML-CSS": { linebreaks: { automatic: true } },
    "SVG": { linebreaks: { automatic: true } },
    "fast-preview": { disabled: true }
  });
</script>

<script
  src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.5/MathJax.js?config=TeX-MML-AM_CHTML"
  integrity="sha512-0doc9hKxR3PYwso42RD1p5ySZpzzuDiOwMrdCEh2WdJZCjcmFKc/wEnL+z8fBQrnHoiNWbo+3fiGkOYXBdQp4A=="
  crossorigin="anonymous"
></script>

<style>
  /* mathjax containers */
  .math.display > span:not(.MathJax_Preview) {
    /* turn inline element (no dimensions) into block (allows fixed width and thus scrolling) */
    display: flex !important;
    overflow-x: auto !important;
    overflow-y: hidden !important;
    justify-content: center;
    align-items: center;
    margin: 0 !important;
  }

  /* right click menu */
  .MathJax_Menu {
    border-radius: 5px !important;
    border: solid 1px #bdbdbd !important;
    box-shadow: none !important;
  }

  /* equation auto-number */
  span[id^="eq:"] > span.math.display + span {
    font-weight: 600;
  }

  /* equation */
  span[id^="eq:"] > span.math.display > span {
    /* nudge to make room for equation auto-number and anchor */
    margin-right: 60px !important;
  }
</style>
</body>
</html>
